{
  "cells": [
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "0AlThSbST6WF"
      },
      "outputs": [],
      "source": [
        "import requests\n",
        "import xml.etree.ElementTree as ET\n",
        "import pandas as pd\n",
        "import numpy as np\n",
        "import time\n",
        "from tqdm import tqdm"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "AlgM1-od3xH7"
      },
      "outputs": [],
      "source": [
        "def search_pubmed(query, max_results=1000):\n",
        "    ### API ENDPOINT DIRECTLY DERIVED FROM URL: https://www.ncbi.nlm.nih.gov/books/NBK25500/#chapter1.Searching_a_Database\n",
        "    base_url = \"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/\"\n",
        "    # search_url = f\"{base_url}esearch.fcgi?db=pubmed&term={query}&retmax={max_results}&usehistory=y\"\n",
        "    search_url = f\"{base_url}esearch.fcgi?db=pmc&term={query}+AND+open+access[filter]&retmax={max_results}&usehistory=y\"\n",
        "    ### END OF API ENDPOINT DIRECTLY DERIVED FROM URL\n",
        "\n",
        "    print(\"Initiate paper-ids fetch...\")\n",
        "    response = requests.get(search_url)\n",
        "    print(\"Completed id fetch.\")\n",
        "    root = ET.fromstring(response.content)\n",
        "\n",
        "    id_list = [id_elem.text for id_elem in root.findall(\".//Id\")]\n",
        "    print(\"Total IDs: \", len(id_list))\n",
        "\n",
        "    return id_list\n",
        "\n",
        "def fetch_paper_details(id_list, fetch_full_text = False):\n",
        "    ### API ENDPOINT DIRECTLY DERIVED FROM URL: https://www.ncbi.nlm.nih.gov/books/NBK25500/#chapter1.Searching_a_Database\n",
        "    base_url = \"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/\"\n",
        "    fetch_url = f\"{base_url}efetch.fcgi?db=pmc&id={','.join(id_list)}&retmode=xml\"\n",
        "    ### END OF API ENDPOINT DIRECTLY DERIVED FROM URL\n",
        "\n",
        "    print(\"Start fetching papers. Total request count: \", len(id_list))\n",
        "    response = requests.get(fetch_url)\n",
        "\n",
        "    root = None\n",
        "    if response and response.status_code == 200:\n",
        "        print(\"Completed request\")\n",
        "        root = ET.fromstring(response.content)\n",
        "\n",
        "    else:\n",
        "        print(response.status_code)\n",
        "\n",
        "    articles = root.findall(\".//article\")\n",
        "    print(\"Total article count: \", len(articles))\n",
        "    papers = []\n",
        "    if root and articles:\n",
        "        for article in tqdm(articles):\n",
        "            pmid = article.find(\".//article-meta/article-id[@pub-id-type='pmid']\")\n",
        "            if pmid is not None:\n",
        "                pmid = pmid.text if pmid.text else None\n",
        "            pmcid = article.find(\".//article-meta/article-id[@pub-id-type='pmc']\")\n",
        "            if pmcid is not None:\n",
        "                pmcid = pmcid.text if pmcid.text else None\n",
        "\n",
        "\n",
        "            title = article.find(\".//article-title\").text\n",
        "\n",
        "            # if not type(title) == type(\"\"):\n",
        "            #     title = title.text\n",
        "\n",
        "            abstract = article.find(\".//abstract\")\n",
        "            abstract = \" \".join(list(abstract.itertext())) if abstract else None\n",
        "\n",
        "            fulltext = \"\\n\".join([str(e.text) for e in find_tags(article.findall(\".//body\"), {'title', 'p'}) if e.text])\n",
        "\n",
        "            methods_materials = None\n",
        "            results = None\n",
        "            discussion = None\n",
        "            conclusion = None\n",
        "            for sec in article.findall(\".//body//sec\"):\n",
        "                if \"sec-type\" in sec.attrib.keys():\n",
        "                    if \"methods\" in sec.attrib[\"sec-type\"] or \"material\" in sec.attrib[\"sec-type\"]:\n",
        "                        sectitle = sec.find(\".//title\")\n",
        "                        if sectitle is not None:\n",
        "                            sectitle = sectitle.text if sectitle.text else \"\"\n",
        "                        methods_materials =  str(sectitle) + \"\\n\" + \" \".join([str(p.text) for p in sec.findall(\".//p\") if p.text is not None])\n",
        "                    if \"results\" in sec.attrib[\"sec-type\"]:\n",
        "                        sectitle = sec.find(\".//title\")\n",
        "                        if sectitle is not None:\n",
        "                            sectitle = sectitle.text if sectitle.text else \"\"\n",
        "                        results = str(sectitle) + \"\\n\" + \" \".join([str(p.text) for p in sec.findall(\".//p\") if p.text is not None])\n",
        "                    if \"discussion\" in sec.attrib[\"sec-type\"]:\n",
        "                        sectitle = sec.find(\".//title\")\n",
        "                        if sectitle is not None:\n",
        "                            sectitle = sectitle.text if sectitle.text else \"\"\n",
        "                        discussion = str(sectitle) + \"\\n\" + \" \".join([str(p.text) for p in sec.findall(\".//p\") if p.text is not None])\n",
        "                    if \"conclusion\" in sec.attrib[\"sec-type\"]:\n",
        "                        sectitle = sec.find(\".//title\")\n",
        "                        if sectitle is not None:\n",
        "                            sectitle = sectitle.text if sectitle.text else \"\"\n",
        "                        conclusion = str(sectitle) + \"\\n\" + \" \".join([str(p.text) for p in sec.findall(\".//p\") if p.text is not None])\n",
        "\n",
        "            author_cotrib = article.find(\".//contrib-group\")\n",
        "            if author_cotrib:\n",
        "                authors = \", \".join([\" \".join(name.itertext()) for name in author_cotrib.findall(\".//contrib/name\")])\n",
        "\n",
        "            papers.append({\n",
        "                \"pmid\": pmid,\n",
        "                \"pmcid\": pmcid,\n",
        "                \"title\": title,\n",
        "                \"abstract\": abstract,\n",
        "                \"authors\": authors,\n",
        "                \"methods_materials\": methods_materials,\n",
        "                \"results\": results,\n",
        "                \"conclusion\": conclusion,\n",
        "                \"fulltext\": fulltext\n",
        "            })\n",
        "\n",
        "    return papers\n",
        "\n",
        "def find_tags(element, tags):\n",
        "    result = []\n",
        "    for child in element:\n",
        "        if child.tag in tags:\n",
        "            result.append(child)\n",
        "        result.extend(find_tags(child, tags))\n",
        "    return result\n"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Rough Testing"
      ],
      "metadata": {
        "id": "13h9n_dlBwjL"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "m48gMBRauOd9"
      },
      "outputs": [],
      "source": [
        "query = \"Alzheimer's Disease Biomarkers\"\n",
        "# id_list = search_pubmed(query, max_results=1000)\n",
        "# chunks = np.array_split(id_list, 3)\n",
        "\n",
        "base_url = \"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/\"\n",
        "# search_url = f\"{base_url}esearch.fcgi?db=pubmed&term={query}&retmax={max_results}&usehistory=y\"\n",
        "search_url = f\"{base_url}esearch.fcgi?db=pmc&term={query}+AND+open+access[filter]&retmax={20}&usehistory=y\"\n",
        "\n",
        "response = requests.get(search_url)\n",
        "root = ET.fromstring(response.content)\n",
        "id_list = [id.text for id in root.findall(\".//Id\")]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "xzwatht4vmqQ"
      },
      "outputs": [],
      "source": [
        "search_url = f\"{base_url}efetch.fcgi?db=pmc&id={','.join(chunks[0])}&retmode=xml\"\n",
        "response = requests.get(search_url)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "ak3Siy8u0iim"
      },
      "outputs": [],
      "source": [
        "root = ET.fromstring(response.content)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "cw92Mhkpk6p8"
      },
      "outputs": [],
      "source": [
        "\n",
        "# articles = root.findall(\".//article\")\n",
        "# for article in tqdm(articles):\n",
        "    # pmid = article.find(\".//article-meta/article-id[@pub-id-type='pmid']\")\n",
        "    # pmid = pmid.text if pmid.text else None\n",
        "    # pmcid = article.find(\".//article-meta/article-id[@pub-id-type='pmc']\")\n",
        "    # pmcid = pmcid.text if pmcid else None\n",
        "\n",
        "    # title = article.find(\".//article-title\").text\n",
        "\n",
        "\n",
        "    # abstract = article.find(\".//abstract\")\n",
        "    # abstract = \" \".join(list(abstract.itertext())) if abstract else None\n",
        "    # fulltext = \"\\n\".join([str(e.text) for e in find_tags(article.findall(\".//body\"), {'title', 'p'}) if e.text])\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "wtjByWNk3yxn",
        "outputId": "83626609-ae5b-4612-d7b3-247f2db36e1c"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Initiate paper-ids fetch...\n",
            "Completed id fetch.\n",
            "Total IDs:  1000\n",
            "Start fetching papers. Total request count:  334\n",
            "Completed request\n",
            "Total article count:  334\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "100%|██████████| 334/334 [00:00<00:00, 1061.81it/s]\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Starting merge, papers scraped: 334\n",
            "Start fetching papers. Total request count:  333\n",
            "Completed request\n",
            "Total article count:  333\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "100%|██████████| 333/333 [00:00<00:00, 1140.44it/s]\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Starting merge, papers scraped: 333\n",
            "Start fetching papers. Total request count:  333\n",
            "Completed request\n",
            "Total article count:  333\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "100%|██████████| 333/333 [00:00<00:00, 444.81it/s]\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Starting merge, papers scraped: 333\n"
          ]
        }
      ],
      "source": [
        "# Search for papers\n",
        "query = \"Alzheimer's Disease Biomarkers\"\n",
        "id_list = search_pubmed(query, max_results=1000)\n",
        "chunks = np.array_split(id_list, 3)\n",
        "df = None\n",
        "\n",
        "# Fetch paper details\n",
        "for chunk in chunks:\n",
        "    papers = fetch_paper_details(chunk)\n",
        "    if len(chunk) != 0 and len(papers) == 0:\n",
        "        print(\"Retrying scrape\")\n",
        "        time.sleep(3)\n",
        "        papers = fetch_paper_details(chunk)\n",
        "    print(f\"Starting merge, papers scraped: {len(papers)}\")\n",
        "    df = pd.concat([df, pd.DataFrame(papers)])\n",
        "    time.sleep(3)\n",
        "df.reset_index(drop=True, inplace=True)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "m-B2DYcBxVYx",
        "outputId": "7f74db24-a51f-4dc3-d470-318c9ac01282"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 1000,\n  \"fields\": [\n    {\n      \"column\": \"pmid\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 994,\n        \"samples\": [\n          \"39329952\",\n          \"39274884\",\n          \"39273143\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"pmcid\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1000,\n        \"samples\": [\n          \"11398445\",\n          \"11445186\",\n          \"11445035\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 993,\n        \"samples\": [\n          \"Disulfidptosis: A New Target for Parkinson\\u2019s Disease and Cancer\",\n          \"Isolation of Anti-Prion Compounds from \",\n          \"Heme Oxygenase-1 and Prostate Cancer: Function, Regulation, and Implication in Cancer Therapy\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 979,\n        \"samples\": [\n          \"Excessive production and response to Type I interferons (IFNs) is a hallmark of systemic lupus erythematosus (SLE). Neuropsychiatric lupus (NPSLE) is a common manifestation of human SLE, with major depression as the most common presentation. Clinical studies have demonstrated that IFN\\u03b1 can cause depressive symptoms. We have shown that the kallikrein\\u2013kinin system (KKS) [comprised of kallikreins (Klks) and bradykinins] and angiotensin-converting enzyme inhibitors suppressed Type I IFN responses in dendritic cells from lupus-prone mice and human peripheral blood mononuclear cells. Tissue Klk genes are decreased in patients with lupus, and giving exogenous Klk1 ameliorated kidney pathology in mice. We retro-orbitally administered mouse  klk1  gene-carrying adenovirus in the Murphy Roths Large lymphoproliferative (MRL/lpr) lupus-prone mice at early disease onset and analyzed immune responses and depressive-like behavior. Klk1 improved depressive-like behavior, suppressed interferon-responsive genes and neuroinflammation, and reduced plasma IFN\\u03b1 levels and proinflammatory cytokines. Klk1 also reduced IFNAR1 and JAK1 protein expression, important upstream molecules in Type I IFN signaling. Klk1 reduced bradykinin B1 receptor expression, which is known to induce proinflammatory response. Together, these findings suggest that Klk1 may be a potential therapeutic candidate to control IFN\\u03b1 production/responses and other inflammatory responses in SLE and NPSLE.\",\n          \"Amyloid-beta peptide (A\\u03b2) is a neurotoxic constituent of senile plaques in the brains of Alzheimer\\u2019s disease (AD) patients. The detailed mechanisms by which protein kinase C-delta (PKC\\u03b4) contributes to A\\u03b2 toxicity is not yet entirely understood. Using fully differentiated primary rat cortical neurons, we found that inhibition of A\\u03b225-35-induced PKC\\u03b4 increased cell viability with restoration of neuronal morphology. Using cyclin D1, proliferating cell nuclear antigen (PCNA), and histone H3 phosphorylated at Ser-10 (p-Histone H3) as the respective markers for the G1-, S-, and G2/M-phases, PKC\\u03b4 inhibition mitigated cell cycle reentry (CCR) and subsequent caspase-3 cleavage induced by both A\\u03b225-35 and A\\u03b21-42 in the post-mitotic cortical neurons. Upstream of PKC\\u03b4, signal transducers and activators of transcription (STAT)-3 mediated PKC\\u03b4 induction, CCR, and caspase-3 cleavage upon A\\u03b2 exposure. Downstream of PKC\\u03b4, aberrant neuronal CCR was triggered by overactivating cyclin-dependent kinase-5 (CDK5) via calpain2-dependent p35 cleavage into p25. Finally, PKC\\u03b4 and CDK5 also contributed to A\\u03b225-35 induction of p53-upregulated modulator of apoptosis (PUMA) in cortical neurons. Together, we demonstrated that, in the post-mitotic neurons exposed to A\\u03b2s, STAT3-dependent PKC\\u03b4 expression triggers calpain2-mediated p35 cleavage into p25 to overactivate CDK5, thus leading to aberrant CCR, PUMA induction, caspase-3 cleavage, and ultimately apoptosis.\",\n          \"Cell membranes are crucial elements in living organisms, serving as protective barriers and providing structural support for cells. They regulate numerous exchange and communication processes between cells and their environment, including interactions with other cells, tissues, ions, xenobiotics, and drugs. However, the complexity and heterogeneity of cell membranes\\u2014comprising two asymmetric layers with varying compositions across different cell types and states (e.g., healthy vs. diseased)\\u2014along with the challenges of manipulating real cell membranes represent significant obstacles for in vivo studies. To address these challenges, researchers have developed various methodologies to create model cell membranes or membrane fragments, including mono- or bilayers organized in planar systems. These models facilitate fundamental studies on membrane component interactions as well as the interactions of membrane components with external agents, such as drugs, nanoparticles (NPs), or biomarkers. The applications of model cell membranes have extended beyond basic research, encompassing areas such as biosensing and nanoparticle camouflage to evade immune detection. In this review, we highlight advancements in the engineering of planar model cell membranes, focusing on the nanoarchitectonic tools used for their fabrication. We also discuss approaches for incorporating challenging materials, such as proteins and enzymes, into these models. Finally, we present our view on future perspectives in the field of planar model cell membranes.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"authors\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 987,\n        \"samples\": [\n          \"Razavi S. Adeleh, Kalari Mohadeseh, Haghzad Tahereh, Haddadi Fatemeh, Nasiri Shirzad, Hedayati Mehdi\",\n          \"Chen Yisheng, Fan Zhongcheng, Luo Zhiwen, Kang Xueran, Wan Renwen, Li Fangqi, Lin Weiwei, Han Zhihua, Qi Beijie, Lin Jinrong, Sun Yaying, Huang Jiebin, Xu Yuzhen, Chen Shiyi\",\n          \"Wu Yongqi, Zhang Zhida, Chen Lin, Sun Shisheng\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"methods_materials\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 289,\n        \"samples\": [\n          \"2 Materials and methods\\nAll analyses were conducted using Python ( To develop the EnsembleOmicsAE model, we used the following proteomics datasets available through the AD Knowledge Portal ( Feature selection was performed independently on the female and male datasets to preserve sex-specific molecular events linked with AD. Additionally, we applied two approaches, i.e., the f-statistic and the mutual information algorithm, to select features that are associated with the outcome (control or AD) both in a linear and non-linear manner. After separating the combined dataset into a female and a male set, we applied the ANOVA F-test to rank the proteins based on the extent to which their expression levels are statistically different between the control and diseased groups. Additionally, the mutual information (MI) score ( \\n All autoencoder models were built using the TensorFlow ( \\n To calculate the importance of the input features (i.e., the proteins) relative to each neuron in the latent layer, we devised the following algorithm: Perturbed data and score calculation: Original mutual information, median score, and score variability calculation: Feature selection criteria: \\n \\n i: index for input features in  n: index for latent features in  j: index for the number of perturbation iterations (1\\u2264j \\u2264 20). I, N, and J: ranges representing the total number of input features (I = 1,080), latent features (N = 7), and iterations (J = 20), respectively. The shuffle () function is applied to the  \\n \\n mutual_info_regression (): function from the library scikit-learn ( shuffle (): function from the library NumPy (numpy.random.shuffle) ( median (): function from the library NumPy (numpy.median) ( To calculate the importance of each protein in determining the condensed data representation (i.e., the latent features), we applied the following approach for each of the proteins that are used as an input for the model. The expression values for one protein were randomly scrambled across the 559 samples, in order to alter the relation between protein expression levels and diagnosis. The scrambled values were re-positioned within the set of all proteins (1,080 protein expression values, across the 559 samples). Then, the ensemble encoder was used to predict new latent features ( To obtain the protein clusters based on linear methods ( \\n RNA-sequencing data for VIM expression were obtained from the Mayo clinic and ROSMAP studies. RNA-seq data from the Mayo study represented both the cerebellum (syn5201007) and the temporal cortex (syn4650265), while data from the ROSMAP study were for the bulk brain (syn3505720). For each data set, relative RNA abundance was calculated by normalizing individual FPKM values to the median value for each protein, before log Plasma protein abundances over healthy aging were obtained from  \\n\",\n          \"Patients and methods\\nThis study was conducted at the University Hospital Center Zagreb from February 2018 to April 2023. The research was approved by the Ethics Committee of the University Hospital Center Zagreb and the Ethics Committee of the University of Zagreb School of Medicine. The study enrolled patients who underwent drainage placement due to suspected iNPH manifesting as cognitive impairment, gait disturbances, and urinary incontinence. To quantify cognitive impairment, the Mini-Mental Status Examination (MMSE) was used. The Japanese iNPH scale (JiNPHS) was used to quantify other symptoms. A MMSE score of less than 24 indicated significant cognitive deterioration ( Patients were required to meet the neuroradiological criteria of iNPH, including ventriculomegaly, especially of the frontal and temporal horns, Evans index >0.3, bulging of the corpus callosum more cranially, callosal angle <90\\u00b0, significant expansion of the Sylvian fissure disproportionate to the expansion of the convexity sulcus (especially in the parietal region), and increased signal in T2-measured periventricular MRI sequences. In patients with clinical and radiological suspicion of iNPH, standard testing involved placing an external lumbar drain and draining a large amount of CSF (\\u2009~\\u200910 mL/h) over a 72-hour period. The exclusion criteria were underlying neurological or non-neurological disorders that can cause the same symptoms and a history of severe brain trauma. The opening CSF pressure during ELD placement had to be within reference values, meaning \\u226420 cm H Flow diagram of patient selection. MRI \\u2013 magnetic resonance imaging; iNPH \\u2013 idiopathic normal-pressure hydrocephalus; MMSE \\u2013 Mini-mental State Examination; JiNPHS \\u2013 Japanese iNPH Scale; CSF \\u2013 cerebrospinal fluid. Lumbar drainage was placed between the intervertebral space L Immediately after ELD placement, 1-2 mL of CSF was sampled for biomarker quantification (\\u03b2-amyloid proteins [A\\u03b2 1-42, A\\u03b2 1-40], total tau [t-tau], and phosphorylated-tau [p-tau]). Sampling was repeated after 36 h and 72 h. CSF samples were stored in Eppendorf pure polypropylene tubes (Eppendorf, Merck KGaA, Darmstadt, Germany). After centrifugation at 2000\\u2009\\u00d7\\u2009g for 10 min, CSF samples were stored at -80 \\u00b0C until further analysis. The biomarker levels in CSF were determined with enzyme-linked immunosorbent assay according to the manufacturer's protocols: A\\u03b21-42 with Innotest \\u03b2-amyloid 1-42; total tau with Innotest hTau Ag; and p-tau181 with Innotest Phospho-Tau (181P) (all by Fujirebio, Gent, Belgium). Since biomarker concentrations can sometimes vary significantly ( Continuous variables are presented as median (range or interquartile range), and nominal variables as absolute (relative) frequency. Differences between the groups in continuous variables were assessed with the Mann-Whitney U test. Differences between the groups in nominal variables were assessed with the \\u03c7\",\n          \"2. Methods\\nIn this single-center case-control study, we enrolled 37 patients with probable or definite ALS, admitted to the Neurology Unit of Modena University Hospital from 2015 to 2020. These patients underwent lumbar puncture (LP) as part of their diagnostic process and had at least 0.7 mL of CSF and serum stored at the Neurobiobank of Modena. Healthy controls (HCs;  The study received approval from the Ethical Committee of Area Vasta Emilia Nord (file number: 63/2022). All participants provided informed consent for the LP procedure, biobanking, and participation in research studies. We gathered a comprehensive set of demographic and clinical variables for patients including onset site, genotype, comorbidities, diagnostic latency, body mass index (BMI), forced vital capacity (FVC%), weight loss, and the ALS Functional Rating Scale\\u2014Revised (ALSFRS-R) score at sampling and last observation. The progression rate from onset to sampling and from sampling to last observation was calculated as previously reported [ The time to generalization was defined as the duration from the onset to the spread of clinical signs from the spinal or bulbar localization to both. Tracheostomy-free survival was calculated from the onset to tracheostomy or death. Patients were classified as \\u201cslow progressors\\u201d if their progression rate at sampling was \\u22640.9 points/month, and as \\u201cfast progressors\\u201d if the progression rate was >0.9 points/month. Samples were processed within two hours after collection and centrifuged for 10 min at 1300\\u00d7  Neurosteroid quantification in CSF was performed using liquid chromatography\\u2013electrospray tandem mass spectrometry (LC-MS/MS). All standards and chemicals were purchased from Sigma-Aldrich (St. Louis, MO, USA). Internal standards with isotope labeling were the following: 5\\u03b1-pregnan-3\\u03b1-ol-20-one-17\\u03b1, 21, 21, 21-d4, and sodium pregnenolone-17\\u03b1, 21, 21, 21-d4 sulfate. All solvents for high-performance LC/electrospray ionization tandem MS (HPLC/ESI-MS/MS) were LC-MS purity grade, whereas other solvents used for sample preparation were analytical grade). A stock solution was serially diluted with methanol to obtain calibration solutions at ten concentrations. The internal standard solution was prepared separately. Sample processing was performed as previously described [ Given the non-normal and left-skewed distribution of neurosteroid concentrations, the Mann\\u2013Whitney U test for two-group comparisons and the Kruskal\\u2013Wallis test for multiple-group comparisons were used, with a post hoc Dunn correction for multiple comparisons. Correlations between biomarkers and clinical variables were evaluated using Spearman\\u2019s rank correlation test. To specifically evaluate the role of age and sex on neurosteroid concentrations in ALS patients and healthy controls, we performed a robust regression analysis. Age and sex were included as independent variables, and neurosteroid concentrations were the dependent variable. An interaction term between age and sex was also examined to assess whether the effect of sex on neurosteroid levels varied with age. Tracheostomy-free survival was analyzed using univariate Cox regression analysis, followed by multivariate Cox proportional hazard modeling using a stepwise backward selection method with a retention criterion of  Data analysis was conducted with STATA (Stata Corp (2017) Stata Statistical Software: Release 15. College Station, TX, USA: StataCorp LLC).\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"results\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 419,\n        \"samples\": [\n          \"3. Results and Discussion\\nThe fabrication of the biosensor is shown in  Several variables were optimized, including the mass of the graphene, concentration of the DNA, immobilization time of the DNA, and hybridization time of the RNA, to obtain the highest response of the biosensor with the shortest detection time. It is crucial to consider graphene loading for an electrochemical biosensor. By analyzing the loadings tested (2 \\u03bcg, 4 \\u03bcg, 10 \\u03bcg, and 75 \\u03bcg of graphene), a balanced approach was chosen to ensure reliable detection without compromising sensitivity or causing interference. As shown in  It is essential to consider various DNA concentrations and their effects on sensor performance to optimize DNA concentration for the designed biosensor. Five different DNA concentrations, 0.1 \\u03bcM, 1 \\u03bcM, 2 \\u03bcM, 2.5 \\u03bcM, and 10 \\u03bcM, were analyzed, and the signal from the sensor was observed.  The effect of DNA immobilization time was studied to determine its impact on the biosensor\\u2019s performance. Various immobilization times were chosen, including 30 min, 60 min, 90 min, and 120 min.  Optimizing the hybridization time for the electrochemical biosensor is crucial for enhancing its performance. Four different time durations were chosen to explore the impact of the time intervals on the sensor\\u2019s efficiency, namely 30 min, 60 min, 90 min, and 120 min. As shown in  The specificity of the developed biosensor was evaluated using two non-complementary miRNA sequences, as shown in  Understanding the stability of the biosensor is crucial to ensure it provides reliable and consistent behavior. The stability experiments were carried out using two different methods. In the first method, the DNA-immobilized biosensor was evaluated for stability over 0, 3, 7, and 10 days. The biosensor was stored in an immobilization buffer at 4 \\u00b0C. Once removed, the miRNA was added and allowed to hybridize. As shown in  Therefore, stability experiments were conducted using the PBSE-modified biosensor in the second method. It was stored in 1 mM PB at 4 \\u00b0C and evaluated for stability over 0, 3, 7, and 10 days. As shown in  The standard method was applied to buffer and serum samples to evaluate the potential application of the proposed biosensor. For this experiment, a two-fold diluted serum was incubated on the surface for one hour to reduce the matrix effect. Different concentrations of synthetic samples were incubated directly with the constructed interface. \",\n          \"RESULTS\\nRNA-seq data (n = 34) were utilized to identify DEGs between NP and control (hereafter named \\u201cNP dataset\\u201d) and between EM and control (hereafter named \\u201cEM dataset\\u201d). In total, 1,877 DEGs were obtained of which 1,327 genes (70.7%) were upregulated and 550 genes (29.3%) down-regulated from the NP dataset ( KEGG, Kyoto Encyclopedia of Genes and Genomes; NET, neutrophil extracellular trap; NP, nasal polyp; EM, ethmoidal mucosa. In total, 1,124 DEGs were found in the EM dataset ( In  The differential gene expression between T2- and non-T2-NP, were used to identify functional difference. In total, 1,790 and 2,012 DEGs were found between T2-NP and controls (hereafter named the \\u201cT2-NP dataset\\u201d) ( T2, type 2; KEGG, Kyoto Encyclopedia of Genes and Genomes; NET, neutrophil extracellular trap; NP, nasal polyp; EM, ethmoidal mucosa. The KEGG pathway analysis showed that cytokine-cytokine receptor interaction (36 genes), NET formation (32 genes) and hematopoietic cell lineage (28 genes) were significantly enriched in up-regulated genes. cAMP signaling pathway (13 genes), salivary secretion (11 genes) and pancreatic secretion (9 genes) were enriched in down-regulated genes from T2-NP dataset ( The KEGG results of the non-T2-NP dataset showed that NET formation (52 genes), SLE (50 genes) and cytokine-cytokine receptor interaction (36 genes) were significantly enriched in up-regulated genes. cAMP signaling pathway (17 genes), salivary secretion (12 genes) and pancreatic secretion (10 genes) were enriched in down-regulated genes ( A Venn diagram showed that 558 genes in the T2-NP dataset were not shared with the non-T2-NP dataset ( Next, 740 genes were found between T2-EM and controls (hereafter named the \\u201cT2-EM dataset\\u201d) ( T2, type 2; KEGG, Kyoto Encyclopedia of Genes and Genomes; NET, neutrophil extracellular trap; NP, nasal polyp; EM, ethmoidal mucosa. In the non-T2-EM dataset, cytokine-cytokine receptor interaction (40 genes), phagosome (35 genes), SLE (34 genes) and NET formation (32 genes) were enriched in up-regulated genes. Additionally, pancreatic secretion (9 genes) and salivary secretion (8 genes) were enriched in down-regulated genes ( We represented 272 non-shared genes of T2-EM dataset and 1,085 non-shared genes of non-T2-EM dataset using a Venn diagram ( Between T2-NP and T2-EM, 792 genes were up-regulated, and 342 genes were down-regulated ( T2, type 2; KEGG, Kyoto Encyclopedia of Genes and Genomes; NET, neutrophil extracellular trap; NP, nasal polyp; EM, ethmoidal mucosa. Eighty up-regulated genes and 114 down-regulated genes were found between non-T2-NP and T2-NP, while 533 up-regulated genes and 99 down-regulated genes were found between non-T2-EM and T2-EM. The top 20 up- and down-regulated genes are listed in  The RNA-seq results showed prominent expressions of NET formation in non-T2 CRSwNP compared to T2 CRSwNP. Immunofluorescence staining with the NET markers, CitH3 and HNE, was performed to detect NETs. As expected, CitH3+HNE+ was highly increased in non-T2-NP vs. control and T2-NP. T2-NP showed higher CitH3+HNE+ levels than the control tissues ( T2, type 2; HNE, human neutrophil elastase; CitH3, citrullinated histone H3; NP, nasal polyp; NET, neutrophil extracellular trap; EM, ethmoidal mucosa; IL, interleukin. HNE, human neutrophil elastase; T2, type 2; IL, interleukin.\",\n          \"Results\\n\\n Flowchart of study participants. Abbreviations: CN = cognitively normal; MCI = mild cognitive impairment; pMCI = progressive MCI; sMCI = stable MCI. Descriptive statistics corresponding to the baseline visits for pMCI and sMCI cohorts, respectively, are shown in  Descriptive statistics of the pMCI and sMCI cohorts, respectively. All variables corresponded to the baseline visit. Range from [0,30], lower scores indicating more impairment. Range from [\\u22123,3] with lower values indicating more impairment. Range from [\\u22123,1] with lower values indicating more impairment. Abbreviations: MMSE = Mini-Mental State Examination; SD = standard deviation. HiMAL outperformed all baseline methods including traditional ML classifiers, single task and unimodal baselines in terms of both AUROC and AUPRC ( Performance on the main task (AD progression prediction) in terms of AUROC (higher = better) and AUPRC (higher = better) for HiMAL and the baselines. Imaging only Cognition only Clinical only Imaging only Cognition only Clinical only The numbers in [] indicate 95% confidence intervals using the pivot bootstrap estimator by sampling participants from the test dataset with replacement 200 times. The  Significant  0.01\\u2009<\\u2009 0.005\\u2009<\\u2009 \\n Abbreviations: MMMT = multimodal multitask; MMST = multimodal single task; RF = Random Forest; SMMT = single modality multitask; SMST = Single-modality single task; SVM = Support Vector Machines. We observed similar trends of HiMAL performing better (lower MSE) than the unimodal and single-task baselines for the auxiliary task regression ( Performance on the auxiliary task regression (memory, executive functioning, language, and visual-spatial) in terms of MSE (lower = better) for HiMAL and the baselines. Imaging only Cognition only Clinical only Imaging only Cognition only Clinical only The numbers in [] indicate 95% confidence intervals using the pivot bootstrap estimator by sampling participants from the test dataset with replacement 200 times.\\u00a0Bold values indicate the model with the best performance (lowest MSE) for each cognitive function Abbreviations: MMMT = multimodal multitask; MMST = multimodal single task; SMMT = single modality multitask; SMST = single-modality single task.\\u00a0 The forecasted composite scores and the corresponding relevance scores at each timepoint provided insights (explanations) about how the contributions of different composites change over time from baseline visit to the onset of progression. Here we visualized the longitudinal explanations (every 6 months) for a particular pMCI individual who progressed 48 months after the first baseline visit ( Model explanations of an pMCI individual who progressed 48 months (4 years) after the baseline visit. The topmost plot represents progression risk while the following points represent the composite functions. The  Using all 3 modalities (imaging + cognition + clinical) as input resulted in the best prediction performance of HiMAL for both AD progression prediction (AUROC, AUPRC) and composite cognitive score regression (MSE) ( Performance of HiMAL for different combinations of input modalities in terms of (A) AUROC, (B) AUPRC, (C) MSE (memory), (D) MSE (executive), (E) MSE (language), and (F) visual-spatial. The red bars represent the 95% confidence interval using the pivot bootstrap estimator by sampling participants from the test dataset with replacement 200 times.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"conclusion\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 453,\n        \"samples\": [\n          \"10. Conclusions\\nIn conclusion, the oral microbiome is a diverse and complex ecosystem that plays a significant role in maintaining oral and systemic health. Comprising a plethora of bacterial, fungal, and archaeal species, the oral microbiome\\u2019s composition and dynamics are intricately linked to various physiological processes. Disruptions in this delicate balance, leading to oral dysbiosis, can precipitate a spectrum of oral ailments like dental caries, gingivitis, periodontitis, halitosis, and oral candidiasis, while also exerting profound effects on systemic health, including gastrointestinal disorders, cardiovascular diseases, endocrine imbalances, and neurological conditions. The interplay between the oral and gut microbiomes, underscored by microbial translocation, emphasizes the bidirectional relationship between oral and systemic health, wherein disturbances in either ecosystem can cascade into the other, exacerbating disease states. Recent research illuminating connections between specific oral bacteria, such as  As our understanding of the oral microbiome expands, it becomes increasingly evident that fostering a balanced oral microbial ecosystem is paramount for holistic health. Embracing evidence-based dental care practices, such as integrating dental probiotics like \",\n          \"Conclusions\\nThe lack of screening tests, albeit for subjects at risk due to previous asbestos exposure, together with the generally late diagnosis due to the absence of specific symptoms, makes Malignant pleural mesothelioma a tumor with a poor prognosis. For this reason, AI approaches for a better definition of prognostic models assume an important role in modern medicine in general, even more so for rare pathologies such as the one object of this work. Our proposed approach is particularly valuable as it aims to provide these insights without the need for invasive procedures or incurring additional costs beyond those typically associated with standard clinical practice. In addition, our model could represent a part of a multimodal strategy by including it in a clinical workflow not only to improve the prognostic assessment of patients with MPM but also to identify subgroups of patients with different responsiveness to treatments. A further qualifying element of this approach is represented by the possibility of using AI techniques based on CT scans and images normally used in clinical practice, therefore not requiring invasive or additional tests for patients. Although further validation on a larger sample population is needed, our study supports the hypothesis that AI techniques, by evaluating radiomics and other parameters (clinical, pathological, and genomic data), could generate algorithms potentially useful to predict patient outcomes. Such a global approach deserves further investigation given that innovative treatments are being studied and are about to become more widely available in MPM; in this scenario, the choice of the treatment strategy should consider the costs of new drugs and the need to avoid unnecessary toxicities, thus preserving the quality of life of unresponsive patients and finally leaning towards the goal of precision medicine in patients with MPM as well.\",\n          \"7. Conclusions\\nThe integration of multi-omics data with ML offers a powerful approach to delve into a better understanding of the molecular hallmarks of aging. Advances in these technologies can have a significant impact on aging research. This approach has the potential to identify critical biomarkers and therapeutic targets, paving the way for personalized interventions aimed at extending both health span and lifespan. The future of aging research lies in the seamless integration of these technologies, supported by collaborative efforts in data sharing and open science. Such advancements will be essential in overcoming current challenges and translating research findings into clinical practice. Ultimately, by leveraging the power of multi-omics data and ML, we can move closer to a future where healthy aging is achievable for all.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"fulltext\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1000,\n        \"samples\": [\n          \"Introduction\\nAlzheimer\\u2019s disease (AD) is a progressive neurodegenerative disorder that is associated with hippocampal atrophy.\\nAmyloid and tau deposition, axonal damage and astrocyte activation are integral to neurodegenerative cerebral atrophy.\\nThe hippocampus is one of the first brain structures affected with atrophy in AD.\\nOverall, most AD biomarker studies have primarily been conducted with Western populations samples made up of predominantly White individuals of European ancestry. However, non-White populations may differ from White populations in terms of cognitive reserve, cultural background, lifestyle, and genetics. Specifically, cognitive testing is not well standardized for sub-Saharan populations and costly AD biomarkers, such as CSF or PET, are little available. Thus, understanding the role of plasma AD biomarkers within non-White populations is crucial for generalizability of AD clinical research, especially in relation to emerging diagnostic and therapeutic tools. There are few studies on the association of core AD biomarkers (A\\u03b2 and p-tau) and hippocampal atrophy in culturally diverse populations. For example, in an enthnoracially diverse sample of community dwelling older adults, hippocampal volume was found to be a significant mediator between A\\u03b242/40 and NfL on baseline episodic memory and executive function measures.\\nThe current study aims to investigate ability of AD core plasma biomarkers (A\\u03b2 42/40, p-tau181) and non-specific AD biomarkers (NfL and GFAP) to discriminate the severity of hippocampal atrophy in adult individuals in Kinshasa, Democratic Republic of the Congo (DRC), in Sub-Saharan Africa (SSA). Although comparing left versus right hippocampal atrophy is not the central focus of this study, we will consider a lateralization approach. Our main goal is to describe the clinical value of these biomarkers in this novel population. This opens the door to more in-depth explorations of their clinical value and the examination of the left-sided vulnerability theory in dementia patients within this African population. We hypothesized that core (A\\u03b242/40 and p-tau181) and non-specific (NfL and GFAP) plasma biomarkers of AD will be associated with hippocampal atrophy in this sample. Given the pathophysiological impact and the usefulness of p-tau181 in the diagnosis of AD pathology,\\nMETHODS\\nStudy population\\nParticipants of this study are community-dwellers from Kinshasa, DRC, selected from our previous study.\\nProcedure\\nParticipants underwent a comprehensive clinical evaluation, which included cognitive testing, self-report questionnaires, and standard psychiatric and neurological assessments. An expert panel consisting of a neurologist, psychiatrist, and neuropsychologist diagnosed participants with dementia or classified them as healthy controls (HC). Subjects were interviewed to gather demographic, socioeconomic, and medical history information. Subsequently, blood samples were collected by a phlebotomist at the Medical Center of Kinshasa (CMK). The sample collection protocol and quantification of fluid biomarkers are detailed below.\\nMeasures\\nPlasma biomarkers\\nBlood samples were drawn in the CMK blood laboratory by venipuncture into dipotassium ethylene diamine tetra acetic acid (K\\nPlasma biomarker concentrations were measured using commercially available Neurology 4-PLEX E (A\\u03b240, A\\u03b242), P-tau181 (P-tau181 v2; l) Quanterix kits for the Simoa HD-X platform (Billerica, MA) at the University of San Francisco. The instrument operator was blinded to clinical variables. All analytes were measured in duplicate. For A\\u03b240 and A\\u03b242, all samples were measured above the lower limit of quantification (LLOQ) of 1.02 pg/mL and 0.378 pg/mL. The average coefficient of variation (CV) for A\\u03b240 and A\\u03b242 were 6.0% and 6.5%. For P-tau181, all samples were measured above the kit lower limit of quantification (LLOQ) of 0.085 pg/mL, with an average CV of 11.6%.\\nNeuroimaging\\nAll subjects were imaged on a 1.5 Tesla MRI unit (Siemens, Magneton Sonata) scanner at HJ Hospitals in Kinshasa using the same standardized imaging acquisition protocol based on the Alzheimer\\u2019s Disease Research Center protocol of Emory University. This consisted of sagittal volumetric T1-weighted (MPRAGE), coronal T2-weighted, and axial diffusion-weighted, T2-weighted, and T2-FLAIR sequences. Typical acquisition parameters for the MPRAGE sequence were TR = 2200 ms, minimum full TE, TI = 1000 ms, flip angle = 8\\u00b0, FOV = 25 cm, with a 192 \\u00d7 184 acquisition matrix, yielding a voxel size of approximately 1.25 \\u00d7 1.25 \\u00d7 1.2 mm.\\nQuantitative volumetric analysis using Freesurfer\\nThe 3D T1w images were segmented using Freesurfer (v.6, MGH, MA), which includes a full processing stream for MR imaging data that involves skull-stripping, bias field correction, registration, and anatomical segmentation as well as cortical surface reconstruction, registration, and parcellation. Regional brain volume for both cortical and subcortical brain regions were calculated. The left and right hippocampal (LH, RH) volume was averaged. Interindividual variation in head size were accounted for in further statistical analysis by controlling for total intracranial volume.\\nStatistical Analyses\\nStatistical analyses were performed using SAS and R Version 4 statistical softwares. Descriptive statistics for continuous, normally distributed variables are presented as mean \\u00b1 standard deviation (SD), while categorical variables are expressed using counts and proportions. Winsorization of plasma biomarkers to the 95th percentile was applied to limit the effect of extreme outliers. Biomarkers were standardized by subtracting the mean biomarker value of the sample and dividing by the sample\\u2019s plasma biomarker standard deviation. Standardized hippocampal volumes and biomarkers were obtained for the left hippocampus (LH), right hippocampus (RH), and total hippocampal (TH) volume. We calculated the differences in hippocampal volumes based on cognitive status and defined hippocampal atrophy using established cutoffs:\\n\\u2265 3000 mm\\n\\u2265 6000 mm\\nGeneral linear regression, adjusting for age, sex, and education, was used to evaluate differences in biomarkers by neurological status. Logistic regression was conducted with left, right, or total hippocampal volume <3000 or <6000 mm\\nRESULTS\\nDemographic data, cognitive screening scores, hippocampal volumes, and plasma biomarker characteristics stratified by neurological status are presented in \\nDiscussion\\nIn the current study, we examined the discriminative ability of core AD plasma biomarkers (A\\u03b242/40, p-tau181) and non-specific AD biomarkers (NfL and GFAP) in predicting hippocampal atrophy in adult Congolese with and without probable AD from Kinshasa, DRC. This is one of the first exploratory studies to use AD plasma biomarkers based on Alzheimer\\u2019s Association criteria\\nIn our previous study, we did not find significant relationships between mood severity and hippocampal volume. Despite the lack of significant association with hippocampal volume, we used mood severity as a covariate to be conservative in our analyses. We found elevated levels of non-specific AD biomarkers (NfL and GFAP) in our patients with probable AD, which is consistent with reports that NfL and GFAP are involved in other neurodegenerative pathology.\\nOur first hypothesis, which predicted that core (A\\u03b242/40 and p-tau181/217) and non-specific (NfL and GFAP) plasma biomarkers of AD would be associated with hippocampal atrophy, was partially supported. We found significant associations between p-tau 181 plasma biomarkers and lower LH volumes. However, core biomarker (A\\u03b242/40) and non-specific (NfL and GFAP) AD plasma biomarkers were not associated with hippocampal atrophy in this sample. These findings are similar to studies in other populations, which have found associations between AD core plasma biomarkers (p-tau 181) and hippocampal atrophy in elderly subjects with dementia.\\nThese findings also highlight the importance of core AD plasma biomarkers in the evaluation of adults with and without AD, making p-tau181 better plasma AD biomarkers in the diagnostic assessment of cognitive aging. Our results also support the utility of using these two plasma biomarkers (A\\u03b242/40 and p-tau181) to discriminate hippocampal atrophy in adults with and without AD in SSA populations. These core AD plasma biomarkers had greater discriminative ability for hippocampal atrophy than non-specific AD biomarkers (NfL and GFAP). These results demonstrated good specificity of plasma biomarkers in discriminating hippocampal atrophy in both crude and adjusted models. However, the sensitivity of plasma biomarkers was still poor in discriminating hippocampal atrophy.\\nWhile lateralized hippocampal findings are frequently observed in the research literature, our study did find some evidence of laterality regarding the ability of plasma biomarkers to discriminate hippocampal atrophy. The ability to discriminate atrophy was slightly better for LH than RH. The discriminative strength remained almost the same when intracranial volume was added as a covariate and after adjustment for other covariates. These results can be explained by evidence suggesting that networks in the left (i.e., dominant) hemisphere may be more vulnerable to neurodegeneration.\\nThis study is the first in SSA to attempt to discriminate hippocampal atrophy based on AD core and non-specific AD plasma protein biomarkers in a novel sample of adults in the DRC with and without dementia. Our findings should be interpreted considering several limitations, such as the cross-sectional nature of the study, low sensitivity, and lack of amyloid PET imaging confirming AD pathology. These analyses should be further validated in longitudinal data. Future studies should follow a cohort to make longitudinal predictions of the rate of hippocampal atrophy based on baseline plasma biomarkers. Another limitation includes a small sample of participants, which limited the detection of differences that could have been clinically and significantly relevant to find adequate discriminative strength of the plasma biomarkers. Thus, future studies should replicate these findings with larger sample sizes. Fourth, the screening measures used (CSID and AQ) have not been validated in SSA in general and the DRC in particular. Fifth, this study may have included cases other than amnestic multidomain dementia, increasing the chances of enrolling patients with dementia caused by conditions other than AD, and thus not showing significant hippocampal atrophy. Future studies should conduct statistical analyses across all four groups (healthy controls, MCI, subjective memory complaints, and dementia). Sixth, only one brain region was analyzed. Associations with other key regions might have been identified using more powerful imaging techniques such as Voxel-Based Morphometry (VBM). Furthermore, future studies should also aim to replicate our findings using amyloid and tau brain PET or mass spectrometry to measure biomarkers. A major caveat is that our AD biomarkers were determined by Simoa, which is not optimal. The gold standard for core AD biomarker assessment is amyloid and tau brain PET. Head-to-head comparisons of amyloid and tau biomarkers have demonstrated the superiority of IPMS techniques over Simoa. Thus, continued investigation into racial disparities in AD biomarkers and their relation to AD-dementia using these gold standard techniques (e.g., brain amyloid PET, CSF) is necessary.\\nConclusions\\nUnderstanding the ability of AD core plasma biomarkers (A\\u03b2\\u221242/40 and p-tau 181) and non-specific plasma biomarkers (NfL and GFAP) to discriminate hippocampal atrophy in adults is a promising next step in clinical and research settings. While blood biomarkers are not equivalent to an AD diagnosis, they can be utilized as a screening tool before resorting to PET-scan neuroimaging or CSF biomarker analysis. Future longitudinal studies should test AD-related blood and CSF biomarkers from the same individuals for better discrimination of hippocampal atrophy. Additionally, larger studies with greater sample sizes and diversity in races and ethnicities should be conducted to increase generalizability.\",\n          \"1 Introduction\\nGlyphosate (GLY), or N-(phosphonomethyl) glycine, is an organophosphorus compound used as a broad-spectrum non-selective herbicide, considered the most used worldwide (\\nThe shikimate pathway in plants. Image has been created with \\n2 Glyphosate authorization overview\\nGLY was first authorised in 1974 in U.S. following the approval of EPA to place GLY on the market as an herbicide (\\nRegulatory timeline of glyphosate. Image has been created with \\n3 Glyphosate mode of action and its physical and chemical properties\\nGLY is a white, odourless, and crystalline solid classified as an organophosphorus compound. Its simple chemical structure and its remarkable stability under a variety of reaction conditions (e.g., pH, temperature, etc.) has allowed it to be manufactured in different synthesis routes. A frequently adopted synthetic pathways is the one based on the multi-step reaction between glycine, formaldehyde and DMP (dimethyl phosphite) leading to the product GLY with high yields, via several non-isolatable intermediates (\\nChemical and physical properties of glyphosate. Image has been created with \\n4 Glyphosate and human exposure routes\\nGLY is able to enter human bodies through different exposure routes, among which the most notable include: dermal absorption, inhalation, ingestion as well as intake of GLY-contaminated foodstuffs (\\nScheme of the main exposure and contamination routes of glyphosate\\u2019s and glyphosate-based herbicide\\u2019s (GBHs) in humans. Image has been created with \\n4.1 Effects of glyphosate exposure on liver, kidney and lungs\\nOxidative stress is a phenomenon which causes an alteration in the equilibrium between production of reactive oxygen species (ROS) and organism\\u2019s activated anti-oxidant systems. This imbalance may lead to several issues, including damage of lipids and proteins, activation of apoptotic pathways and/or the initiation of inflammatory phenomena in both tissues and cells (\\n4.2 Glyphosate exposure a possible cause of gut disease and microbiotic disruption\\nGLY can be regarded as a significant environmental factor in the development of diseases associated with dysbiosis of the gut microbiota, like inflammatory bowel disease (IBD) (\\n4.3 Glyphosate and its possible carcinogenic and/or mutagenic effects\\nBiotransformation processes undergone by xenobiotics, including GLY, in some cases lead to ROS production, which is liable to cause DNA damage. Such injury has the potential to induce mutations and out-of-control cell proliferation, resulting occasionally in the development of cancer. In recent years, GLY\\u2019s potential genotoxic effects became a topic of intense debate, both inside scientific community as well as amongst the various international agencies (\\n4.4 Estrogenic effects of glyphosate and glyphosate-based herbicides\\nSeveral studies have demonstrated how both GLYs and GBHs can be regarded as endocrine disruptors (EDs), as they are able to induce estrogenic or androgenic nuclear receptors-mediated effects (\\n5 Conclusion\\nThe exposure to pesticides is well known to cause serious damage to both environment and human health. Several scientific studies demonstrated how being exposed to GLY and GBHs, may lead to the occurrence of various conditions in humans, like the onset of inflammatory diseases, neurological disorders and reproductive disruptions (\",\n          \"Highlights\\niPLA\\nT1D is a T-cell\\u2013mediated disease; however, the contribution of T-cell\\u2013derived lipid signaling to T1D development has not been studied.\\nHerein, we sought to determine if iDL signaling in CD4\\nWe find that reducing iDL signaling in CD4\\nWe posit that targeting select iDL signaling in T cells is a novel therapeutic avenue to counteract T1D onset.\\nIntroduction\\nType 1 diabetes (T1D) is a consequence of autoimmune destruction of pancreatic islet \\u03b2-cells, involving activation of cellular immunity and inflammation initiated by early stage immune cell infiltration of islets (\\nAn integral component of T1D progression is an overall increase in the inflammatory status of T cells. It is well recognized that some lipid metabolites of arachidonic acid are profoundly inflammatory (\\nProinflammatory iDLs have been reported to induce NF\\u03baB (\\nTo date, the potential involvement of lipid signaling from CD4\\nMethods\\nAnimal generation and monitoring\\nWild-type NOD.\\nSplenocyte adoptive transfer\\nSpleens were excised from pre-diabetic 12-week-old \\nmRNA analyses\\nCells were centrifuged and lysed in 500 \\u03bcL of Trizol (Invitrogen, #15596-026, Carlsbad, CA). Total RNA was prepared and purified using a GeneJET RNA purification kit (Thermo Fisher Scientific, #K0732, Waltham, MA). The RNA was converted to cDNA using the iScript cDNA Synthesis Kit (Bio-Rad, #1708891, Hercules, CA), and the cDNA transcripts were amplified, as described (\\nT-cell adoptive transfer\\nSplenocytes were prepared from pre-diabetic 12-week-old mice, as above, and StemCell EasySep Mouse CD4 positive (catalog no. 18952) and StemCell EasySep Mouse CD8 negative (catalog no. 18952) selection kits were used to prepare separated populations of CD4\\nGlucose tolerance\\nIntraperitoneal glucose tolerance tests (IPGTTs) were performed, as described (\\nInsulitis, islet immunostaining, and microscopy\\nParaffin sections (6 \\u03bcm) of pancreata were prepared and stained with hematoxylin-eosin (H/E) for histological assessment of islet infiltration. Sections containing islets were then processed for immunostaining using an antigen retrieval protocol, as previously described (\\n\\n\\nT1D is recognized to be a Th1-driven disease with IFN\\u03b3 production correlating with diabetes progression in NOD mice (\\n\\n\\nThe CD4\\nSurface and intracellular staining\\nCell surface staining was performed, separately, on single-cell suspensions of CD4\\nAnalysis of T-cell eicosanoids by mass spectrometry\\nThe CD4\\nThe eicosanoids were separated using a Shimadzu Nexera X2 LC-30AD coupled to a SIL-30AC auto injector, coupled to a DGU-20A5R degassing unit in the following way: A 14\\u00a0min, reversed phase LC method utilizing an Acentis Express C18 column (150\\u00a0mm \\u00d7 2.1\\u00a0mm, 2.7 \\u00b5m) was used to separate the eicosanoids at a 0.5 ml/min flow rate at 40\\u00b0C. The column was equilibrated with 100% Solvent A [acetonitrile:water:formic acid (20:80:0.02, v/v/v)] for 5\\u00a0min and then 10 \\u00b5L of sample was injected. One hundred percent Solvent A was used for the first 2\\u00a0min of elution. Solvent B (acetonitrile:isopropanol:formic acid [20:80:0.02, v/v/v]) was increased in a linear gradient to 25% Solvent B at 3\\u00a0min, to 30% at 6\\u00a0min, to 55% at 6.1\\u00a0min, to 70% at 10\\u00a0min, to 100% at 10.10\\u00a0min, 100% Solvent B was held constant until 13.0\\u00a0min, where it was decreased to 0% Solvent B and 100% Solvent A from 13.0\\u00a0min to 13.1\\u00a0min. From 13.1\\u00a0min to 14.0\\u00a0min, Solvent A was held constant at 100%.\\nThe eicosanoids were analyzed via MS using an AB Sciex Triple Quad 5500 Mass Spectrometer. Q1 and Q3 were set to detect distinctive precursor and product ion pairs. Ions were fragmented in Q2 using N2 gas for collisionally-induced dissociation. Analysis used multiple-reaction monitoring in negative-ion mode. Eicosanoids were monitored using precursor \\u2192 product MRM pairs. The MS parameters used were Curtain Gas: 20\\u00a0psi; CAD: Medium; Ion Spray Voltage: \\u22124500 V; Temperature: 300\\u00b0C; Gas 1: 40\\u00a0psi; Gas 2: 60\\u00a0psi; Declustering Potential, Collision Energy, and Cell Exit Potential vary per transition.\\nStatistical analyses\\nSignificant differences in T1D incidence were determined by the Mantel-Cox test. For all other analyses, \\u201cp\\u201d values were determined using the Student\\u2019s t-test (for analyses between two groups) or one-way analysis of variance (ANOVA) followed by Tukey\\u2019s multiple comparisons (for analyses between multiple groups). For lipidomics analyses, \\u201cp\\u201d values were determined by two-way ANOVA, followed by Dunnett\\u2019s multiple comparisons test. All statistical analyses were performed using GraphPad Prism (version 9) and a \\u201cp\\u201d value less than 0.05 was defined as significant.\\nResults\\nSplenocyte iPLA\\nWe previously noted an intriguing direct relationship between islet \\nSplenocytes adoptive transfer. \\nTransfer of splenocytes with reduced iPLA\\nImmunodeficient and spontaneous diabetes-resistant NOD.\\nWeekly blood glucose measurements revealed a rapid T1D onset within 7 weeks post-transfer and 50% incidence by 8 weeks post-transfer in mice administered \\nReduced insulitis and \\u03b2-cell preservation following transfer of splenocytes with mitigated iDL signaling\\nIn view of the reduced diabetes incidence in the recipients of NOD.\\nIslet infiltration and insulin staining. NOD mice were treated as described in \\nTo determine if the reduced diabetes outcomes were due to differences in temporal abundances of splenic T cells, we performed flow analyses. We found that the frequencies of CD4\\nSplenocyte Immune Cell Composition. Age-matched \\nThese findings suggest that the reduced T1D incidence with decreased splenocyte-iDL signaling is associated with mitigated insulitis and preservation of \\u03b2-cells and is independent of differences in splenic CD4\\nTransfer of T cells with reduced iPLA\\nAs mouse splenocytes are enriched in CD4\\nT-cell adoptive transfer. \\nIn initial studies, NOD.\\nTo assess the specific roles of CD4\\nReduced insulitis and \\u03b2-cell preservation in NOD.\\nWe next assessed the impact of transferring CD4\\nIslet infiltration and insulin staining. Mice were treated as described in \\nImpact of T-cell function with inhibition of select lipid signaling\\nAs the above observations suggested participation of CD4\\n\\n\\nIn contrast to the CD4\\n\\n\\nDiscussion\\nCytokine-mediated \\u03b2-cell death occurs in an iPLA\\nTo gain an insight into the potential role of T-cell\\u2013iDL signaling, splenocytes prepared from pre-diabetic \\nIn NOD mice with a selective modification of iPLA\\nThe iPLA\\nAs such, we next addressed if the reduction in T1D incidence was accompanied by alterations in the production of iDLs by CD4\\nIn contrast to the CD4\\niDL signaling is associated with \\u03b2-cell apoptosis (\\nIn summary, the current study offers forward-going insights into the participation of CD4\\nProposed model of CD4\\nThe findings here complement our previous report that identified a similar lipid signature in normoglycemic children that were at high-risk for developing diabetes (\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
              "type": "dataframe",
              "variable_name": "df"
            },
            "text/html": [
              "\n",
              "  <div id=\"df-9f5735cc-11ce-4418-a6b0-332100fafc38\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmid</th>\n",
              "      <th>pmcid</th>\n",
              "      <th>title</th>\n",
              "      <th>abstract</th>\n",
              "      <th>authors</th>\n",
              "      <th>methods_materials</th>\n",
              "      <th>results</th>\n",
              "      <th>conclusion</th>\n",
              "      <th>fulltext</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>39377039</td>\n",
              "      <td>11458320</td>\n",
              "      <td>Advancements and Innovative Strategies in Indu...</td>\n",
              "      <td>The effectiveness and safety of mesenchymal st...</td>\n",
              "      <td>Shi Xiaoyu, Zhang Kun, Yu Fengshi, Qi Qi, Cai ...</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>1. Introduction\\nRegenerative medicine and cel...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>39374157</td>\n",
              "      <td>11457879</td>\n",
              "      <td>Neuroprotective effects of punicalagin and/or ...</td>\n",
              "      <td>Abstract Background Manganism, a central nervo...</td>\n",
              "      <td>Abu‐Elfotuh Karema, Abbas Ashwaq N., Najm Mazi...</td>\n",
              "      <td>Supporting information\\n\\nData S1.\\n</td>\n",
              "      <td>RESULTS\\nIn this work, three separate normal c...</td>\n",
              "      <td>CONCLUSIONS\\nIn conclusion, as shown in the gr...</td>\n",
              "      <td>INTRODUCTION\\nManganese is a vital trace eleme...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>38869924</td>\n",
              "      <td>11457624</td>\n",
              "      <td>Impact of a keto diet on symptoms of Parkinson...</td>\n",
              "      <td>Aim:  Evidence suggests low-carbohydrate diets...</td>\n",
              "      <td>Tidman  Melanie M, White Dawn Reid, White Tim A</td>\n",
              "      <td>Supplementary Material\\n</td>\n",
              "      <td>Results\\nFasting blood work to assess common b...</td>\n",
              "      <td>Conclusion\\nIn this 24-week dietary interventi...</td>\n",
              "      <td>Background\\nParkinson's disease (PD) is the nu...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>39363536</td>\n",
              "      <td>11457610</td>\n",
              "      <td>Role of RNA polymerase III transcription and r...</td>\n",
              "      <td>ABSTRACT Ischaemic stroke is a leading cause o...</td>\n",
              "      <td>Tsang Chi Kwan, Zheng X.F. Steven</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nStroke, defined as prolonged acu...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>39308280</td>\n",
              "      <td>11457211</td>\n",
              "      <td>Investigating patient eligibility for anti-amy...</td>\n",
              "      <td>Background Pharmacological treatment options f...</td>\n",
              "      <td>Defrancesco Michaela, Gizewski Elke R., Manges...</td>\n",
              "      <td>Method\\nThis was a retrospective, observationa...</td>\n",
              "      <td>Results\\nA total of 587 out-patients attended ...</td>\n",
              "      <td>None</td>\n",
              "      <td>Since the approval of donepezil in 1997, pharm...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>995</th>\n",
              "      <td>39328348</td>\n",
              "      <td>11424528</td>\n",
              "      <td>Optimal dose and type of exercise to improve c...</td>\n",
              "      <td>Background Mild cognitive impairment (MCI) rep...</td>\n",
              "      <td>Yu Yingying, Wang Junjie, Xu Jian</td>\n",
              "      <td>None</td>\n",
              "      <td>Results\\nAs a result of the search strategy, 8...</td>\n",
              "      <td>Conclusion\\nThis network meta-analysis has sho...</td>\n",
              "      <td>Introduction\\nElderly people with MCI\\nAs the ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>996</th>\n",
              "      <td>39328407</td>\n",
              "      <td>11424418</td>\n",
              "      <td>Identification of cellular senescence-related ...</td>\n",
              "      <td>Background Intervertebral disc degeneration (I...</td>\n",
              "      <td>Wang Muyi, Wang Hao, Wang Xin, Shen Yifei, Zho...</td>\n",
              "      <td>Materials and methods\\nThe transcriptome profi...</td>\n",
              "      <td>Results\\nAfter conducting PCA analysis on the ...</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nLow back pain (LBP), a prevalent...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>997</th>\n",
              "      <td>39323550</td>\n",
              "      <td>11423858</td>\n",
              "      <td>The roles of TAF1 in neuroscience and beyond</td>\n",
              "      <td>The transcriptional machinery is essential for...</td>\n",
              "      <td>Crombie Elisa M., Cleverley Karen, Timmers H. ...</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nTATA box binding-protein (TBP)-a...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>998</th>\n",
              "      <td>39323903</td>\n",
              "      <td>11423842</td>\n",
              "      <td>Phase-Amplitude Coupling in Theta and Beta Ban...</td>\n",
              "      <td>Background Phase-amplitude coupling (PAC) betw...</td>\n",
              "      <td>Zhang Chan, Wang Yanhui, Li Mengjie, Niu Pengp...</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nObstructive sleep apnea (OSA) a ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>999</th>\n",
              "      <td>39323935</td>\n",
              "      <td>11423824</td>\n",
              "      <td>Circulating Cell-Free Mitochondrial DNA in Pla...</td>\n",
              "      <td>Purpose Cognitive domains are affected in pati...</td>\n",
              "      <td>Garcia-de la Cruz Dulce Dajheanne, Juarez-Rojo...</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nMitochondria are double-membrane...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>1000 rows × 9 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-9f5735cc-11ce-4418-a6b0-332100fafc38')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-9f5735cc-11ce-4418-a6b0-332100fafc38 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-9f5735cc-11ce-4418-a6b0-332100fafc38');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-347e8d8c-e1eb-446c-924c-ff16705aa462\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-347e8d8c-e1eb-446c-924c-ff16705aa462')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-347e8d8c-e1eb-446c-924c-ff16705aa462 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "  <div id=\"id_cdb955cf-0dbe-41d8-8948-d1467322e2a1\">\n",
              "    <style>\n",
              "      .colab-df-generate {\n",
              "        background-color: #E8F0FE;\n",
              "        border: none;\n",
              "        border-radius: 50%;\n",
              "        cursor: pointer;\n",
              "        display: none;\n",
              "        fill: #1967D2;\n",
              "        height: 32px;\n",
              "        padding: 0 0 0 0;\n",
              "        width: 32px;\n",
              "      }\n",
              "\n",
              "      .colab-df-generate:hover {\n",
              "        background-color: #E2EBFA;\n",
              "        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "        fill: #174EA6;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate {\n",
              "        background-color: #3B4455;\n",
              "        fill: #D2E3FC;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate:hover {\n",
              "        background-color: #434B5C;\n",
              "        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "        fill: #FFFFFF;\n",
              "      }\n",
              "    </style>\n",
              "    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('df')\"\n",
              "            title=\"Generate code using this dataframe.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "    <script>\n",
              "      (() => {\n",
              "      const buttonEl =\n",
              "        document.querySelector('#id_cdb955cf-0dbe-41d8-8948-d1467322e2a1 button.colab-df-generate');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      buttonEl.onclick = () => {\n",
              "        google.colab.notebook.generateWithVariable('df');\n",
              "      }\n",
              "      })();\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "text/plain": [
              "         pmid     pmcid                                              title  \\\n",
              "0    39377039  11458320  Advancements and Innovative Strategies in Indu...   \n",
              "1    39374157  11457879  Neuroprotective effects of punicalagin and/or ...   \n",
              "2    38869924  11457624  Impact of a keto diet on symptoms of Parkinson...   \n",
              "3    39363536  11457610  Role of RNA polymerase III transcription and r...   \n",
              "4    39308280  11457211  Investigating patient eligibility for anti-amy...   \n",
              "..        ...       ...                                                ...   \n",
              "995  39328348  11424528  Optimal dose and type of exercise to improve c...   \n",
              "996  39328407  11424418  Identification of cellular senescence-related ...   \n",
              "997  39323550  11423858       The roles of TAF1 in neuroscience and beyond   \n",
              "998  39323903  11423842  Phase-Amplitude Coupling in Theta and Beta Ban...   \n",
              "999  39323935  11423824  Circulating Cell-Free Mitochondrial DNA in Pla...   \n",
              "\n",
              "                                              abstract  \\\n",
              "0    The effectiveness and safety of mesenchymal st...   \n",
              "1    Abstract Background Manganism, a central nervo...   \n",
              "2    Aim:  Evidence suggests low-carbohydrate diets...   \n",
              "3    ABSTRACT Ischaemic stroke is a leading cause o...   \n",
              "4    Background Pharmacological treatment options f...   \n",
              "..                                                 ...   \n",
              "995  Background Mild cognitive impairment (MCI) rep...   \n",
              "996  Background Intervertebral disc degeneration (I...   \n",
              "997  The transcriptional machinery is essential for...   \n",
              "998  Background Phase-amplitude coupling (PAC) betw...   \n",
              "999  Purpose Cognitive domains are affected in pati...   \n",
              "\n",
              "                                               authors  \\\n",
              "0    Shi Xiaoyu, Zhang Kun, Yu Fengshi, Qi Qi, Cai ...   \n",
              "1    Abu‐Elfotuh Karema, Abbas Ashwaq N., Najm Mazi...   \n",
              "2      Tidman  Melanie M, White Dawn Reid, White Tim A   \n",
              "3                    Tsang Chi Kwan, Zheng X.F. Steven   \n",
              "4    Defrancesco Michaela, Gizewski Elke R., Manges...   \n",
              "..                                                 ...   \n",
              "995                  Yu Yingying, Wang Junjie, Xu Jian   \n",
              "996  Wang Muyi, Wang Hao, Wang Xin, Shen Yifei, Zho...   \n",
              "997  Crombie Elisa M., Cleverley Karen, Timmers H. ...   \n",
              "998  Zhang Chan, Wang Yanhui, Li Mengjie, Niu Pengp...   \n",
              "999  Garcia-de la Cruz Dulce Dajheanne, Juarez-Rojo...   \n",
              "\n",
              "                                     methods_materials  \\\n",
              "0                                                 None   \n",
              "1                 Supporting information\\n\\nData S1.\\n   \n",
              "2                             Supplementary Material\\n   \n",
              "3                                                 None   \n",
              "4    Method\\nThis was a retrospective, observationa...   \n",
              "..                                                 ...   \n",
              "995                                               None   \n",
              "996  Materials and methods\\nThe transcriptome profi...   \n",
              "997                                               None   \n",
              "998                                               None   \n",
              "999                                               None   \n",
              "\n",
              "                                               results  \\\n",
              "0                                                 None   \n",
              "1    RESULTS\\nIn this work, three separate normal c...   \n",
              "2    Results\\nFasting blood work to assess common b...   \n",
              "3                                                 None   \n",
              "4    Results\\nA total of 587 out-patients attended ...   \n",
              "..                                                 ...   \n",
              "995  Results\\nAs a result of the search strategy, 8...   \n",
              "996  Results\\nAfter conducting PCA analysis on the ...   \n",
              "997                                               None   \n",
              "998                                               None   \n",
              "999                                               None   \n",
              "\n",
              "                                            conclusion  \\\n",
              "0                                                 None   \n",
              "1    CONCLUSIONS\\nIn conclusion, as shown in the gr...   \n",
              "2    Conclusion\\nIn this 24-week dietary interventi...   \n",
              "3                                                 None   \n",
              "4                                                 None   \n",
              "..                                                 ...   \n",
              "995  Conclusion\\nThis network meta-analysis has sho...   \n",
              "996                                               None   \n",
              "997                                               None   \n",
              "998                                               None   \n",
              "999                                               None   \n",
              "\n",
              "                                              fulltext  \n",
              "0    1. Introduction\\nRegenerative medicine and cel...  \n",
              "1    INTRODUCTION\\nManganese is a vital trace eleme...  \n",
              "2    Background\\nParkinson's disease (PD) is the nu...  \n",
              "3    Introduction\\nStroke, defined as prolonged acu...  \n",
              "4    Since the approval of donepezil in 1997, pharm...  \n",
              "..                                                 ...  \n",
              "995  Introduction\\nElderly people with MCI\\nAs the ...  \n",
              "996  Introduction\\nLow back pain (LBP), a prevalent...  \n",
              "997  Introduction\\nTATA box binding-protein (TBP)-a...  \n",
              "998  Introduction\\nObstructive sleep apnea (OSA) a ...  \n",
              "999  Introduction\\nMitochondria are double-membrane...  \n",
              "\n",
              "[1000 rows x 9 columns]"
            ]
          },
          "execution_count": 281,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "df"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "e43UYkce6N-l"
      },
      "outputs": [],
      "source": [
        "df.to_csv(\"/content/alzheimer_biomarker_1000.csv\", index=False)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "rmqokO-r6cfv",
        "outputId": "acfaf66d-ec89-4f90-c735-2f7b0a10bbc8"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Initiate paper-ids fetch...\n",
            "Completed id fetch.\n",
            "Total IDs:  2000\n",
            "Start fetching papers. Total request count:  334\n",
            "Completed request\n",
            "Total article count:  334\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "100%|██████████| 334/334 [00:00<00:00, 790.40it/s]\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Starting merge, papers scraped: 334\n",
            "Start fetching papers. Total request count:  333\n",
            "Completed request\n",
            "Total article count:  333\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "100%|██████████| 333/333 [00:00<00:00, 520.43it/s]\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Starting merge, papers scraped: 333\n",
            "Start fetching papers. Total request count:  333\n",
            "Completed request\n",
            "Total article count:  333\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "100%|██████████| 333/333 [00:00<00:00, 490.17it/s]\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Starting merge, papers scraped: 333\n"
          ]
        }
      ],
      "source": [
        "# Search for papers\n",
        "query = \"Alzheimer's Disease Biomarkers\"\n",
        "id_list = search_pubmed(query, max_results=2000)\n",
        "chunks = np.array_split(id_list[1000:], 3)\n",
        "df = None\n",
        "\n",
        "# Fetch paper details\n",
        "for chunk in chunks:\n",
        "    papers = fetch_paper_details(chunk)\n",
        "    if len(chunk) != 0 and len(papers) == 0:\n",
        "        print(\"Retrying scrape\")\n",
        "        time.sleep(3)\n",
        "        papers = fetch_paper_details(chunk)\n",
        "    print(f\"Starting merge, papers scraped: {len(papers)}\")\n",
        "    df = pd.concat([df, pd.DataFrame(papers)])\n",
        "    time.sleep(3)\n",
        "df.reset_index(drop=True, inplace=True)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "b7UvOxqs7QZ9",
        "outputId": "62f0895e-c0e2-4ac2-d77c-b309379b6963"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 1000,\n  \"fields\": [\n    {\n      \"column\": \"pmid\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 997,\n        \"samples\": [\n          \"39209459\",\n          \"39149774\",\n          \"39279993\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"pmcid\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1000,\n        \"samples\": [\n          \"11348879\",\n          \"11366618\",\n          \"11363304\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 985,\n        \"samples\": [\n          \"Genetic architecture of epigenetic cortical clock age in brain tissue from older individuals: alterations in \",\n          \"Advancements in NMN biotherapy and research updates in the field of digestive system diseases\",\n          \"Abstracts Presented at the 2024 WIN Symposium in Partnership with Burjeel Holdings: March 1-2, 2024, Abu Dhabi, United Arab Emirates\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 983,\n        \"samples\": [\n          \"SUMMARY Fluorescent genetically encoded voltage indicators report transmembrane potentials of targeted cell-types. However, voltage-imaging instrumentation has lacked the sensitivity to track spontaneous or evoked high-frequency voltage oscillations in neural populations. Here we describe two complementary TEMPO voltage-sensing technologies that capture neural oscillations up to ~100 Hz. Fiber-optic TEMPO achieves ~10-fold greater sensitivity than prior photometry systems, allows hour-long recordings, and monitors two neuron-classes per fiber-optic probe in freely moving mice. With it, we uncovered cross-frequency-coupled theta- and gamma-range oscillations and characterized excitatory-inhibitory neural dynamics during hippocampal ripples and visual cortical processing. The TEMPO mesoscope images voltage activity in two cell-classes across a ~8-mm-wide field-of-view in head-fixed animals. In awake mice, it revealed sensory-evoked excitatory-inhibitory neural interactions and traveling gamma and 3\\u20137 Hz waves in the visual cortex, and previously unreported propagation directions for hippocampal theta and beta waves. These technologies have widespread applications probing diverse oscillations and neuron-type interactions in healthy and diseased brains.\",\n          \"A number of studies have reported that drug addiction is associated with microRNAs (miRNAs). However, the roles of plasma miRNAs in methamphetamine (METH) addicts have not been clearly explained. This study aimed to profile a panel of miRNAs as non-invasive predictive biomarkers and therapeutic targets for METH addiction. Differentially expressed miRNAs were derived from next-generation sequencing technology (NGS) and were validated by quantitative real-time PCR (RT-qPCR). The diagnostic value of specific altered miRNAs was evaluated by receiver operating characteristic (ROC) analysis and area under the curve (AUC). NGS results revealed that 63 miRNAs were significantly altered in the METH-exposed paradigm. The levels of hsa-miR-592, hsa-miR-9-3p, hsa-miR-206 and hsa-let-7b-3p were significantly elevated in the plasma of METH addicts. Hsa-miR-9-3p was a useful biomarker discriminating METH addicts from normal (AUC was 0.756). Importantly, combining detection of hsa-miR-592 and hsa-miR-9-3p achieved the highest AUC of 0.87, with a sensitivity and specificity of 82.7% and 78.9%, respectively. Target gene BDNF decreased significantly in METH addicts. Although METH addicts showed significant depressive symptoms, there was no correlation between the expression level of miR-592 and miR-9-3p and the degree of depression. Our findings suggested that hsa-miR-592, hsa-miR-9-3p, hsa-miR-206, and hsa-let-7b-3p may play a potential role in the pathology of METH addiction, and a combination of hsa-miR-592 and hsa-miR-9-3p could serve as potential peripheral biomarker and therapeutic target for METH addiction.\",\n          \"Background Ulcerative colitis (UC) is a common chronic disease associated with inflammation and oxidative stress. This study aimed to construct a long noncoding RNA (lncRNA)-microRNA (miRNA)-messenger RNA (mRNA) network based on bioinformatics analysis and to explore oxidative stress-related genes underlying the pathogenesis of UC. Methods The  GSE75214 ,  GSE48959 , and  GSE114603  datasets were downloaded from the Gene Expression Omnibus database. Following differentially expressed (DE) analysis, the regulatory relationships among these DERNAs were identified through miRDB, miRTarBase, and TargetScan; then, the lncRNA-miRNA-mRNA network was established. The Molecular Signatures Database (MSigDB) was used to search oxidative stress-related genes. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses were performed for functional annotation and enrichment analyses. Based on the drug gene interaction database DGIdb, drugs that interact with oxidative stress-associated genes were explored. A dextran sulfate sodium (DSS)-induced UC mouse model was used for experimental validation. Results A total of 30 DE-lncRNAs, 3 DE-miRNAs, and 19 DE-mRNAs were used to construct a lncRNA-miRNA-mRNA network. By comparing these 19 DE-mRNAs with oxidative stress-related genes in MSigDB, three oxidative stress-related genes ( CAV1, SLC7A11 , and  SLC7A5 ) were found in the 19 DEM sets, which were all negatively associated with miR-194. GO and KEGG analyses showed that  CAV1, SLC7A11 , and  SLC7A5  were associated with immune inflammation and steroid hormone synthesis. In animal experiments, the results showed that dexamethasone, a well-known glucocorticoid drug, could significantly decrease the expression of  CAV1, SLC7A11 , and  SLC7A5  as well as improve UC histology, restore antioxidant activities, inhibit inflammation, and decrease myeloperoxidase activity. Conclusion SLC7A5  was identified as a representative gene associated with glucocorticoid therapy resistance and thus may be a new therapeutic target for the treatment of UC in the clinic.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"authors\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 994,\n        \"samples\": [\n          \"Lepping Rebecca J., Hess Benjamin J., Taylor Jasmine M., Hanson-Abromeit Deanna, Williams Kristine N.\",\n          \"Sampani Konstantina, Ness Steven, Tuz-Zahra Fatima, Aytan Nurgul, Spurlock Elizabeth E., Alluri Sreevardhan, Chen Xuejing, Siegel Nicole H., Alosco Michael L., Xia Weiming, Tripodis Yorghos, Stein Thor D., Subramanian Manju L.\",\n          \"Liu Shufang, Zhang Chenwei, Zhang Yukai, Wu Zhifang, Wu Ping, Tian Shouyuan, Zhang Min, Lang Limin, Li Li, Wang Ruonan, Liu Haizhao, Zhang Jingfen, Mao Xiaolu, Li Sijin\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"methods_materials\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 332,\n        \"samples\": [\n          \"Supplementary Information\\n\\n Supplementary Material 1\",\n          \"Methods\\nThis study (clinicaltrials.gov identifier NCT03233646) was approved by the Duke Health Institutional Review Board (Pro00082598). The study followed the Health Insurance Portability and Accountability Act of 1996 and adhered to all tenets of the Declaration of Helsinki. Written informed consent was provided by all study subjects or their legally authorized representative prior to study participation. Study participant images were collected from 2017 to 2023 from the existing Eye Multimodal Imaging in Neurodegenerative Disease (iMIND) database. Prior to enrollment, all patients with PD were evaluated by an experienced movement disorders specialist (BLS), and the clinical diagnosis of PD was confirmed according to the clinical criteria of the International Parkinson and Movement Disorder Society. Non-mydriatic images were acquired by experienced iMIND study personnel using the ZEISS CIRRUS 5000 Spectral-Domain OCT with AngioPlex software (version 11.0.0.29946; Carl Zeiss Meditec, Dublin, CA), and layer segmentation was done using the integrated ZEISS software. Active retinal motion tracking software was used to reduce motion artifact during acquisition. Images were each reviewed manually by trained study personnel (AK, DSG) for quality assessment. Images with lower than 7/10 signal strength index, motion artifact, segmentation artifact, shadow artifact, or focal signal loss were excluded. These designations were designed to exclude images where analysis would be hindered by the severe artifact. Only subjects that had good-quality images as determined by trained iMIND study staff (AK, SF, DSG) for all four image inputs, GC-IPL color thickness maps, 6 \\u00d7 6-mm OCTA superficial capillary plexus images, UWF color photographs, and UWF autofluorescence (FAF) photographs, were included in the analysis. One 512 \\u00d7 128-\\u00b5m macular cube scan was acquired for each eye, and the GC-IPL thickness color map was exported in tagged image file format (TIFF) for analysis. For each eye, a 6 \\u00d7 6-mm OCTA scan centered on the fovea was acquired, and the superficial capillary plexus (SCP) image was exported in bitmap format for analysis. UWF fundus color and FAF photographs were obtained by trained study personnel using nonmydriatic SLO (Optos California; Optos, Marlborough, MA). Fundus photos with peripheral distortion or motion and eyelid artifact within the major vascular arcades were excluded. In total, 296 control eyes (80%) were used for training, 36 (10%) for validation, and 39 (10%) for testing; 59 PD eyes (79%) were used for training, six for validation (8%), and 10 (14%) for testing. Two hundred and four eyes of control subjects and 39 eyes of PD subjects were used for development and testing of the model that considered only right eyes, randomly assigned to an 80% train, 10% validation, and 10% test split. Also, 169 eyes of control subjects and 36 eyes of PD subjects were used for development and testing of the model that considered only left eyes, randomly assigned to an 80% train, 10% validation, and 10% test split. Deep learning is a branch of machine learning that uses neural networks with multiple layers to model complex patterns. Three separate deep-learning models were developed, using images from the right eye only, left eye only, or both eyes. The single-eye models were developed because some subjects had only one imageable eye or a poor image quality for one eye. All models were trained to receive inputs of GC-IPL thickness color maps, OCTA SCP 6 \\u00d7 6-mm en face images, and UWF fundus color and FAF photographs to produce a score suggesting whether a subject carried a clinical diagnosis of PD. For dataset preprocessing, all images were resized with Python Imaging Library to 256 \\u00d7 256 pixels and converted to 224 \\u00d7 224 PyTorch tensors to be compatible with the VGG19 architecture. For dataset augmentation, all images were subjected to random horizontal and vertical flips with a probability of 0.5. For all models, an 80% train, a 10% validation, and a 10% test split were used. To ensure that there was no overlap among the training, validation, and test groups, random assignment was done by patient ID so that eyes of the same subject were assigned to the same group (i.e., training, validation, or test). The model parameters with the best performance on the validation split were selected for testing. The median area under the receiver operating characteristic (ROC) curve (AUC) of 10 model training runs was selected. The model consisted of a shared pretrained convolutional encoder, parallel image modality\\u2013specific feature transformations, and prediction heads for each modality that converge into a single output ( Illustration of the PD classification model architecture. The model leverages a pretrained VGG19 CNN as a shared-weight image feature extractor (shown in the  The average age of the cognitively normal control cohort was compared with that of the PD cohort using an independent sample \",\n          \"Supporting information\\n\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"results\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 434,\n        \"samples\": [\n          \"Results\\nOur final sample comprised data from 189 participants including 82 PD (aged 65\\u2009\\u00b1\\u20099\\u2009years; 68% male) and 107 HC (aged 66\\u2009\\u00b1\\u20097\\u2009years; 48% male;  Demographic and clinical information, and quantitative substantia nigra measurements for Parkinson\\u2019s disease and control groups, compared between groups. MDS-UPDRS-III, Movement Disorders Society Unified Parkinson Disease Rating Scale Part III; H&Y, Hoehn & Yahr; LEDD, levodopa equivalent daily dose; QSM, quantitative susceptibility mapping; NMS, neuromelanin-sensitive.** Mean (standard deviation); groups compared by independent samples t-test. Frequency (percent); groups compared by Chi-square test. Median (inter-quartile range); groups compared by Mann\\u2013Whitney U test. Descriptive statistics for quantitative substantia nigra MRI measurements, split by hemisphere and compared between Parkinson\\u2019s disease and healthy control groups. QSM, quantitative susceptibility mapping; NMS, neuromelanin-sensitive.** Median (inter-quartile range); groups compared by Mann\\u2013Whitney U test. The full classification performance results are summarized in  Results of the binary Parkinson\\u2019s disease versus healthy control classification by each model based on bilateral and single hemispheric findings. AUC, area under the receiver operating characteristic curve; NMS, neuromelanin-sensitive; QSM, quantitative-susceptibility mapping. Comparison of model performances for Parkinson\\u2019s disease diagnostic classification using high resolution midbrain MRI. Coloured bars are based on original analysis performed in this study, while white bars represent classification performed by  Heuron IPD could also classify PD patients with excellent performance (AUC\\u2009=\\u20090.92, accuracy\\u2009=\\u200990% sensitivity\\u2009=\\u2009100%, specificity\\u2009=\\u200983%). The 18/189 (9.5%) cases that were incorrectly classified (all false positives) had smaller left Heuron NI volumes ( Heuron NI (N1 volume) classified PD patients with moderate performance (AUC\\u2009=\\u20090.90, accuracy\\u2009=\\u200985%, sensitivity\\u2009=\\u200984%, specificity\\u2009=\\u200985%). There were both false positive (16/188; 8.5%) and false negative (13/188; 7.0%) classifications which, together, did not differ from the correctly classified cases in any demographic, clinical or imaging measure (Mann\\u2013Whitney U test; all  As a benchmark, our experienced neuroradiologist performed a classification using the SMWI alone with excellent performance (AUC\\u2009=\\u20090.98, accuracy\\u2009=\\u200997%, sensitivity\\u2009=\\u200999%, specificity\\u2009=\\u200996%), with the most notable difference from the DL and quantitative approaches being the high specificity. A neuroradiologist visual  Examples of misclassification by DL model(s) in three healthy controls (false positives),  The left hemisphere had consistently better performance than the right across all models except for Heuron IPD. The left\\u2013right discrepancy in AUCs for Heuron NI volume, NMS contrast range, NMS-based volume, QSM-based volume, iron-neuromelanin composite and neuroradiologist was 0.059, 0.005, 0.026, 0.052, 0.030 and 0.002, respectively. Our PD cohort was 100% right-handed, so we could not further evaluate the impact of handedness. The side of initial symptom onset for PD patients was left for  Correlation of clinical motor symptom severity (MDS-UPDRS-III, H&Y stage) with the continuous imaging measures when both the PD and HC groups were included showed more severe disease correlating with smaller Heuron NI-, QSM- and NMS-based volumes and lower NMS contrast (Spearman \\u03c1\\u2009<\\u2009\\u22120.476,  Based on the hemispheric differences we observed, we explored  Correlation (correlation coefficient,  MDS-UPDRS-III, Movement Disorders Society Unified Parkinson Disease Rating Scale Part III; H&Y, Hoehn & Yahr; LEDD, levodopa equivalent daily dose; QSM, quantitative-susceptibility mapping; NMS, neuromelanin-sensitive. **significant after multiple comparison correction, \\u03b1\\u2009<\\u20090.00125.* Spearman correlation, reported as \\u03c1. Kendall Tao Beta correlation, reported as \\u03c4 Significant left-sided correlations in the PD group between nigrosome-1 (N1) imaging measures and clinical parameters. LEDD, levodopa equivalent daily dose; NMS, neuromelanin-sensitive.\",\n          \"3. Results\\nThe antioxidant effects of extracts from  The Total Antioxidant Capacity (TAC) assay was used to measure the general antioxidant potential. For this assay, a concentration of 100 \\u00b5g/mL was selected for both the ethanolic (EE) and aqueous (EA) extracts. The results indicated that the antioxidant activity of EE was almost three times higher than that of the EA extract ( Moreover, in the DPPH assay, both extracts exhibited identical activities at all three tested concentrations (50 \\u00b5g/mL, 100 \\u00b5g/mL, and 250 \\u00b5g/mL), achieving 100% DPPH scavenging potential ( For the phenolic compounds, both extracts presented similar amounts ( In addition, the identification of biomolecules using HPLC followed by GC-MS/SIM indicated the presence of phytol, vitexin, rutin, and \\u03b1-tocopherol, as detailed in  Both extracts, EE and EA, at concentrations of 50 \\u00b5g/mL, 100 \\u00b5g/mL, and 250 \\u00b5g/mL exhibited no cytotoxicity to NIH/3T3 cells, as evidenced by nearly 100% MTT reduction ( Both extracts, EE and EA, were evaluated in cell lines to determine whether they could protect against copper-induced oxidative stress. For the EE extract, no significant differences were observed between the different concentrations of 100 and 250 \\u00b5g/mL, but it was effective in protecting the cell line against oxidative stress compared to the positive control ( The previously obtained results for cell line treatments guided the choice of concentrations for this assay: 100 \\u00b5g/mL and 250 \\u00b5g/mL for both the EE and EA extracts from  As the extracts from \",\n          \"Results\\nBased on 337 normal samples and 509 tumor samples from the TCGA and GTEx datasets, systematic clustering demonstrated the genomic differences between the normal tissues and THCA tissues ( Genes within a common functional group exhibit similar expression patterns and share expression regulation mechanisms ( A total of 426 overlapping genes were identified between the DEGs and the top two most relevant CEMGs. These genes have been identified as candidate hub genes for THCA and were subsequently used in further analyses ( The BP category was enriched with GO terms that predominantly encompassed synapse organization, wound healing, epidermis development and skin development ( A PPI network was established to assess the interconnections among the protein-encoding DEGs associated with THCA. Among these DEGs, the foremost five genes, namely FN1, mucin-1 (MUC1), keratin (KRT) 19, intracellular adhesion molecule 1 (ICAM1) and neural cell adhesion molecule (NCAM1), were identified as key molecules due to their notable connectivity ( Immunohistochemistry staining outcomes for five key genes were derived from the Human Protein Atlas (HPA) database ( To assess of the roles of the identified key molecules in TME immune cell infiltration, correlation analyses were performed between key genes, TME-infiltrating cells and immune checkpoint inhibitors. Using the PCA algorithm, unique enrichments of immune cell populations were revealed within the two groups ( A higher tumor (T) stage, node (N) stage, pathological stage, extrathyroidal extension and PFI event was significantly associated with increased levels of FN1 expression ( Using TCGA gene expression data, GSEA was performed to determine the biological and functional pathways between high- and low-FN1 gene expression groups. Based on the normalized enrichment scores, the enrichment signaling pathway that was determined to be the most relevant for FN1 gene expression was chosen ( FN1 serves a crucial role in cell adhesion and migration processes across several biological events, including embryo development, wound healing, blood clotting, host defense, metastasis and cell proliferation ( Patients with papillary carcinoma and follicular carcinoma were excluded due to their favorable prognosis. The prognostic value of FN1 was evaluated using Kaplan-Meier survival analysis. Patients diagnosed with THCA with low FN1 expression levels demonstrated a significant survival advantage compared with those with high FN1 expression levels (\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"conclusion\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 397,\n        \"samples\": [\n          \"Discussion\\nComputational genomic approaches have been valuable for linking candidate disease-associated genetic variants to regulatory function in specific tissues and cell types. However, these inferences may be confounded when compared across cell types that are highly similar. Here, we propose a computational framework which enables prediction of variant effects on regulatory activity in disease-relevant cell types. First, we use stratified LD score regression (S-LDSC) to directly compare GWAS association data with open chromatin peaks in multiple cell types. Next, for the implicated cell types, we train regression convolutional neural networks that are able to predict the cell type-specific effect of candidate genetic variants. We are also able to validate the increased sensitivity of our models using MPRA data. Using this framework, we systematically compare the regulatory impact of genetic variants on multiple neural and immune cell types that have been implicated in the predisposition to AD, some of which are highly similar. Surprisingly, both S-LDSC and the regression CNN approach suggest strong roles for peripheral monocytes as well as brain-resident microglia. First, we find that both blood cell types and microglia are enriched for LOAD-associated genetic variants relative to other tissues. We also find that for some GWASs, blood myeloid cell OCRs display a stronger enrichment than microglia, the brain-resident myeloid cell type. This suggests that, while many AD-associated variants lie in regulatory elements shared between blood and brain immune cell types, a significant subset lie in regulatory elements that are active in only blood immune cell types. Second, several of the candidate FAD-SNPs we identify are predicted by CNN regression models to have a stronger impact on gene regulation in CD14+-monocytes than microglia, are part of finemapped credible sets derived from AD GWAS and have relevant eQTL associations in the corresponding cell types. While multiple groups have sought to understand the function of regulatory variants in microglia [ We acknowledge that the variants unique to CD14+-monocytes may act only in specific microglial states such as activated microglial states associated with AD related inflammation that our microglia datasets might not have captured. Therefore, future research on the function of regulatory AD-associated variants needs to include not only models of inactivated microglia but also other microglial states, variety of which have been observed in the AD brain [ Using our models, we identified both known and novel variants that are predicted to disrupt regulatory activity in the relevant cell types identified in S-LDSC analysis. For example, our models were able to identify a peripheral immune-specific effect of rs28834970 in the CLU/PTK2B locus which recapitulates its previously reported role in altering chromatin accessibility. Model interpretation revealed that this variant disrupts nearby binding sites for CEBP factors which are expressed in both microglia and monocytes. Notably, CEBP motifs are more enriched in peripheral immune OCRs relative to microglia OCRs suggesting certain peripheral immune specific mechanisms. Future work investigating the peripheral immune-specific roles of the CLU/PTK2B, rs28834970, and its disrupted transcription factors is required. We also found rs76726049, a variant near the ALPK2 gene as having a CD-14+ monocyte-specific effect relative to microglia from our bulk open chromatin- trained models ( Genetic variants associated with complex diseases can have relatively subtle effects on regulatory activity. Therefore, it is important that computational approaches that aim to predict variant effects can capture these subtle effects. In our framework, we hypothesized that modeling quantitative signals from regulatory assays could improve variant effect prediction performance compared to the commonly trained binary classifiers in the literature. We show that models trained on separate open chromatin measurements (DNase-seq/ATAC-seq) that solely use inherent genetic variation across the genome reference instead of being explicitly fed in sequences with reference and alternate alleles of variants perform comparably well at predicting the direction of variant effects compared to the models trained directly on MPRA data. As hypothesized, we also found that regression models perform better than our classifiers as well as the Sei model [ We rely on MPRA data for validation since it is not influenced by LD as much as other datasets that have been used for variant effect validation such as eQTL data [ We show that models trained on DNase-seq display moderate performance on predicting expression in an MPRA. However, these DNase-seq trained models underperform a MPRA-trained model. MPRA datasets are inherently noisy with low replicate concordance, and hence, it is possible that this difference in performance may be due to the fact that the MPRA-trained model is overfitting to noise in the MPRA data. In addition, MPRA is primarily an episomal assay whereas DNase-seq and ATAC-seq assay chromosomal DNA directly. Further, open chromatin regions sometimes do not function as enhancers or promoters, and this contaminates the training set for open chromatin trained models. All of these factors could be contributing to the drop in performance for both the prediction of expression and variant effect and future work could explore the specific contribution of each factor. Interestingly, we found that using transfer learning improves performance at predicting expression in MPRA. A model pre-trained on DNase-seq data and fine-tuned on a subset of the MPRA data performed better than models trained solely on DNase-seq or solely on MPRA data. These results suggest that sharing of information and training across assays can help better predict MPRA regulatory activity. For primary tissues, open chromatin datasets are more readily available than MPRA datasets, and training models on open chromatin can help better design MPRA assays. Further, fine-tuning open chromatin pre-trained models on small MPRA datasets can in turn better optimize design of future MPRA experiments. In this way, when there are experimental constraints, it may be possible to attain accurate models of MPRA regulatory activity with a much smaller number of MPRA sequences. This approach could be used to combine open chromatin data and data from a small MPRA to design another small MPRA that would contain likely active regulatory elements, enabling active regulatory elements to be identified without the need for large numbers of cells or especially deep sequencing. We note that using data from LCLs to train these models isn\\u2019t sufficient to investigate the function of many disease-associated variants since they may only act  Our model predictions of variant effect in AD can aid in the prioritization of variants for experimental assays or could serve as additional sources of information that help validate experimental results. In the former case, we foresee that variants identified by our models could be prioritized for MPRA experiments in primary microglial or peripheral myeloid cells for direct assaying of their regulatory effects. This would be useful when the number of variants/sequences that can be assayed in the experiment is limited due to experimental constraint. Previous MPRAs have largely been conducted in cell lines that are easy to manipulate and allow for assaying of a large number of sequences. Since such cell lines contain chromosomal aberrations and are not especially representative of any primary tissue in a behaving individual, it is important to adapt MPRA experiments to primary tissues. This has proven challenging in the past ([ One limitation of our models is that they do not take into account enhancer-gene linking data. In most cases, we use the nearest gene as a proxy for which gene an enhancer may be regulating [ Overall, our framework enables cell type targeted studies of variants associated with specific diseases. As opposed to training on hundreds of cell types or tissues [\",\n          \"5. Conclusions\\nThis pilot study provides the first observation of changes in NSs\\u2019 peripheral levels in GBA-PD. The involvement of the observed changes in the development of neuro-psychological and motor symptoms of GBA-PD deserves further attention.\",\n          \"Conclusion\\nIn summary, this study assessed the causal relationships between IL-6Ri and various diseases, discovering that IL-6Ri can reduce the risk of three diseases while potentially increasing the risk of asthma and asthmatic pneumonia. This provides preliminary evidence for the potential expansion of therapeutic applications for IL-6Ri.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"fulltext\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 999,\n        \"samples\": [\n          \"Introduction\\nDementia, for which aging is the greatest risk factor, is a social burden in an aging society. According to the World Alzheimer Report 2022, the number of dementia patients will increase from 57.4 million in 2019 to 152.8 million in 2050 [\\nAccumulating evidence suggests that lifestyle habits after middle age (approximately 45\\u201354 years-old) significantly impact the maintenance of cognitive function in older adults [\\nRandomized, double-blind, placebo-controlled trials have reported beneficial effects of matcha on cognitive function [\\nAging causes sleep quality to deteriorate. Approximately 20% of adults in Japan experience sleep disturbance [\\nPreventing disease progression and disease onset is crucial for mitigating the prevalence of dementia and other lifestyle-related diseases. Controlling blood pressure and blood sugar levels in midlife is vital for the prevention of cardiovascular disease and diabetes mellitus, respectively. Several cohort studies have identified blood-based biomarkers for dementia risk assessment, including phosphorylated tau-217 [\\nHere, we conducted a 12-month randomized, double-blind, and placebo-controlled clinical trial to assess the potential of matcha in improving sleep quality and delaying cognitive decline in older adults.\\nMaterials and methods\\nStudy design\\nA 12-month intervention was executed employing a randomized, double-blind, placebo-controlled, parallel-group design. Before the initiation of the study, the clinical trial was registered with the UMIN Clinical Trials Registry (UMIN-CTR), which is recognized by the International Committee of Medical Journal Editors (ICMJE) as an approved registry. This study was conducted and reported as a randomized controlled trial, adhering strictly to the CONSORT guidelines.\\nA meta-analysis was conducted to ascertain whether the effect size and statistical power were commensurate with the determined sample size, referencing three intervention studies that employed the Montreal Cognitive Assessment (MoCA) as the evaluative criterion [\\nThe required sample size was calculated using G*Power 3 software (Heinrich Heine University, D\\u00fcsseldorf, Germany). We chose to assess the difference between the medians of two independent groups using the Wilcoxon or Mann\\u2013Whitney test. With the parameters set as two-tailed, logistic distribution, an effect size of 0.7, a significance level of 0.05, and a power of 0.8, it was determined that each group required 31 cases. Anticipating a dropout rate of 7 of 40 recruited cases per group, we established a target sample size of at least 33 cases per group to validate the robustness of our findings (details are described in \\nRandomization\\nSequence generation\\nEligible participants with SCD or MCI were randomized using a computer-generated sequence. Participants were stratified based on two stratification factors, age at baseline (\\u2265 74 years / < 74 years) and \\nAllocation concealment mechanism\\nParticipants underwent randomization using a dynamic program implemented via a Microsoft Excel macro, employing simple randomization rather than permuted block randomization. Upon running the program, the participants were assigned to either Group A or Group B. However, the program did not reveal which was the intervention or control group. The randomization table created by the program was retained until all baseline stratification factors were fully aligned and participant inclusion was complete, ensuring concealment of the allocation.\\nAll eligible participants who provided informed consent and met the predefined inclusion criteria underwent randomization, a process administered at the discretion of the study site personnel overseeing recruitment and medical interviews. Subsequently, the study site transmitted a stratification factor response form to an independent allocation manager who was responsible for the allocation process, ensuring separation from the outcome evaluation analysis. The allocation manager utilized a dynamic randomization program incorporating provided stratification factors to assign participants to their respective groups based on the program\\u2019s output. Physicians and nurses responsible for participant inclusion and symptom assessment remained blinded to group assignments.\\nThe allocation sequence utilized a stratified approach, considering study site and medication compliance, employing a dynamic allocation method with the minimization technique. Concealment of randomization details from data management personnel and statistical analysts was upheld throughout the study while ensuring the database remained publicly accessible. The randomization list was securely maintained by designated allocation personnel throughout the study, mitigating any potential influence from principal investigators involved in the research.\\nBlinding\\nThroughout the study duration, participants were unaware of whether the capsules they consumed contained matcha or placebo until the study\\u2019s conclusion was obtained. Physicians, nurses, and principal investigators at the study site were informed that the participants had been assigned to provisional groups (Groups A and B) yet remained uninformed about the specific allocation to either the matcha or placebo group until the randomization list was released after the study period. No member of the research team knew the allocated sequence until the end of the statistical analysis, after the database was locked.\\nParticipants\\nCommunity-dwelling older adults aged 60\\u201385 years were recruited and assessment for eligibility was performed at the University of Tsukuba Hospital (Tsukuba, Ibaraki, Japan) and Memory Clinic Toride (Toride, Ibaraki, Japan) (\\nStudy design and flowchart of participants throughout the study.\\nMCI, mild cognitive impairment; SCD, subjective cognitive decline.\\nIn this study, we included patients with SCD and those with MCI to evaluate the effects of matcha consumption in the pre-stage of cognitive impairment. Although SCD is not classified as a disease, an increasing number of studies have suggested an association between SCD and an elevated risk of developing cognitive impairment [\\nParticipants with SCD and MCI were randomly divided into matcha and placebo groups (see Materials and Methods in \\nNeurocognitive tests were performed before and after the intervention (baseline and 12 months) and 6 months after the intervention. Amyloid PET was performed at the baseline and 12 months after the intervention in 12 participants each in the matcha group and placebo group. Written informed consent was obtained from all the eligible participants. The study was conducted in accordance with the Declaration of Helsinki, and all protocols used in this study were approved by the Institutional Ethics Committee of the University of Tsukuba, Memory Clinic Toride, and MCBI. Prior to inclusion in the study, written informed consent was obtained from each participant, and all samples were rendered anonymous before being sent to the laboratory for analysis. We actively embraced the principles of diversity, equity, and inclusion (DEI) throughout the entirety of our research process. Our commitment to DEI was reflected not only in our team composition but also in our study design, execution, and interpretation of results. Each participant was treated with respect, dignity, and fairness, and every effort was made to ensure that the research process was culturally competent and responsive to the needs of the diverse participants.\\nProcedures for diagnosis\\nThe MCI and SCD criteria were applied to individuals without dementia after the conference. The team reviewed functional, medical, neurological, psychiatric, and neuropsychological data to reach a diagnostic consensus on dementia and MCI according to the DSM-5 (American Psychiatric Association, 2013) criteria. SCD was defined according to recent criteria [\\nConsistent with the standard criteria, for all subtypes of MCI described below, the participants considered to have MCI were required to have (1) objective impairment in one cognitive domain based on the average of scores on neuropsychological measures within that domain, and 1 standard deviation (SD) and 1.5 SD cutoffs derived from normative corrections for age, years of education, and sex; (2) essentially preserved activities of daily living; (3) presence of memory complaints; and (4) no diagnosis of dementia by group consensus.\\nThe participants labeled as having SCD were required to have (1) no diagnosis of MCI or dementia; (2) subjective complaints of forgetfulness; (3) decline in cognitive functioning unrelated to an event explaining the cognitive deficits; (4) Clinical Dementia Rating (CDR) global score 0 and (5) MMSE-J score between 27 and 30, mild hippocampal atrophy on MRI, or mild regional cerebellar blood flow (rCBF) reduction on single-photon emission computed tomography (SPECT).\\nIntervention and outcome\\nThe participants were required to consume nine capsules of matcha (equivalent to 2 g of matcha) or placebo daily at any time. As the amount of matcha in the traditional tea ceremony, Tea Otemae, is 2 g, the daily intake quantity of matcha was set at 2 g in this study. The matcha used, \\u201cHojin no shiro\\u201d (provided by ITO EN, LTD., Tokyo, Japan) contained 170.8 mg of catechin, 48.1 mg of theanine, and 66.2 mg of caffeine per daily serving. The catechins consisted of 105.3 mg of EGCG, 1.5 mg of gallocatechin gallate, 20.3 mg of epicatechin gallate, 0.1 mg of catechin gallate, 33.1 mg of epigallocatechin, 1.6 mg of gallocatechin, 8.1 mg of epicatechin, and 0.8 mg of catechin. The placebo capsules, which were identical in appearance, color, and odor to the matcha capsules, were composed of cornstarch. Both capsules consisted of white porcine gelatin. A daily supply (nine capsules) was sealed in a sachet, and a large bag containing 1 month\\u2019s supply of capsules was provided to the participants every 3 months. During the 3-, 6-, 9-, and 12-month visits throughout the intervention periods, intake compliance was assessed by counting the number of capsules that remained in the bag.\\nPrimary outcomes measures were Montreal Cognitive Assessment-Japanese version (MoCA-J) and Alzheimer\\u2019s Disease Cooperative Study Activity of Daily Living (ADCS-MCI-ADL) scores, and secondary outcome measures were MMSE-J, Repeatable Battery for the Assessment of Neuropsychological Status-Japanese version (RBANS), Alzheimer\\u2019s Disease Assessment Scale-Cognitive Subscale-Japanese version (ADAS-Jcog), CNS vital signs computerized neurocognitive battery, Cognitrax [\\nCognitive function and sleep quality measures\\nCognitive function was assessed using MMSE-J, MoCA-J, ADAS-Jcog, RBANS, and CNS vital signs. CNS vital signs contains 6 neurocognitive domain scores assessed by 10 tests. In this study, we used the stroop test and tests for shift attention, continuous performance, and perception of emotion to evaluate cognitive functioning, based on neurocognitive domain scores of social acuity, reaction time, complex attention, cognitive flexibility, executive function, and simple attention (\\nNeuroimaging\\nAmyloid PET was performed at the AIC Imaging and Inspection Center in Tsukuba, Japan. Scans were performed using a TruePoint Biograph-PET-CT system (Siemens AG, Munich, Germany). 18F-Florbetapir (Fujifilm, Tokyo, Japan) were used as imaging agents. Transmission imaging (CT imaging for attenuation correction) was performed for approximately 30 s, and emission imaging (PET imaging) was performed in ListMode for 20 min. The standard uptake ratio (SUVR) was calculated from acquired PET images.\\nThe voxel-based specific regional analysis system for atrophy detection (VSRAD) approach in MRI allows the evaluation of atrophy in specific regions of interest (ROIs) within medial temporal structures, particularly the entorhinal cortex, hippocampus, and amygdala. Each processed segmented image was compared with the mean and standard deviation of gray matter or white matter images derived from 80 healthy (40 male and 40 female, aged 54 and 86 years) controls. The severity of atrophic changes was calculated using the Z-score, defined as follows: Z-score = ([control mean]\\u2013[individual value]) / (control standard deviation). Each voxel had dimensions of 2 \\u00d7 2 \\u00d7 2 mm. The severity of ROI atrophy was determined by using the following equation: sum of positive Z-scores in the target ROI/total number of voxels in the target ROI. ROI atrophy was categorized into four states: 0\\u20131 representing normal, 1\\u20132 mild, 2\\u20133 moderate, and > 3 severe, based on the reference literature [\\nThe rCBF was evaluated using SPECT. The Z-score also allowed comparisons of the SPECT scans of participants and age-matched normal controls. Statistical analysis of the perfusion reduction was conducted. Severity perfusion reduction in (posterior cingulate gyrus, precuneus, and parietal cortex) was calculated using the following equation: severity = sum of positive Z-scores in the target ROI / total voxels in the target ROI [\\nSerum and plasma sampling\\nSerum and EDTA plasma stored at \\u221280\\u00b0C were transferred to the laboratory for analysis and thawed on ice; 40-\\u03bcl aliquots were generated and stored at \\u221280\\u00b0C for immunoassay. Almost all serum samples were frozen and thawed twice, whereas some were frozen and thawed three times. Freezing and thawing up to three times did not affect the results of the immunoassay.\\nImmunoassay\\nPlasma A\\u03b21\\u201340 and A\\u03b21\\u201342 levels were determined by immunoassay (Euroimmun, L\\u00fcbeck, Germany). Plasma levels of amyloid precursor protein (APP) and beta-site amyloid precursor protein cleaving enzyme 1 (BACE1), which are involved in A\\u03b2 production, were measured using enzyme-linked immunosorbent assay (ELISA). The human sAPP\\u03b1 assay kit (IBL 27734) and human sAPP\\u03b2-w assay kit (IBL 27732) were purchased from Immuno-Biological Laboratories (Gumma, Japan), and BACE-1 ELISA (Euroimmun) was used. The immunoassay kit for brain-derived neurotrophic factor was purchased from R&D Systems (Minneapolis, MN). The blood levels of malondialdehyde (MDA)-modified low-density lipoprotein (MDA-LDL), IL-6, and IL-10 were measured at the Clinical Chemistry Laboratory (BIKEN, Osaka, Japan). Plasma levels of apoA1, transthyretin, and complement protein C3 were determined as previously described [\\nStatistical analyses\\nThe analysis population consisted of all participants, excluding those who met the discontinuation criteria assessed by a physician, discontinued participation in the study, and conformed to the study protocol. We evaluated the effects of the matcha intervention on cognitive function and sleep quality measures using a mixed-effects model that flexibly modeled the variance structure of the effect measures, including intra-individual correlations generated by repeated measurements as follows. Specifically, we employed an unstructured variance that did not presuppose a specific correlation structure and included group effects, timing effects, and group-by-time interaction terms.\\n\\u03b1 is the intercept, \\u03b2\\nThe primary hypothesis to be examined is a two-group comparison of the mean change between baseline and time point 3 (and time point 2). We denote this hypothesis as \\nWe also estimated the change (mean difference) by time period, \\u0394(BL, 6M) = 6M \\u2013baseline or \\u0394(BL, 12M) = 12M-baseline. The difference between groups was defined by the following equation to estimate the difference in means (Estimate) between groups. Estimate ((difference of means A6M \\u2013A0M)\\u2013(difference of means B6M \\u2013B0M)), or Estimate ((difference in means A12M \\u2013A0M)\\u2013(difference in means B12M \\u2013B0M)). A t-test was conducted with the null (H) and alternative (K) hypotheses. \\nFor biomarker analysis, differences between the intervention and non-intervention groups were tested using the Mann\\u2013Whitney U test. The associations between the cognitive tests were analyzed using Spearman\\u2019s correlation coefficients. \\nResults\\nParticipants\\nAs shown in \\nEight participants discontinued the study due to withdrawal of consent and disease onset, which could have affected continuation of the intervention. Discontinuations due to illnesses included one patient who developed lung cancer, one patient who developed pancreatic cancer, one patient with increased intraocular pressure, one patient with increased transaminases, and two patients with poor physical condition. Two participants withdrew consent.\\nThere were no cases of a causal relationship between harmful events and matcha consumption. The total number of completed questionnaires was 46 in the matcha group and 45 in the placebo group. During the intervention, each participant with MCI in the matcha and placebo groups was diagnosed with dementia at the 12-month evaluation. After excluding these two participants, the per-protocol analysis of the intervention effects included 45 and 44 participants in the matcha and placebo groups, respectively. According to the intention-to-treat principle, we analyzed the changes in outcome variables from baseline to 12 months were statistically analyzed using a mixed-effects model, with 49 and 50 participants in the matcha and placebo groups, respectively.\\nBaseline characteristics of participants\\nAll the baseline characteristics of biochemical blood tests and neuroimaging data of the participants, including \\nDemographic characteristics of participants in the matcha and placebo groups at baseline.\\n* mean \\u00b1 SD\\nBrain atrophy and rCBF levels of participants in matcha and placebo groups at baseline.\\n*median (IQR, Interquartile range)\\nAmyloid PET of participants in matcha and placebo groups at baseline.\\n* median (IQR, Interquartile range)\\nIn amyloid PET, participants > 75 years of age and participants with \\nThere was no difference in the average age (\\nEffect of 12-month matcha consumption on sleep quality and cognitive function\\nIntake compliance was assessed by counting the number of capsules that remained in the bag at the 3-, 6-, 9-, and 12-month visits. The intake rates were 98\\u201399% throughout the intervention periods. We also verified matcha intake during the intervention period by measuring theanine levels in red blood cells. The matcha group demonstrated a significant increase in theanine levels (\\nFor the intention-to-treat analysis of each outcome, we tested the intervention effects using mixed-effects models, with 46 and 45 participants in the matcha group and placebo groups, respectively. The model equation consisted of a time effect (baseline, 6 months, and 12 months), group effect (matcha and placebo), and interaction between time and group (see \\nEffect of 12-month matcha intervention on sleep quality and cognitive functioning.\\n* Estimate: The two-group difference in mean change from baseline to 12-momths calculated as described in the Methods. Positive value means higher value and negative value means lower value in matcha group comparing to placebo group, respectively.\\nThe effect of match consumption on the outcomes was evaluated at 12 months. Social acuity in the neurocognitive domain scores of CNS vital signs showed significant improvement (difference; -1.39, 95%CI; -2.78, 0.002) (\\nSince the score for social acuity was assessed by the perception of emotion, we analyzed the outcomes of each cognitive test performed in CNS vital signs (\\nEffect of 12-month matcha consumption on neurocognitive domain score in CNS vital signs.\\nThe intervention effects on neurocognitive domain score in CNS vital signs are shown. Differences in the mean change from baseline to 12 months between the two groups were calculated using the mixed-effects model. Black circles represent the matcha group, and open circles represent the placebo group. The tables provided below represent the estimated differences between the groups; a positive value implies a higher value in the matcha group than in the placebo group, whereas a negative value suggests a lower value in the matcha group than in the placebo group.\\nAlthough there was no significant improvement in cognitive function as evaluated by conventional cognitive functioning tests used in clinical diagnosis (\\nEffect of 12-month matcha consumption on sleep quality and cognitive function.\\nThe intervention effects on PSQI, ADCS-MCI-ADL, cognitive function assessed by MMSE-J, MoCA-J, ADAS-Jcog, and RBANS are shown. Differences in the mean change from baseline to 12 months between the two groups were calculated using the mixed-effects model. Black circles represent the matcha group, and open circles represent the placebo group. The tables provided below represent the estimated differences between the groups; a positive value implies a higher value in the matcha group than in the placebo group, whereas a negative value suggests a lower value in the matcha group than in the placebo group.\\nNeuroimaging biomarkers\\nThe amyloid PET SUVR displayed no difference from the baseline and at 12 months in either group (\\nEffect of 12-month matcha intervention on neuroimaging.\\n* Estimate: The two-group difference in mean change from baseline to 12-momths calculated as described in the Methods. Positive value means higher value and negative value means lower value in matcha group comparing to placebo group, respectively.\\nDiscussion\\nIn this study, unbiased randomization was conducted, and the trial was implemented as a triple-blind study with blinding for the intervention and placebo groups as well as the statistical analyst, thus enhancing the reliability of the study. In the current study, amyloid PET measurements using 18F-Florbetapir were performed in 24 (9 male and 15 female) participants who were \\nPresent findings suggest that the consumption of matcha enhances certain cognitive functions, such as facial expression recognition and attention, and improves sleep quality, which are beneficial for maintaining cognitive function in older adult. Matcha consumption may be considered as a lifestyle improvement strategy for dementia prevention.\\nThe PSQI total score indicated a trend toward improved sleep with matcha consumption. Despite the presence of caffeine, which disrupts sleep, matcha demonstrated a sleep-enhancing effect. This beneficial effect is attributed to theanine, a constituent of matcha. A crossover study by Hidese et al. on healthy participants with an average age of 48 years revealed that theanine intake (200 mg/day) for 4 weeks significantly reduced the total PSQI score [\\nSubstantial evidence suggests a close relationship between sleep and cognitive function [\\nIn this study, a significant enhancement in social cognition was observed alongside an improvement in the total PSQI scores. It has been postulated that matcha may improve sleep quality and prevent or ameliorate cognitive decline. Thus, improvements in sleep quality associated with matcha intake may lead to cognitive function enhancement, even without the implementation of full-fledged sleep therapy. In the placebo group, a certain level of theanine was detected in the blood cells. This intervention study did not impose any restrictions on green tea intake before or during the study period. This may account for the presence of theanine, which may have been ingested during habitual green tea consumption.\\nFacial emotion recognition is impaired in patients with SCD and MCI [\\nThe matcha used in this study was powdered tea leaves. Its cultivation method differs from that of conventional green tea, as the tea leaves designated for matcha production are protected from sunlight prior to harvesting. This alternative cultivation approach increases the content of catechins and theanine, making it potentially superior to green tea in terms of the health benefits attributable to its ingredients. Participants drinking seven cups of matcha were excluded since it is equivalent to consuming 2 g of matcha contain 170 mg of catechins and 48 mg of theanine every day, which is the same amount of matcha as in this intervention study. In terms of catechin content, a single cup of matcha in a traditional tea ceremony, Tea Otemae, is equivalent to 250 mL (2\\u20133 cups) of green tea; however, it should be noted that various types of green tea exist, and not all are directly comparable to matcha. Given the established health benefits of green tea and its constituents, we hypothesized that the increased catechin and theanine contents in matcha could help suppress cognitive decline and improve sleep quality.\\nThe beneficial effects of matcha consumption revealed in this study may contribute to dementia prevention strategies, as these are relatively easy lifestyle changes to implement in daily life. Matcha consumption improved facial emotion recognition, as assessed by a computerized neurocognitive battery, since conventional psychological tests may not be suitable for evaluating cognitive decline in very mild cases. Thus, more sensitive neuropsychological testing, along with intervention strategies that are easy for elderly people to maintain, is necessary to assess the efficacy of these interventions in dementia prevention.\\nA limitation of the current study is the small number of participants. The observed PSQI score improvement within the intervention group suggests a potential influence of matcha consumption on sleep quality. However, future studies should employ more objective measures beyond the PSQI to comprehensively assess sleep architecture, including quantification of REM and non-REM sleep stages. A comprehensive clinical trial is required for a more robust evaluation of its effects on facial expression recognition. Furthermore, it is crucial to analyze the association between intervention outcomes and biomarkers to enhance our understanding of the mechanisms underlying the effects of matcha. Such investigations would make a significant contribution to the establishment of an effective ecosystem for dementia prevention.\\nConclusions\\nThe findings of this long-term intervention study suggest that regular matcha consumption can enhance emotional perception and sleep quality in older adults experiencing cognitive decline.\\nSupporting information\\nSupporting materials and methods.\\n(PDF)\\nSubsets of neurocognitive test corresponding to neurocognitive domain score used in the matcha intervention study.\\n(PDF)\\nBaseline clinical chemistry parameters of participants in the matcha and placebo groups.\\n(PDF)\\nEffect of 12-month matcha intervention on sleep quality and cognitive functioning in per-protocol analysis.\\n(PDF)\\nEffect of 12-month matcha intervention on neuroimaging in per-protocol analysis.\\n(PDF)\\n\\n\\nPONE-D-24-02571Effect of matcha green tea on cognitive functions and sleep quality in older adults with cognitive decline: A randomized controlled study over 12 monthsPLOS ONE\\nDear Dr. Uchida,\\nThank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE\\u2019s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.\\nPlease include the following items when submitting your revised manuscript:\\nA rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.\\nA marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.\\nAn unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.\\nIf you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.\\nIf applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: \\nWe look forward to receiving your revised manuscript.\\nKind regards,\\nTamlyn Julie Watermeyer\\nAcademic Editor\\nPLOS ONE\\nJournal Requirements:\\n1. When submitting your revision, we need you to address these additional requirements.\\nPlease ensure that your manuscript meets PLOS ONE's style requirements, including those for file naming. The PLOS ONE style templates can be found at\\u00a0\\n\\n\\n2. Thank you for stating the following in the Competing Interests section:\\u00a0\\nAuthors with competing interests\\nEnter competing interest details beginning with this statement:\\nI have read the journal's policy and the authors of this manuscript have the following competing interests: [Kazuhiko Uchida serves as a board member of MCBI Inc. Kohji Meno, Tatsumi Korenaga, Liu Shan, and Hideaki Suzuki are employees of MCBI Inc. Yoshitake Baba, Chika Tagata, Yoshiharu Araki, Shuto Tsunemi, Kenta Aso, Shun Inagaki, Sae Nakagawa, Makoto Kobayashi, and Takanobu Takihara are employees of ITO EN , LTD. This research received no external funding. ]\\n\\u00a0 \\u00a0\\nWe note that one or more of the authors are employed by a commercial company.\\u00a0\\na. Please provide an amended Funding Statement declaring this commercial affiliation, as well as a statement regarding the Role of Funders in your study. If the funding organization did not play a role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript and only provided financial support in the form of authors' salaries and/or research materials, please review your statements relating to the author contributions, and ensure you have specifically and accurately indicated the role(s) that these authors had in your study. You can update author roles in the Author Contributions section of the online submission form.\\nPlease also include the following statement within your amended Funding Statement.\\u00a0\\n\\u201cThe funder provided support in the form of salaries for authors, but did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript. The specific roles of these authors are articulated in the \\u2018author contributions\\u2019 section.\\u201d\\nIf your commercial affiliation did play a role in your study, please state and explain this role within your updated Funding Statement.\\u00a0\\nb. Please also provide an updated Competing Interests Statement declaring this commercial affiliation along with any other relevant declarations relating to employment, consultancy, patents, products in development, or marketed products, etc.\\u00a0\\u00a0\\nWithin your Competing Interests Statement, please confirm that this commercial affiliation does not alter your adherence to all PLOS ONE policies on sharing data and materials by including the following statement: \\\"This does not alter our adherence to\\u00a0 PLOS ONE policies on sharing data and materials.\\u201d (as detailed online in our guide for authors \\nPlease include both an updated Funding Statement and Competing Interests Statement in your cover letter. We will change the online submission form on your behalf.\\n3. In the online submission form, you indicated that The data cannot be shared publicly without permission from all authors. Access to the data is available through the Institutional Data Access/Ethics Committee (contactable via the corresponding author) for researchers who meet the criteria for accessing confidential data.\\nAll PLOS journals now require all data underlying the findings described in their manuscript to be freely available to other researchers, either a. In a public repository, b. Within the manuscript itself, or c. Uploaded as supplementary information.\\nThis policy applies to all data except where public deposition would breach compliance with the protocol approved by your research ethics board. If your data cannot be made publicly available for ethical or legal reasons (e.g., public availability would compromise patient privacy), please explain your reasons on resubmission and your exemption request will be escalated for approval.\\u00a0\\n[Note: HTML markup is below. Please do not edit.]\\nReviewers' comments:\\nReviewer's Responses to Questions\\n\\n\\n\\n1. Is the manuscript technically sound, and do the data support the conclusions?\\nThe manuscript must describe a technically sound piece of scientific research with data that supports the conclusions. Experiments must have been conducted rigorously, with appropriate controls, replication, and sample sizes. The conclusions must be drawn appropriately based on the data presented. \\nReviewer #1:\\u00a0Yes\\nReviewer #2:\\u00a0Partly\\nReviewer #3:\\u00a0Yes\\n**********\\n2. Has the statistical analysis been performed appropriately and rigorously? \\nReviewer #1:\\u00a0Yes\\nReviewer #2:\\u00a0I Don't Know\\nReviewer #3:\\u00a0Yes\\n**********\\n3. Have the authors made all data underlying the findings in their manuscript fully available?\\nThe \\nReviewer #1:\\u00a0No\\nReviewer #2:\\u00a0Yes\\nReviewer #3:\\u00a0Yes\\n**********\\n4. Is the manuscript presented in an intelligible fashion and written in standard English?\\nPLOS ONE does not copyedit accepted manuscripts, so the language in submitted articles must be clear, correct, and unambiguous. Any typographical or grammatical errors should be corrected at revision, so please note any specific errors here.\\nReviewer #1:\\u00a0Yes\\nReviewer #2:\\u00a0Yes\\nReviewer #3:\\u00a0Yes\\n**********\\n5. Review Comments to the Author\\nPlease use the space provided to explain your answers to the questions above. You may also include additional comments for the author, including concerns about dual publication, research ethics, or publication ethics. (Please upload your review as an attachment if it exceeds 20,000 characters)\\n-Data statement \\u2013 no explanation as to why data can\\u2019t be shared. States without permission from all authors but no explanation as to why all authors wouldn\\u2019t give permission. Is data owned by a third party?\\n- Typo on page 6, line 3, double comma\\n- Line 8 \\u2013 could some context be given as to what is being classified as \\u2018middle age\\u2019 here.\\n- Within intervention section it would be useful to provide some justification as to why this dosage was decided upon.\\n- For statistical analysis is there justification as to why per protocol participants were analysed rather than Intent To Treat. CONSORT recommends reporting both in parallel, so some acknowledgement of ITT would be beneficial here, were the results the same?\\n- The results are interesting and placed well within previous literature. However there were no effects on primary outcomes and this should be acknowledged somewhere in the discussion.\\nCOMMENTS: Kindly note that your ABSTRACT is well drafted (in my opinion), but is \\u2018assay type\\u2019. It is preferable [refer to item 1b of CONSORT checklist 2010: Structured summary of trial design, methods, results, and conclusions] to divide the ABSTRACT with small sections like \\u2018Objective(s)\\u2019, \\u2018Methods\\u2019, \\u2018Results\\u2019, \\u2018Conclusions\\u2019, etc. which is an accepted practice of most of the good/standard journals [including this one, though \\u2018The PLoS One Guidelines to Authors\\u2019 did not specify an Abstract format, it is desirable]. It will definitely be more informative then, I guess, whatever the article type may be. Since your \\u2018Article Type: Clinical Trial\\u2019, it very much desirable.\\nIn addition, there are few issues/observations about which I have different opinion. Such observations/concerns are given below [if acceptable/agreed, try to incorporate them (consider positively wherever possible)]:\\nFirst is regarding choice of \\u2018Sample size\\u2019. One of the very important items in CONSORT guidelines is \\u2019How sample size was determined (Item 7a)\\u2019 and expected/desired to described in ample details including all assumptions and software used. In fact, it is very surprising to note the \\u2018absence\\u2019 of the important term \\u201cCONSORT\\u2019\\u2019 despite that the manuscript is on \\u2018A randomized controlled clinical study/trial\\u2019. It is well-known that while reporting [findings from and even planning that is in \\u2018protocol\\u2019 of] \\u2018Clinical Trial\\u2019 one should follow/cover CONSORT guidelines.\\nIn \\u2018Discussion\\u2019 it is just said/mentioned that \\u201cIn this study, unbiased randomization was conducted, and the trial was implemented as a triple-blind study\\u201d. Regarding these important issues more discussion/detailed account is expected. Few important items {and all \\u2018Clinical Trials\\u2019 must follow} in CONSORT guidelines are \\u2018How sample size was determined (Item 7a)\\u2019,\\u2019 Random Sequence generation (Item 8a)\\u2019, \\u2018Allocation concealment (Item 9)\\u2019, Blinding (Item 11a). Since your article type is \\u2018Clinical Trial\\u2019, you are supposed to cover these items in the report or even in \\u2018Protocol\\u2019.\\nYou may know that \\u2018Permuted Block Randomization\\u2019 ensures same group sizes (not simple randomization) and that randomization is a process [not only sequence generation] which includes \\u2018Allocation Concealment\\u2019}. Which \\u2018Randomization\\u2019 procedure is used in this study is not specified but almost equal group sizes are obtained. It must be \\u2018Permuted Block Randomization\\u2019 but such clear mention is desired.\\nIn \\u2018Results-Participants\\u2019 section (as well as in Figure-1), you may use the word \\u2018screened\\u2019 in place of \\u2018recruited\\u2019 [As shown in Fig 1, we recruited 939 community-dwelling older adults aged 60\\u201385 years and 124 participants were enrolled], in my opinion. May please confirm as word \\u2018recruited\\u2019 implies that 939 participated in trial. It may be known to you that when random allocation/assignment is used/done, any statistical comparison of baseline characteristics is not required. In this context [refer to Table-1], I request authors to read following note pasted from one famous standard textbook on \\u2018Medical Research Methodology\\u2019:\\nTo provide a description of baseline characteristics is entirely reasonable (since it is clearly important in assessing to whom the results of the trial can be applied), however, statistical comparison of baseline characteristics when random allocation/assignment is used/done [often for good/standard/leading journals these days] is not required, because even if P-value(s) turn(s) out to be significant (while comparing baseline characteristics despite random allocation), it is, by definition, a false positive as you then are supposed to be testing \\u2018randomization\\u2019 then, which in any single trial may not balance all baseline characteristics (particularly when sample sizes are small). Remember that \\u2018randomization\\u2019 is a sort of \\u2018insurance\\u2019 and not a guarantee scheme. Authors may please refer to following articles:\\nReferences:\\n1. Stuart J. Pocock, et al., \\u2018Subgroup analysis, covariate adjustment and baseline comparisons in clinical trial reporting: current practice and problems\\u2019, Statistics in medicine, 2002; 21:2917\\u20132930 [Particularly page 2927]\\n2. Harrington D, et al., \\u2018New guidelines for statistical reporting in the journal\\u2019, N Engl J Med 2019;381:285-6\\n[Important message (indirectly/ultimately indicated) from these articles: Never do any comparison with respect to \\u2018baseline\\u2019 characteristics {by applying statistical significance test(s)}, when allocation is done randomly].\\nHowever, Statistical comparison [only with respect to important/indicated variables] of baseline characteristics may be performed, to find out if analysis adjustment (say stratified analyses or else) is required with respect to these variables.\\nWhat is the purpose of Table-3? Why Between group comparison is performed separately at \\u2018Baseline\\u2019 and \\u201912-months\\u2019? What is/are interpretation of insignificant (both) P-vale(s)? Why not the comparison using \\u2018change scores\\u2019? In all the tables, test name is indicated only for few \\u2018variables/parameters\\u2019 {by giving symbol as superscript on P-value (last column)}. Is not it required to indicate test name in case of (i.e. with respect to) other/remaining variables/parameters? Or is it left for readers to guess? Remember/mind you that this is a scientific/academic document and so all details should be clearly/correctly communicated (do not take readers\\u2019 for granted). Most of the findings are reported without properly interpreting them or given relevance to study (example: page 26).\\nOnly limitation of the study [A limitation of the current study is the small number of participants] mentioned on page 32 is agreed. However, does that mean {according to authors} there are none others? As pointed out in \\u2018important note\\u2019 above \\u201cThis review pertains only to \\u2018statistical aspects\\u2019 of the study and so \\u2018clinical aspects\\u2019 should be assessed separately/independently. In my opinion, to make this article acceptable some amount of re-vision (re-drafting) may be needed. However, please do not limit the revision only (with respect) to comments made here. \\u2018Major revision\\u2019 is recommended.\\n- Introduction is well written with relevant links to supporting green tea literature. It would be useful to include rationale for the intervention timeframe (12 months). There does seem to be some literature on matcha tea interventions that haven't been included, which is more relevant than the included green tea literature - for example a paper by Sakurai (2020) used matcha for a 12 week intervention in elderly Japanese individuals.\\n- Page 7 line 5 - missing refernece for frequency of green tea consumption. It would also be useful here to have a brief explanation of this finding and the relation to this trial\\n- Page 7 line 8 - missing refernece for sleep disturbance/disorder stats, it would be more useful here to have the stats for sleep disorder prevalence in Japan rather than the US\\n- I am interested to know why age was stratified as >74 years old/ <74 years old, when the age range was 60-85 years. Is there a reason that 74 was chosen rather than 72? A brief comment at Page 9 line 2 would be useful\\n- Page 9 Line 16 - points 3 and 4 of the eligibility criteria are the same, please check\\n- Within eligibility criteria you mention that participants could not consume more that 7 cups of matcha per week, there is a comment within the discussion that the study did not impose restrictions on green tea intake during the trial, why was this decided? Did you collect information on habitual intake of green tea/matcha during the trial? If so, were there any differences in consumption between treatment groups? High consumption in the placebo group could explain lack of findings.\\n- Intervention information - I'd be interested to know what 2g of matcha equates to in comparison with consuming as a tea and/or how the catechin/theanine/caffeine content compares with green tea. I think this is particularly interesting given the citation of various green tea literature and the decision to exclude participants consuming 7+ cups matcha each week.\\n- Page 16 Line 2 - Please provide comment on if the repeated freezing/thawing of samples may have impacted the immunoassay results. Was this per protocol? It would also be useful here to include details of the blood sampling process. It would also be useful to include some information about the biomarkers assessed (e.g. assessing vitamin status)\\n- Page 23 Line 7 - compliance - earlier a pill count was mentioned, do you have the information on treatment compliance based on this? Was there a compliance cut off used for including participants in analysis?\\n- Within sample size calculation information I don't think it is clear why a medium-large effect size (cohen's d = 0.7) was used. Was this based off previous green tea literature? Please expand.\\n- Whilst this is recognised several times, consider highlighting each time that the PSQI finding is a trend only to be careful when over interpreting this finding.\\nI also found a few minor typos:\\nPage 6 Line 3 \\u2013 extra comma\\nPage 20 line 4 \\u2013 matcha misspelled\\n**********\\n6. PLOS authors have the option to publish the peer review history of their article (\\nIf you choose \\u201cno\\u201d, your identity will remain anonymous but your review may still be made public.\\nReviewer #1:\\u00a0No\\nReviewer #2:\\u00a0No\\nReviewer #3:\\u00a0No\\n**********\\n[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link \\\"View Attachments\\\". If this link does not appear, there are no attachment files.]\\nWhile revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool,\\u00a0\\nSubmitted filename: \\n\\n\\nReponse to the reviewers\\u2019 comments\\nReviewer 1:\\nThank you for your insightful suggestions, we have made several changes in the revised manuscript according to the Reviewer\\u2019s comments. These changes have enriched the context of our research, providing a clearer pathway for our readers to understand the implications of our findings.\\n1. Data statement \\u2013 no explanation as to why data can\\u2019t be shared. States without permission from all authors but no explanation as to why all authors wouldn\\u2019t give permission. Is data owned by a third party?\\nAns: After careful consideration and discussion among the authors, we decided to share the data in this study. The data has been uploaded to Dryad. \\n\\n\\nWe have added the following sentence.\\nPage 36, lines 563-564\\nThe data in this study can be accessed in Dryad Digital Repository (\\n2. Typo on page 6, line 3, double comma\\nPage 5, lines 53\\nAns: We have corrected it in the revised manuscript.\\n3. Line 8 \\u2013 could some context be given as to what is being classified as \\u2018middle age\\u2019 here.\\nAns: According to reference (5), people aged 45-54 years were classified as \\u201cmiddle age.\\u201d We changed the sentences as follows in the revised manuscript. \\nPage 5, lines 58-60\\nAccumulating evidence suggests that lifestyle habits after middle age (approximately 45\\u201354 years old) significantly impact the maintenance of cognitive function in older adults [3\\u20135]\\n4. Within intervention section it would be useful to provide some justification as to why this dosage was decided upon.\\nAns: As the amount of matcha in the traditional tea ceremony (Tea Otemae) is 2 g, a daily intake of matcha was set at 2 g.\\nWe have added the following sentence.\\nPage 15, lines 233-235\\nAs the amount of matcha in the traditional tea ceremony (Tea Otemae) is 2 g, the daily intake quantity of matcha was set at 2 g in this study.\\n5. For statistical analysis is there justification as to why per protocol participants were analysed rather than Intent To Treat (ITT). CONSORT recommends reporting both in parallel, so some acknowledgement of ITT would be beneficial here, were the results the same?\\nAns: We have clarified the analytical methodologies concerning the intent-to-treat (ITT) and per-protocol (PP) in the revised manuscript. According to the ITT principle, we presented the data by ITT analysis in the original and revised manuscript. We analyzed changes in outcome variables from baseline to 12 months using a mixed-effects model using 46 participants in the matcha group and 45 in the placebo group. In addition to the ITT analysis, PP analysis was conducted, the results are shown in the S3 and S4 Tables of the revised manuscript, and corresponding modifications and textual additions have been made in the revised manuscript as follows:\\nPage 21, lines 350-352\\nA total of 99 participants were randomized and allocated to the matcha group (n = 49) and matched placebo group (n = 50).\\nPage 22, lines 366-373\\nThe total number of completed questionnaires was 46 in the matcha group and 45 in the placebo group. During the intervention, each participant with MCI in the matcha and placebo groups was diagnosed with dementia at the 12-month evaluation. After excluding these two participants, the per-protocol analysis of the intervention effects included 45 and 44 participants in the matcha and placebo groups, respectively. According to the intention-to-treat principle, we analyzed the changes in outcome variables from baseline to 12 months were statistically analyzed using a mixed-effects model, with 49 and 50 participants in the matcha and placebo groups, respectively. \\nPage 28, lines 422-425\\nWe obtained results from both intent-to-treat and per-protocol analyses. The per-protocol analysis showed that differences in social acuity and PSQI scores between the groups were -1.44 (95% CI: -2.88, 0.001; P = 0.077) and 0.89 (95% CI: -0.001, 1.74; P = 1.08), respectively (S3 and S4 Tables).\\n6. The results are interesting and placed well within previous literature. However, there were no effects on primary outcome, and this should be acknowledged somewhere in the discussion.\\nAns: We agree that the results of the primary outcome should be acknowledged. A corresponding statement has been added to the Discussion section. In the revised manuscript, we added the following sentences.\\nPage 28, lines 419-422\\nOther cognitive functions, including the primary outcomes (MoCA-J and ADCS-MCI-ADL), showed no significant changes in the matcha group compared with the placebo control group.\\nPage 34, lines 524-527\\nHowever, the matcha intervention had no effect on the primary outcomes (MoCA-J and ADCS-MCI-ADL) possibly due to their sensitivity to only slight declines in cognitive function. These tests are typically used during clinical diagnoses rather than for evaluating early-stage cognitive decline in the disease.\\n\\u2003\\nReviewer 2:\\nThank you for your crucial insights regarding the statistical elements of our study and the noted deviations from the CONSORT guidelines. These observations led us to meticulously revise the manuscript in accordance with the CONSORT standards. We believe that these adjustments have fortified the methods and results, and sharpened the conclusions, thereby enhancing the overall integrity of the manuscript.\\n1. Kindly note that your ABSTRACT is well drafted (in my opinion), but is \\u2018assay type\\u2019. It is preferable [refer to item 1b of CONSORT checklist 2010: Structured summary of trial design, methods, results, and conclusions] to divide the ABSTRACT with small sections like \\u2018Objective(s)\\u2019, \\u2018Methods\\u2019, \\u2018Results\\u2019, \\u2018Conclusions\\u2019, etc. which is an accepted practice of most of the good/standard journals [including this one, though \\u2018The PLoS One Guidelines to Authors\\u2019 did not specify an Abstract format, it is desirable]. It will definitely be more informative then, I guess, whatever the article type may be. Since your \\u2018Article Type: Clinical Trial\\u2019, it very much desirable.\\nAns: Following the reviewer's comment, we have revised the abstract in accordance with the CONSORT guidelines\\nPages 3-4, Abstract\\nObjective: Lifestyle habits after middle age significantly impact the maintenance of cognitive function in older adults. Nutritional intake is closely related to lifestyle habits; therefore, nutrition is a pivotal factor in the prevention of dementia in the preclinical stages. Matcha green tea powder (matcha), which contains epigallocatechin gallate, theanine, and caffeine, has beneficial effects on cognitive function and mood. We conducted a randomized, double-blind, placebo-controlled clinical study over 12 months to examine the effect of matcha on cognitive function and sleep quality. \\nMethods: Ninety-nine participants, including 64 with subjective cognitive decline and 35 with mild cognitive impairment, were randomized, with 49 receiving 2 g of matcha and 50 receiving a placebo daily. Participants were stratified based on two factors: age at baseline and APOE genotype. Changes in cognitive function and sleep quality were analyzed using a mixed-effects model.\\nResults: Matcha consumption led to significant improvements in social acuity score (difference: -1.39, 95% confidence interval; -2.78, 0.002) (P = 0.028) as evaluated by the perception of facial emotions in cognitive function. The primary outcomes, that is, Montreal Cognitive Assessment and Alzheimer\\u2019s Disease Cooperative Study Activity of Daily Living scores, showed no significant changes with matcha intervention. Meanwhile, Pittsburgh Sleep Quality Index scores indicated a trend toward improvement with a difference of 0.86 (95% confidence interval; -0.002, 1.71) (P = 0.088) between the groups in changes from baseline to 12 months.\\nConclusions: The present study suggests regular consumption of matcha could improve emotional perception and sleep quality in older adults with mild cognitive decline. Given the widespread availability and cultural acceptance of matcha green tea, incorporating it into the daily routine may offer a simple yet effective strategy for cognitive enhancement and dementia prevention.\\n2. In addition, there are few issues/observations about which I have different opinion. Such observations/concerns are given below [if acceptable/agreed, try to incorporate them (consider positively wherever possible)]:\\nFirst is regarding choice of \\u2018Sample size\\u2019. One of the very important items in CONSORT guidelines is \\u2019How sample size was determined (Item 7a)\\u2019 and expected/desired to described in ample details including all assumptions and software used. \\nIn fact, it is very surprising to note the \\u2018absence\\u2019 of the important term \\u201cCONSORT\\u2019\\u2019 despite that the manuscript is on \\u2018A randomized controlled clinical study/trial\\u2019. It is well-known that while reporting [findings from and even planning that is in \\u2018protocol\\u2019 of] \\u2018Clinical Trial\\u2019 one should follow/cover CONSORT guidelines.\\nIn \\u2018Discussion\\u2019 it is just said/mentioned that \\u201cIn this study, unbiased randomization was conducted, and the trial was implemented as a triple-blind study\\u201d. Regarding these important issues more discussion/detailed account is expected. Few important items {and all \\u2018Clinical Trials\\u2019 must follow} in CONSORT guidelines are \\u2018How sample size was determined (Item 7a)\\u2019,\\u2019 Random Sequence generation (Item 8a)\\u2019, \\u2018Allocation concealment (Item 9)\\u2019, Blinding (Item 11a). Since your article type is \\u2018Clinical Trial\\u2019, you are supposed to cover these items in the report or even in \\u2018Protocol\\u2019.\\nYou may know that \\u2018Permuted Block Randomization\\u2019 ensures same group sizes (not simple randomization) and that randomization is a process [not only sequence generation] which includes \\u2018Allocation Concealment\\u2019}. Which \\u2018Randomization\\u2019 procedure is used in this study is not specified but almost equal group sizes are obtained. It must be \\u2018Permuted Block Randomization\\u2019 but such clear mention is desired.\\nAns: According to the reviewer\\u2019s comment, we added the following sentences and references in the Materials and Methods: \\u201cStudy design,\\u201d \\u201cRandomization,\\u201d and \\u201cParticipants\\u201d of the revised manuscript.\\nPage 8, line 110\\uff0dpage 9, line 123\\nStudy design\\nA 12-month intervention was executed employing a randomized, double-blind, placebo-controlled, parallel-group design. Before the initiation of the study, the clinical trial was registered with the UMIN Clinical Trials Registry (UMIN-CTR), which is recognized by the International Committee of Medical Journal Editors (ICMJE) as an approved registry. This study was conducted as a randomized controlled trial, adhering strictly to the CONSORT guidelines. The required sample size was calculated using G*Power 3 software (Heinrich Heine University, D\\u00fcsseldorf, Germany). We chose to assess the difference between the medians of two independent groups using the Wilcoxon or Mann\\u2013Whitney test. With the parameters set as two-tailed, logistic distribution, an effect size of 0.7, a significance level of 0.05, and a power of 0.8, it was determined that each group required 31 cases. Anticipating a dropout rate of 7 of 40 cases per group, we established a target sample size of at least 33 cases per group to validate the robustness of our findings (details are described in S1 File).\\nPage 9, lines 124-133\\nRandomization\\nSequence generation\\nEligible participants with SCD or MCI were randomized using a computer-generated sequence. Participants were stratified based on two stratification factors, age at baseline (\\u2265 74 years / < 74 years) and APOE genotype (\\u03b54 positive/negative). Participants were stratified based on age at 74 years because a 10-year AD risk is higher in people \\u2265 74 years old than in those < 74 years old [30]. Allocation to either the control or intervention group was intended to be randomized in a 1:1 ratio using computerized minimization, which aims to minimize differences between groups with respect to the stratification factors [31]. \\nPage 9, line 134-page 11, line 166 \\nAllocation concealment mechanism\\nParticipants underwent randomization using a dynamic program implemented via a Microsoft Excel macro, employing simple randomization rather than permuted block randomization. Upon running the program, the participants were assigned to either Group A or Group B. However, the program did not reveal which was the intervention or control group. The randomization table created by the program was retained until all baseline stratification factors were fully aligned and participant inclusion was complete, ensuring concealment of the allocation.\\nAll eligible participants who provided informed consent and met the predefined inclusion criteria underwent randomization, a process administered at the discretion of the study site personnel overseeing recruitment and medical interviews. Subsequently, the study site transmitted a stratification factor response form to an independent allocation manager who was responsible for the allocation process, ensuring separation from the outcome evaluation analysis. The allocation manager utilized a dynamic randomization program incorporating provided stratification factors to assign participants to their respective groups based on the program's output. Physicians and nurses responsible for participant inclusion and symptom assessment remained blinded to group assignments.\\nThe allocation sequence utilized a stratified approach, considering study site and medication compliance, employing a dynamic allocation method with the minimization technique. Concealment of randomization details from data management personnel and statistical analysts was upheld throughout the study while ensuring the database remained publicly accessible. The randomization list was securely maintained by designated allocation personnel throughout the study, mitigating any potential influence from principal investigators involved in the research.\\nBlinding\\nThroughout the study duration, participants were unaware of whether the capsules they consumed contained matcha or placebo until the study's conclusion was obtained. Physicians, nurses, and principal investigators at the study site were informed that the participants had been assigned to provisional groups (Groups A and B), yet remained uninformed about the specific allocation to matcha or placebo groups until the randomization list was released after the study period. No member of the research team knew the allocated sequence until the end of the statistical analysis after the database was locked.\\nPage 11, lines 167-170\\nParticipants\\nCommunity-dwelling older adults aged 60\\u201385 years were recruited and assessment for eligibility was performed at the University of Tsukuba Hospital (Tsukuba, Ibaraki, Japan) and Memory Clinic (Toride, Ibaraki, Japan) (Fig 1).\\n3. In \\u2018Results-Participants\\u2019 section (as well as in Figure-1), you may use the word \\u2018screened\\u2019 in place of \\u2018recruited\\u2019 [As shown in Fig 1, we recruited 939 community-dwelling older adults aged 60\\u201385 years and 124 participants were enrolled], in my opinion. May please confirm as word \\u2018recruited\\u2019 implies that 939 participated in trial. It may be known to you that when random allocation/assignment is used/done, any statistical comparison of baseline characteristics is not required. In this context [refer to Table-1], I request authors to read following note pasted from one famous standard textbook on \\u2018Medical Research Methodology\\u2019.\\nAns: I would like to thank you very much for your valuable comments and recommendation of the textbook. I think that this is the book by Joseph Abramson. \\nI am sorry for the misuse of the term. We have corrected the word \\u201cscreened\\u201d to \\u201crecruited.\\u201d We have also revised Fig 1 and Table 1.\\n4. To provide a description of baseline characteristics is entirely reasonable (since it is clearly important in assessing to whom the results of the trial can be applied), however, statistical comparison of baseline characteristics when random allocation/assignment is used/done [often for good/standard/leading journals these days] is not required, because even if P-value(s) turn(s) out to be significant (while comparing baseline characteristics despite random allocation), it is, by definition, a false positive as you then are supposed to be testing \\u2018randomization\\u2019 then, which in any single tri\\nSubmitted filename: \\n\\n\\nPONE-D-24-02571R1Effect of matcha green tea on cognitive functions and sleep quality in older adults with cognitive decline: A randomized controlled study over 12 monthsPLOS ONE\\nDear Dr. Uchida,\\nThank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE\\u2019s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.\\nPlease submit your revised manuscript by Jul 26 2024 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at\\u00a0\\nPlease include the following items when submitting your revised manuscript:\\nA rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.\\nA marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.\\nAn unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.\\nIf applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: \\nWe look forward to receiving your revised manuscript.\\nKind regards,\\nValerio Manippa\\nAcademic Editor\\nPLOS ONE\\nJournal Requirements:\\nPlease review your reference list to ensure that it is complete and correct. If you have cited papers that have been retracted, please include the rationale for doing so in the manuscript text, or remove these references and replace them with relevant current references. Any changes to the reference list should be mentioned in the rebuttal letter that accompanies your revised manuscript. If you need to cite a retracted article, indicate the article\\u2019s retracted status in the References list and also include a citation and full reference for the retraction notice.\\n[Note: HTML markup is below. Please do not edit.]\\nReviewers' comments:\\nReviewer's Responses to Questions\\n\\n\\n\\n1. If the authors have adequately addressed your comments raised in a previous round of review and you feel that this manuscript is now acceptable for publication, you may indicate that here to bypass the \\u201cComments to the Author\\u201d section, enter your conflict of interest statement in the \\u201cConfidential to Editor\\u201d section, and submit your \\\"Accept\\\" recommendation.\\nReviewer #2:\\u00a0(No Response)\\nReviewer #3:\\u00a0All comments have been addressed\\n**********\\n2. Is the manuscript technically sound, and do the data support the conclusions?\\nThe manuscript must describe a technically sound piece of scientific research with data that supports the conclusions. Experiments must have been conducted rigorously, with appropriate controls, replication, and sample sizes. The conclusions must be drawn appropriately based on the data presented. \\nReviewer #2:\\u00a0(No Response)\\nReviewer #3:\\u00a0Yes\\n**********\\n3. Has the statistical analysis been performed appropriately and rigorously? \\nReviewer #2:\\u00a0(No Response)\\nReviewer #3:\\u00a0Yes\\n**********\\n4. Have the authors made all data underlying the findings in their manuscript fully available?\\nThe \\nReviewer #2:\\u00a0(No Response)\\nReviewer #3:\\u00a0Yes\\n**********\\n5. Is the manuscript presented in an intelligible fashion and written in standard English?\\nPLOS ONE does not copyedit accepted manuscripts, so the language in submitted articles must be clear, correct, and unambiguous. Any typographical or grammatical errors should be corrected at revision, so please note any specific errors here.\\nReviewer #2:\\u00a0(No Response)\\nReviewer #3:\\u00a0Yes\\n**********\\n6. Review Comments to the Author\\nPlease use the space provided to explain your answers to the questions above. You may also include additional comments for the author, including concerns about dual publication, research ethics, or publication ethics. (Please upload your review as an attachment if it exceeds 20,000 characters)\\nReviewer #2:\\u00a0COMMENTS: Since all of the comments made on earlier draft are considered positively & mostly attended satisfactorily, I recommend the acceptance after minor revision. Although many modifications could be seen clearly, few details/modifications given while addressing comments were/are not very convincing [example: account regarding choice of \\u2018Sample size\\u2019]. \\u2018Minor revision\\u2019 is recommended so that authors can re-visit earlier comments and further modify by taking clues from earlier comments. However, please do not limit the revision only (with respect) to comments made there. More improvement is expected {note that \\u2018clinical implications\\u2019 are valuable}.\\nReviewer #3:\\u00a0(No Response)\\n**********\\n7. PLOS authors have the option to publish the peer review history of their article (\\nIf you choose \\u201cno\\u201d, your identity will remain anonymous but your review may still be made public.\\nReviewer #2:\\u00a0No\\nReviewer #3:\\u00a0No\\n**********\\n[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link \\\"View Attachments\\\". If this link does not appear, there are no attachment files.]\\nWhile revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool,\\u00a0\\n\\n\\nResponse to the reviewer\\u2019s comments\\nReviewer 2:\\nCOMMENTS: \\nSince all of the comments made on earlier draft are considered positively & mostly attended satisfactorily, I recommend the acceptance after minor revision. Although many modifications could be seen clearly, few details/modifications given while addressing comments were/are not very convincing [example: account regarding choice of \\u2018Sample size\\u2019]. \\n\\u2018Minor revision\\u2019 is recommended so that authors can re-visit earlier comments and further modify by taking clues from earlier comments. However, please do not limit the revision only (with respect) to comments made there. More improvement is expected {note that \\u2018clinical implications\\u2019 are valuable}.\\nANSWER:\\nWe are sincerely grateful for the opportunity to further improve our manuscript. In response to your valuable feedback, we have deepened the discussion regarding the clinical implications and limitations of our findings. We have incorporated a paragraph that explicitly outlines how our results concerning the effects of matcha consumption may contribute to dementia prevention strategies (page 35, line 554 \\u2013 page 36, line 561). This paragraph also discusses the implications of our findings for the field of preventive medicine, thereby strengthening the connection between our research and its practical applications.\\nRegarding the chosen sample size, we acknowledge your feedback indicating that our rationale was not sufficiently articulated. To address this, we have anchored our determination of sample size in a robust statistical rationale, incorporating considerations such as power analysis and effect size, consistent with the guidelines recommended for studies of this nature. We have elaborated on this discussion in the revised manuscript, providing a more detailed justification and citing studies [30\\u201332] employing analogous methodologies (page 8, line 120 \\u2013 page 9, line 126).\\nWe have diligently reviewed the manuscript to ensure consistency of content. Additionally, we have verified the references to ensure they are complete and correct, deleted duplicated references, and added three new references.\\nWe added the following sentences and references in the Materials and Methods: \\nPage 8, line 120 \\u2013 Page 9, line 126\\nA meta-analysis was conducted to ascertain whether the effect size and statistical power were commensurate with the determined sample size, referencing three intervention studies that employed the Montreal Cognitive Assessment (MoCA) as the evaluative criterion [29\\u201331]. Although the efficacy indices in this study vary slightly from those in previous research, we concluded that observing the anticipated effects outlined in the meta-analysis would render the data from all 66 patients a significant evidence base for preparing the subsequent validation study.\\nReferences\\n30. Tsolaki M, Kounti F, Agogiatou C, Poptsi E, Bakoglidou E, Zafeiropoulou M, et al. Effectiveness of nonpharmacological approaches in patients with mild cognitive impairment. Neurodegener Dis. 2011;8: 138\\u2013145.\\n31. Marzolini S, Oh P, McIlroy W, Brooks D. The effects of an aerobic and resistance exercise training program on cognition following stroke. Neurorehabil Neural Repair. 2013;27: 392\\u2013402.\\n32. Sukontapol C, Kemsen S, Chansirikarn S, Nakawiro D, Kuha O, Taemeeyapradit U. The effectiveness of a cognitive training program in people with mild cognitive impairment: A study in urban community. Asian J Psychiatr. 2018;35: 18\\u201323.\\nPage 9, lines 132 \\u2013 134 \\nAnticipating a dropout rate of 7 of 40 recruited cases per group, we established a target sample size of at least 33 cases per group to validate the robustness of our findings (details are described in S1 File).\\nWe added the following sentences in Discussion.\\nPage 35, line 554 \\u2013 Page 36, line 561\\nThe beneficial effects of matcha consumption revealed in this study may contribute to dementia prevention strategies, as these are relatively easy lifestyle changes to implement in daily life. Matcha consumption improved facial emotion recognition, as assessed by a computerized neurocognitive battery, since conventional psychological tests may not be suitable for evaluating cognitive decline in very mild cases. Thus, more sensitive neuropsychological testing, along with intervention strategies that are easy for elderly people to maintain, is necessary to assess the efficacy of these interventions in dementia prevention.\\nSubmitted filename: \\n\\n\\nEffect of matcha green tea on cognitive functions and sleep quality in older adults with cognitive decline: A randomized controlled study over 12 months\\nPONE-D-24-02571R2\\nDear Dr. Uchida,\\nWe\\u2019re pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it meets all outstanding technical requirements.\\nWithin one week, you\\u2019ll receive an e-mail detailing the required amendments. When these have been addressed, you\\u2019ll receive a formal acceptance letter and your manuscript will be scheduled for publication.\\nAn invoice will be generated when your article is formally accepted. Please note, if your institution has a publishing partnership with PLOS and your article meets the relevant criteria, all or part of your publication costs will be covered. Please make sure your user information is up-to-date by logging into Editorial Manager at \\nIf your institution or institutions have a press office, please notify them about your upcoming paper to help maximize its impact. If they\\u2019ll be preparing press materials, please inform our press team as soon as possible -- no later than 48 hours after receiving the formal acceptance. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact \\nKind regards,\\nMadepalli K. Lakshmana, Ph.D\\nAcademic Editor\\nPLOS ONE\\nAdditional Editor Comments (optional):\\nReviewers' comments:\\nReviewer's Responses to Questions\\n\\n\\n\\n1. If the authors have adequately addressed your comments raised in a previous round of review and you feel that this manuscript is now acceptable for publication, you may indicate that here to bypass the \\u201cComments to the Author\\u201d section, enter your conflict of interest statement in the \\u201cConfidential to Editor\\u201d section, and submit your \\\"Accept\\\" recommendation.\\nReviewer #2:\\u00a0All comments have been addressed\\nReviewer #3:\\u00a0All comments have been addressed\\n**********\\n2. Is the manuscript technically sound, and do the data support the conclusions?\\nThe manuscript must describe a technically sound piece of scientific research with data that supports the conclusions. Experiments must have been conducted rigorously, with appropriate controls, replication, and sample sizes. The conclusions must be drawn appropriately based on the data presented. \\nReviewer #2:\\u00a0(No Response)\\nReviewer #3:\\u00a0Yes\\n**********\\n3. Has the statistical analysis been performed appropriately and rigorously? \\nReviewer #2:\\u00a0(No Response)\\nReviewer #3:\\u00a0Yes\\n**********\\n4. Have the authors made all data underlying the findings in their manuscript fully available?\\nThe \\nReviewer #2:\\u00a0(No Response)\\nReviewer #3:\\u00a0Yes\\n**********\\n5. Is the manuscript presented in an intelligible fashion and written in standard English?\\nPLOS ONE does not copyedit accepted manuscripts, so the language in submitted articles must be clear, correct, and unambiguous. Any typographical or grammatical errors should be corrected at revision, so please note any specific errors here.\\nReviewer #2:\\u00a0(No Response)\\nReviewer #3:\\u00a0Yes\\n**********\\n6. Review Comments to the Author\\nPlease use the space provided to explain your answers to the questions above. You may also include additional comments for the author, including concerns about dual publication, research ethics, or publication ethics. (Please upload your review as an attachment if it exceeds 20,000 characters)\\nReviewer #2:\\u00a0COMMENTS: Further to what I stated earlier that all of the comments made on original draft were attended satisfactorily, I add that comment made on revised draft also is attended and I recommend the acceptance without any hesitation now.\\nReviewer #3:\\u00a0(No Response)\\n**********\\n7. PLOS authors have the option to publish the peer review history of their article (\\nIf you choose \\u201cno\\u201d, your identity will remain anonymous but your review may still be made public.\\nReviewer #2:\\u00a0No\\nReviewer #3:\\u00a0No\\n**********\",\n          \"Introduction\\nProtein aggregation is associated with\\nmany human diseases, including\\nneurodegenerative disorders such as Alzheimer\\u2019s disease (AD)\\nGrowing evidence suggests\\nthat early-stage aggregates of A\\u03b2\\npeptide and tau protein are the most neurotoxic species in the brain\\nof affected individuals.\\nIn order to study the formation\\nof protein amyloids, several groups,\\nThioflavin\\nT (ThT) is the most widely used fluorescent dye to detect\\nprotein aggregation to amyloids \\nChemical structure of\\namyloid-responsive fluorescent molecules\\nand schematic illustration of protein aggregation kinetics. (A) Chemical\\nstructures of thioflavin T (ThT), amino naphthalene 2-cyanoacrylate-spiropyran\\n(AN-SP), and triazole-containing boron-dipyrromethene (taBODIPY) fluorophores.\\n(B) Schematic illustration of different phases (lag phase, elongation\\nphase, and steady/saturation phase) of the protein aggregation pathway.\\nThe insets indicate the presence of various amyloid particles with\\nchanging abundances at different stages of aggregation. These particles\\nrange from monomers, oligomers, and protofibrils to fibrils; the most\\nvisible particles (i.e., the ones with the lowest opacity) are the\\nspecies that are characteristic of each aggregation stage. The parameter \\nResults and Discussion\\nWe compared the two fluorescent\\ndyes AN-SP and taBODIPY with the\\nwell-established amyloid-responsive fluorophore ThT (\\nMonitoring Early-Stage Aggregation of A\\u03b2(1\\u201342)\\nAggregation kinetics\\nof A\\u03b2(1\\u201342) peptide. (A) Aggregation\\nof A\\u03b2(1\\u201342) monitored using AN-SP (orange) and blank\\ncontrol of AN-SP in the absence of A\\u03b2(1\\u201342) (gray). (B)\\nAggregation of A\\u03b2(1\\u201342) monitored using taBODIPY (purple)\\nand blank control of taBODIPY in the absence of A\\u03b2 1\\u201342\\n(gray). (C) Aggregation of A\\u03b2(1\\u201342) monitored using ThT\\n(green) and blank control of ThT in the absence of A\\u03b2(1\\u201342)\\n(gray). (D) Normalized aggregation curves after subtracting the blank\\ncontrol for AN-SP, taBODIPY, and ThT. The light green curve shows\\nthe results of fitting \\nIt is important to note the limitations of AN-SP and taBODIPY\\nin\\ndetecting the early stages of A\\u03b2 aggregation. \\nThe aggregation of\\namyloid proteins can vary from batch to batch,\\nwhich may affect the detection capabilities of AN-SP or taBODIPY for\\nearly-stage aggregation. To explore this effect, we performed an aggregation\\nexperiment using a fresh batch of A\\u03b2(1\\u201342) peptide that\\nwas prepared using a different method (SI \\nMonitoring Early-Stage Aggregation of K18-Tau\\nWe used\\na fast-aggregating isoform of tau, that is, K18-Tau, containing four\\nmicrotubule-binding repeats, in this study. \\nAggregation\\nkinetics of K18-tau protein. (A) Kinetics of K18-tau\\naggregation monitored using AN-SP (orange) along with the blank control\\nof AN-SP without K18-Tau (gray). (B) Kinetics of K18-tau aggregation\\nmonitored by taBODIPY (purple) along with the blank control without\\nprotein (gray). (C) Kinetics of K18-tau aggregation monitored by ThT\\n(green) and ThT fluorescence of the blank control (gray). (D) Comparison\\nof aggregation kinetics of AN-SP, taBODIPY, and ThT after subtracting\\nthe blank control and normalization. The light orange curve, light\\npurple curve, and light green curve show the results of fitting \\nComparison of the Half-Time \\nThe \\nMeaningful estimation of \\nSimilar\\nto the case of A\\u03b2(1\\u201342), we tested a possible\\neffect of the taBODIPY dye on the aggregation kinetics of tau protein.\\nAs with A\\u03b2(1\\u201342), we carried out \\nMonitoring Early-Stage Aggregation of IAPP\\nAggregation kinetics of islet amyloid polypeptide (IAPP).\\n(A) Kinetics\\nof IAPP aggregation monitored using AN-SP (orange) along with the\\nblank control of AN-SP without IAPP (gray). (B) Kinetics of IAPP aggregation\\nmonitored using taBODIPY (purple) along with the blank control (gray).\\n(C) Kinetics of IAPP aggregation monitored using ThT (green) along\\nwith the blank control (gray). (D) Comparison of normalized aggregation\\ncurves for AN-SP, taBODIPY, and ThT after subtracting the fluorescence\\nintensity of the blank control. The light orange curve, light pink\\ncurve, and light green curve result from fits of \\nNote, the lack of a plateau of fluorescence intensity\\nof AN-SP\\nin the steady phase of aggregation of A\\u03b2 and IAPP during the\\ntime frame of the experiment may originate from the formation of amyloid\\noligomers by secondary nucleation mechanisms, which results in a rapid\\nincrease in fluorescence during the elongation phase and a slow, steady\\nincrease during the plateau phase.\\nIn \\nMonitoring Early-Stage Aggregation of \\u03b1-Synuclein\\nFinally, we examined the ability of AN-SP and taBODIPY to detect\\nearly-stage aggregates of \\u03b1Syn. \\nAggregation kinetics of \\u03b1-synuclein protein. (A)\\nKinetics\\nof \\u03b1Syn aggregation monitored in the presence of AN-SP (orange)\\nalong with blank control samples of AN-SP without \\u03b1Syn (gray).\\n(B) Kinetics of \\u03b1Syn aggregation monitored using taBODIPY (purple)\\nalong with blank control samples of AN-SP without \\u03b1Syn (gray).\\n(C) Kinetics of \\u03b1Syn aggregation monitored using ThT (green)\\nalong with blank control samples of AN-SP without \\u03b1Syn (gray).\\n(D) Normalized fluorescence intensity after subtracting the blank\\ncontrol for AN-SP, taBODIPY, and ThT. The light green curve shows\\nthe amyloid aggregation fitting results for the ThT assays. The data\\npoints shown as filled circles are the average of at least three measurements.\\nThe shaded region shows the standard deviation of each data set. The\\nshaded gray region for the ThT blank control is not visible.\\nIn summary, we observed that both AN-SP and taBODIPY\\nwere able\\nto interact with early-stage aggregates of the A\\u03b2 peptide, K18-tau\\nprotein, and IAPP. By using four different amyloid-forming proteins,\\nwe were able to observe preferential binding of AN-SP and taBODIPY\\nto early-stage aggregation of amyloid proteins. For instance, AN-SP\\nperformed best with A\\u03b2, showing a modest increase in fluorescence\\nintensity at an early stage, whereas taBODIPY performed best with\\nK18-tau protein early-stage aggregates. Protein specificity of AN-SP\\nand taBODIPY dyes may enable the use of these dyes for selective detection\\nof soluble amyloid aggregates in biological samples, for example,\\nby comparing the response of these dyes with the response of ThT,\\nas done in this work.\\nConclusions\\nCurrent research focuses on early detection\\nof amyloid aggregates\\nin neurodegenerative diseases. Here, we explored the ability of two\\nfluorescent dyes to monitor the early-stage aggregation of four amyloidogenic\\nproteins. Key observations of this comparative analysis of monitoring\\nearly stages of amyloid aggregation are as follows: (1) AN-SP dye\\nis capable of detecting A\\u03b2(1\\u201342) early-stage aggregates\\nwith a modest increase in fluorescence. (2) taBODIPY responds early\\nto K18-tau protein aggregation. (3) Both AN-SP and taBODIPY exhibit\\nincreases in fluorescence intensity at an earlier stage than ThT during\\nIAPP aggregation, with taBODIPY being the first. (4) As opposed to\\nThT, neither AN-SP nor taBODIPY increase their fluorescence during\\naggregation of \\u03b1Syn. This study demonstrates the potential usefulness\\nof AN-SP and taBODIPY to detect early-stage protein aggregation of\\nA\\u03b2(1\\u201342), K18-Tau, and IAPP. The results show specificity\\nin the sense that an increase in fluorescence of ThT combined with\\nno increase in fluorescence of AN-SP or taBODIPY may indicate aggregation\\nof \\u03b1Syn. We hope that these results will inform researchers\\nabout the choice of the most appropriate fluorescent dyes depending\\non the amyloid-forming proteins under study.\\nMethods\\nMaterials\\nIAPP was obtained as a lyophilized powder\\nfrom AnaSpec (Cat. No. AS-60804). K18-tau protein was obtained from\\nNovus Biologicals (Cat. No. SP-496-100). A\\u03b2(1\\u201342) peptide\\nwas purchased from Bachem (Cat. No. 4014447.1000) as a lyophilized\\npowder. Recombinant \\u03b1-synuclein was obtained from rPeptide (Cat.\\nNo. S-1001-2). 1,1,1,3,3,3-Hexafluoro-2-propanol (HFIP) was purchased\\nfrom Sigma-Aldrich (Cat. No. 18127). Thioflavin T (ThT) (Cat. No.\\nT3516) was purchased from Sigma, USA, dissolved, and aliquoted in\\nwater. AN-SP was synthesized as previously reported.\\nWe carried\\nout \\nWe determined the kinetics of aggregation\\nbased on the following parameters in \\nHere, \\nWe prepared monomeric A\\u03b2(1\\u201342) as explained earlier.\\nWe\\ncarried out \\nWe used\\nHFIP treatment to monomerize IAPP before \\nWe prepared 100 \\u03bcL of a solution containing 100 mg/mL \\u03b1Syn\\nfrom Enzo Life Science and stored 5 \\u03bcL aliquots at \\u221280\\n\\u00b0C. We diluted these 5 \\u03bcL aliquots to a final \\u03b1Syn\\nconcentration of 10 \\u03bcM in 20 mM Tris-HCl buffer, pH 7.5, containing\\n100 mM NaCl and 1 mM MgCl\\nAmyloid Fibril Preparation\\nTwo 100 \\u03bcL of a solution\\nof A\\u03b2(1\\u201342) (20 \\u03bcM) were prepared in A\\u03b2 aggregation\\nbuffer. The protein samples were incubated at 37 \\u00b0C for 20 h.\\nThe same fibril sample preparation procedure was employed for K18-Tau\\n(40 \\u03bcM) and IAPP (20 \\u03bcM). Subsequently, AN-SP or taBODIPY\\nwere added to the samples at final concentrations of 4 and 1 \\u03bcM,\\nrespectively. Different concentrations of protein were achieved by\\ndiluting the sample with the aggregation buffer containing the same\\nconcentration of AN-SP or taBODIPY. Fluorescence emission spectra\\nwere obtained using a Fluorolog-3 (HORIBA Scientific) spectrofluorometer\\nwith excitation wavelengths of 430 and 500 nm, respectively. The relative\\nfluorescence intensity was calculated by dividing the peak fluorescence\\nintensity in the presence of proteins by the value in the absence\\nof proteins.\\nDLS Measurements for AN-SP Aggregates\\nDLS data were\\ncollected at a constant temperature (23 \\u00b0C) on a commercial goniometer\\ninstrument (3D LS Spectrometer, LS Instruments AG, Switzerland). The\\nprimary beam was formed by a linearly polarized and collimated laser\\nbeam (Cobolt 05-01 diode pumped solid state laser, \\u03bb = 660 nm, \",\n          \"1. Introduction\\nReactive oxygen species (ROS) and reactive nitrogen species (RNS) are generated by all aerobic cells, either from endogenous metabolic processes or exogenous sources. When produced in appropriate amounts, these species play beneficial roles such as generating energy via the electron transport chain, participating in defense mechanisms during infections, and acting as signaling molecules, among other functions. In a physiologically normal state, cells can tolerate certain levels of ROS and RNS to maintain redox homeostasis [\\nHowever, when there is an imbalance between the production and degradation of these free radicals, cellular damage can occur [\\nTo mitigate this redox imbalance, cells express various antioxidant enzymes, including catalase, superoxide dismutase, and additional enzymatic systems, to efficiently counteract the deleterious effects of excessive ROS and RNS. Additionally, some antioxidant molecules can be obtained through exogenous supplementation in the diet, such as vitamins C and E, phenolic acids, flavonoids, anthocyanins, and other plant-produced intermediate metabolites [\\nHistorically, various plant parts\\u2014leaves, flowers, fruits, seeds, and bark\\u2014have been utilized for medicinal purposes. This age-old knowledge, transmitted across generations, plays a crucial role in the research and development of new pharmaceuticals [\\nThe native tree of Chile, \\nIn folk medicine, the fruits of \\nFollowing the identification of the diverse traditional uses and nutritional contributions of \\nIn this context, this study aimed to evaluate the antioxidant effects of the ethanolic and aqueous extracts of Cha\\u00f1ar (\\n2. Materials and Methods\\n2.1. Materials\\nEthylenediaminetetraacetic acid (EDTA), gallic acid, ascorbic acid, quercetin, methionine, pirocatechol violet, riboflavin, and ammonium molybdate were obtained from Sigma-Aldrich Co. (St. Louis, MO, USA). Potassium ferricyanide, trichloroacetic acid, Folin\\u2013Ciocalteu reagent, and sulfuric acid were acquired from Merck (Darmstadt, Germany). 2,2-Diphenyl-1-picrylhydrazyl (DPPH) was sourced from Fluka (Seelze, Germany), and copper sulfate was obtained from Chemical Kinetics. Dulbecco\\u2019s Modified Eagle Medium (DMEM) and fetal bovine serum (FBS) were procured from CULTILAB (Campinas, SP, Brazil). Penicillin and streptomycin were sourced from Gibco (Fort Worth, TX, USA). All other reagents used were of analytical grade.\\n2.2. Plant Material\\n2.3. Extraction of Ethanolic and Aqueous Extracts from Geoffroea decorticans Seeds\\nPlant samples (seeds) were crushed/macerated to prepare extracts of varying polarity. One hundred grams of plant material was used per liter of solvent (1:10), following this order: hexane, chloroform, ethanol, methanol, and water [\\n2.4. Phytochemical Characterization\\n2.4.1. Total Phenolic Compound Determination\\nThe quantification of total phenolic compounds in the ethanolic and aqueous extracts was performed using the Folin\\u2013Ciocalteu colorimetric method, with gallic acid as a reference standard. A calibration curve for gallic acid was constructed using concentrations ranging from 0.1 mg/mL to 5 mg/mL (\\n2.4.2. Quantification of Total Flavonoid Content\\nThe concentration of flavonoids was determined using a modified aluminum chloride (AlCl\\n2.4.3. Identification of Phenolic Compounds by HPLC Followed by GC-MS/SIM\\nThe analysis of bioactive compounds was conducted using high-performance liquid chromatography (HPLC) followed by gas chromatography-mass spectrometry with selected ion monitoring (GC-MS/SIM). Initially, the samples were lyophilized and then dissolved in 1.5 mL of methanol for reverse-phase HPLC analysis using a Bondapak C18 column (3.9 mm \\u00d7 300 mm, Waters Corp., Milford, MA, USA). The elution was carried out at a flow rate of 1.5 mL/min, with the mobile phases programmed as follows: from 0 to 5 min, 28% methanol in 1% aqueous acetic acid (isocratic); from 5 to 35 min, a linear gradient from 28% to 86% methanol; from 35 to 36 min, 86% to 100% methanol; and from 36 to 40 min, 100% methanol (isocratic). A total of 50 fractions of 1.5 mL each were collected and prepared for GC-MS analysis in SIM mode (GC system 6890N and selective mass detector 5973, Agilent Technologies, Santa Clara, CA, USA). For each type of substance detected, 1 \\u03bcL of the sample was injected into a DB-1 capillary column (30 mm \\u00d7 0.25 mm ID, 0.25 \\u03bcm film thickness, J & W Scientific Co., Folsom, CA, USA). The GC oven temperature was programmed to start at 60 \\u00b0C for 1 min, followed by an increase of 15 \\u00b0C/min up to 200 \\u00b0C, and then 5 \\u00b0C/min to a final temperature of 285 \\u00b0C. Helium was used as the carrier gas at a pressure of 30 kPa. The GC was directly coupled to a selective mass spectrometer, set with an interface and source temperature of 280 \\u00b0C, an ionization voltage of 70 V, and a dwell time of 100 ms. The analysis was conducted in both full scan mode (initial test) and SIM mode, monitoring the nine main ions of the internal standards (phytol, vitexin, rutin, \\u03b1-tocopherol, Amonafide, Caffeic acid, Isovitexin, Epifriedelanol, and Leucocyanidin) along with the detected compounds. Retention times were determined using hydrocarbon standards, and the quantification of substances was based on the peak ratios of non-deuterated compounds (extracted) to deuterated standards.\\n2.5. Evaluation of In Vitro Antioxidant Activity\\nAntioxidant compounds can be categorized into two main groups: primary (chain-breaking) antioxidants and secondary (preventive) antioxidants. Primary antioxidants primarily function by donating hydrogen atoms to free radicals, effectively neutralizing them. On the other hand, secondary antioxidants operate through multiple mechanisms, including scavenging reactive oxygen species, and metal chelation.\\nIn this study, two types of \\n2.5.1. Total Antioxidant Capacity (TAC)\\nThe antioxidant capacity of the plant extracts was evaluated using a colorimetric test, with results expressed in milligrams of ascorbic acid equivalents per gram of extract (mgE-AA/g Extract). The TAC aims to evaluate the total electron donating power of a compound. The application of a high temperature, acidic pH, and a strong oxidizing agent ensures that all molecules with electron donating potential will be evaluated in the assay, even if their tendency to donate electrons is weak [\\nThis assay involves the reduction of Mo\\n2.5.2. Reducing Power\\nEthanolic and Aqueous Extracts were incubated with a phosphate buffer solution (0.2 M, pH 6.6) and 1% potassium ferricyanide at 50 \\u00b0C for 20 min, with a final volume of 4 mL. The reaction was terminated by adding 10% trichloroacetic acid, followed by homogenization with distilled water and 0.1% ferric chloride. Finally, the absorbance was measured at 700 nm using a spectrophotometer (Epoch Biotek) [\\n2.5.3. DPPH Radical Scavenging Assay\\nUsing a 96-well plate, the samples were added, followed by the addition of the DPPH solution. For the control, only DPPH was added to the wells. This mixture was then incubated in the dark at room temperature for 30 min. The absorbance was measured at 517 nm using a microplate reader (Epoch Biotek, Agilent Technologies, Santa Clara, CA, USA) [\\n2.5.4. Copper Chelation\\nFor this test, samples, 4 mM pirocatechol violet, and 50 \\u00b5g/mL copper sulfate pentahydrate (in acetate buffer) were added to each well of the microplate. The copper ion chelation capacity was measured by reading the absorbance at 632 nm [\\n2.6. Evaluation of Cytotoxic Activity on 3T3 Cell Line\\n2.6.1. MTT Assay (3T3)\\nThe EE and EA extracts were used to assess the effect of treatments on cell viability by reducing MTT to formazan. For this assay, the 3T3 cell line was cultured in 75 cm\\nA statistically significant reduction of MTT was considered as cellular cytotoxicity.\\n2.6.2. In Vitro Wound Healing Assay in 3T3 Cells\\nInitially, 3T3 cell line were plated at a density of 1 \\u00d7 10\\n2.6.3. Copper Sulfate (CuSO\\nThe methodology for conducting this assay was adapted from [\\n2.7. Evaluation of In Vivo Toxicity and Antioxidant Capacity\\n2.7.1. Animal Model\\u2014\\n2.7.2. Toxicity Assay of Extracts in \\nFor the assay, initially, 10 larvae weighing between 100 and 150 mg were selected per treatment group. EE and EA extract samples were prepared at concentrations of 100 and 250 \\u03bcg/mL with PBS. After separating the animals into petri dishes for their respective treatments, the animals were anesthetized on ice. Subsequently, 5 \\u03bcL of EE and EA extracts per treated group was applied between the second and third segments on the ventral side of the hemocoel using a Hamilton syringe (Hamilton Company, Reno, NV, USA). The negative control was inoculated only with PBS. The experiment lasted for 10 days, counting the surviving larvae every 24 h. Larvae with no movement and that were melanized were considered dead. The assay was conducted in triplicate, based on [\\n2.7.3. Copper Sulfate (CuSO\\nFor the assay, 10 \\n2.7.4. Melanization Assay Following Copper Sulfate (CuSO\\nMelanization quantification was performed according to [\\n3. Results\\n3.1. Antioxidant Potential by Biochemistry Assay\\nThe antioxidant effects of extracts from \\nThe Total Antioxidant Capacity (TAC) assay was used to measure the general antioxidant potential. For this assay, a concentration of 100 \\u00b5g/mL was selected for both the ethanolic (EE) and aqueous (EA) extracts. The results indicated that the antioxidant activity of EE was almost three times higher than that of the EA extract (\\nMoreover, in the DPPH assay, both extracts exhibited identical activities at all three tested concentrations (50 \\u00b5g/mL, 100 \\u00b5g/mL, and 250 \\u00b5g/mL), achieving 100% DPPH scavenging potential (\\n3.2. Extract Composition\\nFor the phenolic compounds, both extracts presented similar amounts (\\nIn addition, the identification of biomolecules using HPLC followed by GC-MS/SIM indicated the presence of phytol, vitexin, rutin, and \\u03b1-tocopherol, as detailed in \\n3.3. In Vitro Assays Using NHI/3T3 Cell Line\\nBoth extracts, EE and EA, at concentrations of 50 \\u00b5g/mL, 100 \\u00b5g/mL, and 250 \\u00b5g/mL exhibited no cytotoxicity to NIH/3T3 cells, as evidenced by nearly 100% MTT reduction (\\nBoth extracts, EE and EA, were evaluated in cell lines to determine whether they could protect against copper-induced oxidative stress. For the EE extract, no significant differences were observed between the different concentrations of 100 and 250 \\u00b5g/mL, but it was effective in protecting the cell line against oxidative stress compared to the positive control (\\n3.4. In Vivo Assays Using Tenebrio molitor Animal Model\\nThe previously obtained results for cell line treatments guided the choice of concentrations for this assay: 100 \\u00b5g/mL and 250 \\u00b5g/mL for both the EE and EA extracts from \\nAs the extracts from \\n4. Discussion\\nPlants are a source of diverse biomolecules that can be used as complementary therapies for treating various diseases and for drug development [\\nThe antioxidant activity is often associated with phenolic compounds and flavonoids due to their ability to act as ROS scavengers and free radical eliminators [\\nThe values reported in this study indicate that the EE and EA extracts from \\nThe in vitro tests, Total Antioxidant Capacity (TAC) and Reducing Power, assess the ability of extracts to donate electrons and prevent oxidative damage through a similar mechanism. Both the EE and EA extracts demonstrated excellent electron-donating capacities, exhibiting antioxidant potential comparable to \\nThe chelating power of copper ions, essential in the oxidative stress propagation phase involving the reaction with hydrogen peroxide to generate hydroxyl radicals (OH-) [\\nThe reducing potential of flavonoids, largely attributed to the hydroxyl groups in their structures, has gained recognition for their role in chelating metal ions, as demonstrated with rutin found in the extracts [\\nBoth extracts demonstrated excellent potential for scavenging DPPH radicals at all evaluated concentrations. Similar observations were made by Reynoso et al. [\\nThe search for biomolecules that can be widely used for their pharmacological activities has focused on compounds that can be integrated into biochemical and pharmacological regulatory standards and are generated by plants. To assess the phytochemical composition of the EE and EA extracts of \\nPhytol, a diterpene, is widely recognized for its antimicrobial, anticonvulsant, antinociceptive, anti-inflammatory, and antioxidant activities [\\nKnown as a key member of the vitamin E family, \\u03b1-tocopherol is characterized by its fat solubility and potent antioxidant properties. It primarily acts by neutralizing hydroxyl radicals (OH\\u2022) and singlet oxygen (\\nAnother compound identified in the EE and EA extracts, belonging to the flavonoids group, was rutin. This substance is often used as a standard to assess the antioxidant activity of crude extracts and is recognized for its diverse pharmacological properties, including cardioprotective, anti-inflammatory, anticancer, antiallergic, antidiabetic, and antioxidant effects [\\nThe presence of rutin, along with other compounds in the EE and EA extracts of \\nThe safety of the EE and EA extracts was assessed through cytotoxicity evaluations using the 3T3 mouse embryonic fibroblast cell line (ATCC CRL-1658). This cell line was chosen considering its relevance in processes such as inflammation, cell proliferation, and migration, which are crucial for wound healing and typically involve fibroblasts. The results demonstrated that the extracts, at various concentrations, did not exhibit cytotoxicity or compromise the metabolic activity of the 3T3 fibroblast cell line, thus confirming their safety for further trials. Additionally, the non-cytotoxic nature of these extracts was supported by similar findings from an MTT assay conducted on extracts from the leaves and roots of \\nTo assess the impact of the EE and EA extracts on the cell healing process, given their demonstrated lack of cytotoxicity, antioxidant capacity, and anti-inflammatory activity cited by other research groups [\\nOxidative stress can arise from exposure to various agents such as hydrogen peroxide, UV radiation, and heavy metals, including copper [\\nConsidering the in vitro copper-chelating potential of the EE and EA extracts, this activity was further evaluated in the 3T3 cell line under oxidative stress induced by copper sulfate. In this assay, both extracts demonstrated protective activity against oxidative stress, likely due to the bioactive compounds identified during phytochemical characterization. These biomolecules may act to eliminate or neutralize free radicals generated by copper sulfate, including those produced by the Fenton and Haber\\u2013Weiss reactions, which form hydroxyl radicals (OH\\u2022). Copper chelation is effective during both the initiation and propagation stages of free radical production [\\nThe use of cell lines is an excellent model for assessing cytotoxicity and the effects of extracts. Nevertheless, employing animal models is crucial to understand how extracts may act across different cells, tissues, and signaling pathways and to evaluate their safety concerning the toxicity of certain compounds after body metabolization [\\nThe toxicity evaluation of the EE and EA extracts from \\nGiven that heavy metals such as copper sulfate (CuSO\\nIt has been verified that \\nKey enzymes such as tyrosinase and phenol oxidase (PO) are integral to the immune response and the melanization process. Melanin production not only restricts oxygen and nutrient availability to pathogens, thereby affecting their survival, but the melanization process also generates reactive oxygen species (ROS) and reactive nitrogen species (RNS). These potentially harmful molecules need to be neutralized to prevent cellular damage. To counteract these effects, insects employ detoxifying enzymes and endogenous antioxidants, including superoxide dismutase, peroxidases, catalases, tyrosinase, acetylcholinesterase, carboxylesterase, and glutathione S-transferase. Exogenous antioxidants in their diet can further reinforce these defenses [\\nDespite the benefits of releasing certain toxic substances to combat invasive organisms and injuries, these substances can also harm the insects, potentially reducing their longevity. In studies involving the Cha\\u00f1ar extracts, it was observed that these extracts reduced the melanization content by approximately 45% in treated animals compared to those treated only with copper sulfate. Additionally, phytol, a compound identified during phytochemical characterization, may contribute to this reduction by inhibiting tyrosinase activity and the expression of proteins involved in melanogenesis [\\n5. Conclusions\\nThis study demonstrates the antioxidant potential and protective effects against oxidative stress induced by copper sulfate. The ethanolic (EE) and aqueous (EA) extracts derived from the seeds of \"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
              "type": "dataframe",
              "variable_name": "df"
            },
            "text/html": [
              "\n",
              "  <div id=\"df-d263bcdc-77d2-45d3-ad84-73ff14a9e3fa\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmid</th>\n",
              "      <th>pmcid</th>\n",
              "      <th>title</th>\n",
              "      <th>abstract</th>\n",
              "      <th>authors</th>\n",
              "      <th>methods_materials</th>\n",
              "      <th>results</th>\n",
              "      <th>conclusion</th>\n",
              "      <th>fulltext</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>39324129</td>\n",
              "      <td>11423800</td>\n",
              "      <td>Fast quantitative MRI: Spiral Acquisition Matc...</td>\n",
              "      <td>\\n         Conventional diagnostic images from...</td>\n",
              "      <td>\\n             Perera-Gonzalez \\n             ...</td>\n",
              "      <td>Supplementary Material\\n</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nMagnetic Resonance Imaging (MRI)...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>38973296</td>\n",
              "      <td>11423104</td>\n",
              "      <td>DCAF7 Acts as A Scaffold to Recruit USP10 for ...</td>\n",
              "      <td>Abstract Despite docetaxel combined with cispl...</td>\n",
              "      <td>Li Qing‐Jie, Fang Xue‐Liang, Li Ying‐Qin, Lin ...</td>\n",
              "      <td>Supporting information\\nSupporting Information</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nNasopharyngeal carcinoma (NPC), ...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>38851294</td>\n",
              "      <td>11422792</td>\n",
              "      <td>Opening the doors of precision medicine: novel...</td>\n",
              "      <td>Abstract Mounting evidence underscores the piv...</td>\n",
              "      <td>Iacucci Marietta, Santacroce Giovanni, Majumde...</td>\n",
              "      <td>supplementary material\\n</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>WHAT IS ALREADY KNOWN ON THIS TOPIC\\nThe intes...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>39328457</td>\n",
              "      <td>11422626</td>\n",
              "      <td>Evaluation of the cognitive, physiological, an...</td>\n",
              "      <td>Background and Aim: Individuals exposed to hea...</td>\n",
              "      <td>Mukhi Senna, Manjrekar Poornima Ajay, Srikanti...</td>\n",
              "      <td>Materials and Methods\\nThis study was approved...</td>\n",
              "      <td>Results\\nThe escape latency and time spent in ...</td>\n",
              "      <td>Conclusion\\nThe evaluation revealed the comple...</td>\n",
              "      <td>Introduction\\nThe possible health hazards asso...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>39318060</td>\n",
              "      <td>11422451</td>\n",
              "      <td>Clinical signature and associated immune metab...</td>\n",
              "      <td>Abstract Inflammations have been linked to tum...</td>\n",
              "      <td>Liao Yong, Yang Pinglian, Yang Cui, Zhuang Kai...</td>\n",
              "      <td>Supporting information\\n\\nData S1.\\n</td>\n",
              "      <td>RESULTS\\nNLRP1 expression pattern in different...</td>\n",
              "      <td>CONSENT\\nNot applicable.</td>\n",
              "      <td>INTRODUCTION\\nAccording to the statistics from...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>995</th>\n",
              "      <td>39157732</td>\n",
              "      <td>11328858</td>\n",
              "      <td>Nanomedicines Targeting Ferroptosis to Treat S...</td>\n",
              "      <td>Abstract Ferroptosis, a unique form of regulat...</td>\n",
              "      <td>Kang Hao, Meng Fansu, Liu Fengjie, Xie Mengjie...</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nCell death is crucial for mainta...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>996</th>\n",
              "      <td>39134987</td>\n",
              "      <td>11321064</td>\n",
              "      <td>Amniotic fluid proteomic analysis identifies I...</td>\n",
              "      <td>Background Despite extensive research, the ide...</td>\n",
              "      <td>Li Min, Liu Mengmei, Chen Ping, Duan Sifan, Li...</td>\n",
              "      <td>Electronic supplementary material\\nBelow is th...</td>\n",
              "      <td>Results\\nA total of 44 pregnant women were inc...</td>\n",
              "      <td>Conclusions\\nIn conclusion, our study indicate...</td>\n",
              "      <td>Background\\nPreterm birth (PTB), defined as de...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>997</th>\n",
              "      <td>39135397</td>\n",
              "      <td>11320689</td>\n",
              "      <td>Aesthetic Radiofrequency Associated with</td>\n",
              "      <td>The objective were to evaluate the effects of ...</td>\n",
              "      <td>Tremêa Greissi Tatieli Franke, Kleibert Karine...</td>\n",
              "      <td>Methods\\nThis was an experimental, longitudina...</td>\n",
              "      <td>Results\\nThe study had 32 participants, sociod...</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nRadiofrequency devices promote h...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>998</th>\n",
              "      <td>39133488</td>\n",
              "      <td>11320167</td>\n",
              "      <td>Anti-Inflammatory Diet and Dementia in Older A...</td>\n",
              "      <td>Key Points Question Can an anti-inflammatory d...</td>\n",
              "      <td>Dove Abigail, Dunk Michelle M., Wang Jiao, Guo...</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nCardiometabolic diseases (CMDs),...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>999</th>\n",
              "      <td>39134564</td>\n",
              "      <td>11319786</td>\n",
              "      <td>Multiple Transcriptomic Analyses Explore Poten...</td>\n",
              "      <td>Alzheimer’s disease (AD) is a prevalent neurod...</td>\n",
              "      <td>Nguyen Doan Phuong Quy, Pham Son, Jallow Amado...</td>\n",
              "      <td>Supplementary Information\\n\\n Supplementary Fi...</td>\n",
              "      <td>None</td>\n",
              "      <td>None</td>\n",
              "      <td>Introduction\\nAlzheimer’s disease (AD) is a ne...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>1000 rows × 9 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-d263bcdc-77d2-45d3-ad84-73ff14a9e3fa')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-d263bcdc-77d2-45d3-ad84-73ff14a9e3fa button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-d263bcdc-77d2-45d3-ad84-73ff14a9e3fa');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-767c9cfe-be97-4faa-9acc-f5f56eb7bb52\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-767c9cfe-be97-4faa-9acc-f5f56eb7bb52')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-767c9cfe-be97-4faa-9acc-f5f56eb7bb52 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "  <div id=\"id_066e6e3c-7c53-44f4-babe-75a804ef347a\">\n",
              "    <style>\n",
              "      .colab-df-generate {\n",
              "        background-color: #E8F0FE;\n",
              "        border: none;\n",
              "        border-radius: 50%;\n",
              "        cursor: pointer;\n",
              "        display: none;\n",
              "        fill: #1967D2;\n",
              "        height: 32px;\n",
              "        padding: 0 0 0 0;\n",
              "        width: 32px;\n",
              "      }\n",
              "\n",
              "      .colab-df-generate:hover {\n",
              "        background-color: #E2EBFA;\n",
              "        box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "        fill: #174EA6;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate {\n",
              "        background-color: #3B4455;\n",
              "        fill: #D2E3FC;\n",
              "      }\n",
              "\n",
              "      [theme=dark] .colab-df-generate:hover {\n",
              "        background-color: #434B5C;\n",
              "        box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "        filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "        fill: #FFFFFF;\n",
              "      }\n",
              "    </style>\n",
              "    <button class=\"colab-df-generate\" onclick=\"generateWithVariable('df')\"\n",
              "            title=\"Generate code using this dataframe.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M7,19H8.4L18.45,9,17,7.55,7,17.6ZM5,21V16.75L18.45,3.32a2,2,0,0,1,2.83,0l1.4,1.43a1.91,1.91,0,0,1,.58,1.4,1.91,1.91,0,0,1-.58,1.4L9.25,21ZM18.45,9,17,7.55Zm-12,3A5.31,5.31,0,0,0,4.9,8.1,5.31,5.31,0,0,0,1,6.5,5.31,5.31,0,0,0,4.9,4.9,5.31,5.31,0,0,0,6.5,1,5.31,5.31,0,0,0,8.1,4.9,5.31,5.31,0,0,0,12,6.5,5.46,5.46,0,0,0,6.5,12Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "    <script>\n",
              "      (() => {\n",
              "      const buttonEl =\n",
              "        document.querySelector('#id_066e6e3c-7c53-44f4-babe-75a804ef347a button.colab-df-generate');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      buttonEl.onclick = () => {\n",
              "        google.colab.notebook.generateWithVariable('df');\n",
              "      }\n",
              "      })();\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "text/plain": [
              "         pmid     pmcid                                              title  \\\n",
              "0    39324129  11423800  Fast quantitative MRI: Spiral Acquisition Matc...   \n",
              "1    38973296  11423104  DCAF7 Acts as A Scaffold to Recruit USP10 for ...   \n",
              "2    38851294  11422792  Opening the doors of precision medicine: novel...   \n",
              "3    39328457  11422626  Evaluation of the cognitive, physiological, an...   \n",
              "4    39318060  11422451  Clinical signature and associated immune metab...   \n",
              "..        ...       ...                                                ...   \n",
              "995  39157732  11328858  Nanomedicines Targeting Ferroptosis to Treat S...   \n",
              "996  39134987  11321064  Amniotic fluid proteomic analysis identifies I...   \n",
              "997  39135397  11320689          Aesthetic Radiofrequency Associated with    \n",
              "998  39133488  11320167  Anti-Inflammatory Diet and Dementia in Older A...   \n",
              "999  39134564  11319786  Multiple Transcriptomic Analyses Explore Poten...   \n",
              "\n",
              "                                              abstract  \\\n",
              "0    \\n         Conventional diagnostic images from...   \n",
              "1    Abstract Despite docetaxel combined with cispl...   \n",
              "2    Abstract Mounting evidence underscores the piv...   \n",
              "3    Background and Aim: Individuals exposed to hea...   \n",
              "4    Abstract Inflammations have been linked to tum...   \n",
              "..                                                 ...   \n",
              "995  Abstract Ferroptosis, a unique form of regulat...   \n",
              "996  Background Despite extensive research, the ide...   \n",
              "997  The objective were to evaluate the effects of ...   \n",
              "998  Key Points Question Can an anti-inflammatory d...   \n",
              "999  Alzheimer’s disease (AD) is a prevalent neurod...   \n",
              "\n",
              "                                               authors  \\\n",
              "0    \\n             Perera-Gonzalez \\n             ...   \n",
              "1    Li Qing‐Jie, Fang Xue‐Liang, Li Ying‐Qin, Lin ...   \n",
              "2    Iacucci Marietta, Santacroce Giovanni, Majumde...   \n",
              "3    Mukhi Senna, Manjrekar Poornima Ajay, Srikanti...   \n",
              "4    Liao Yong, Yang Pinglian, Yang Cui, Zhuang Kai...   \n",
              "..                                                 ...   \n",
              "995  Kang Hao, Meng Fansu, Liu Fengjie, Xie Mengjie...   \n",
              "996  Li Min, Liu Mengmei, Chen Ping, Duan Sifan, Li...   \n",
              "997  Tremêa Greissi Tatieli Franke, Kleibert Karine...   \n",
              "998  Dove Abigail, Dunk Michelle M., Wang Jiao, Guo...   \n",
              "999  Nguyen Doan Phuong Quy, Pham Son, Jallow Amado...   \n",
              "\n",
              "                                     methods_materials  \\\n",
              "0                             Supplementary Material\\n   \n",
              "1       Supporting information\\nSupporting Information   \n",
              "2                             supplementary material\\n   \n",
              "3    Materials and Methods\\nThis study was approved...   \n",
              "4                 Supporting information\\n\\nData S1.\\n   \n",
              "..                                                 ...   \n",
              "995                                               None   \n",
              "996  Electronic supplementary material\\nBelow is th...   \n",
              "997  Methods\\nThis was an experimental, longitudina...   \n",
              "998                                               None   \n",
              "999  Supplementary Information\\n\\n Supplementary Fi...   \n",
              "\n",
              "                                               results  \\\n",
              "0                                                 None   \n",
              "1                                                 None   \n",
              "2                                                 None   \n",
              "3    Results\\nThe escape latency and time spent in ...   \n",
              "4    RESULTS\\nNLRP1 expression pattern in different...   \n",
              "..                                                 ...   \n",
              "995                                               None   \n",
              "996  Results\\nA total of 44 pregnant women were inc...   \n",
              "997  Results\\nThe study had 32 participants, sociod...   \n",
              "998                                               None   \n",
              "999                                               None   \n",
              "\n",
              "                                            conclusion  \\\n",
              "0                                                 None   \n",
              "1                                                 None   \n",
              "2                                                 None   \n",
              "3    Conclusion\\nThe evaluation revealed the comple...   \n",
              "4                             CONSENT\\nNot applicable.   \n",
              "..                                                 ...   \n",
              "995                                               None   \n",
              "996  Conclusions\\nIn conclusion, our study indicate...   \n",
              "997                                               None   \n",
              "998                                               None   \n",
              "999                                               None   \n",
              "\n",
              "                                              fulltext  \n",
              "0    Introduction\\nMagnetic Resonance Imaging (MRI)...  \n",
              "1    Introduction\\nNasopharyngeal carcinoma (NPC), ...  \n",
              "2    WHAT IS ALREADY KNOWN ON THIS TOPIC\\nThe intes...  \n",
              "3    Introduction\\nThe possible health hazards asso...  \n",
              "4    INTRODUCTION\\nAccording to the statistics from...  \n",
              "..                                                 ...  \n",
              "995  Introduction\\nCell death is crucial for mainta...  \n",
              "996  Background\\nPreterm birth (PTB), defined as de...  \n",
              "997  Introduction\\nRadiofrequency devices promote h...  \n",
              "998  Introduction\\nCardiometabolic diseases (CMDs),...  \n",
              "999  Introduction\\nAlzheimer’s disease (AD) is a ne...  \n",
              "\n",
              "[1000 rows x 9 columns]"
            ]
          },
          "execution_count": 285,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "df"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "A_xnffe27kW7"
      },
      "outputs": [],
      "source": [
        "df.to_csv(\"/content/alzheimer_biomarker_2000.csv\", index=False)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "2QxvF35c_-Jd",
        "outputId": "345ba5eb-702b-4cde-ea76-bd762b551ff3"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\u001b[?25l     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m0.0/86.0 kB\u001b[0m \u001b[31m?\u001b[0m eta \u001b[36m-:--:--\u001b[0m\r",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m86.0/86.0 kB\u001b[0m \u001b[31m5.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m50.4/50.4 kB\u001b[0m \u001b[31m3.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m244.8/244.8 kB\u001b[0m \u001b[31m11.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.0/1.0 MB\u001b[0m \u001b[31m31.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m383.7/383.7 kB\u001b[0m \u001b[31m22.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m249.2/249.2 kB\u001b[0m \u001b[31m18.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m471.6/471.6 kB\u001b[0m \u001b[31m29.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m139.1/139.1 kB\u001b[0m \u001b[31m9.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m116.3/116.3 kB\u001b[0m \u001b[31m9.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.1/3.1 MB\u001b[0m \u001b[31m61.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m76.4/76.4 kB\u001b[0m \u001b[31m5.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m78.0/78.0 kB\u001b[0m \u001b[31m6.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m325.2/325.2 kB\u001b[0m \u001b[31m20.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m404.4/404.4 kB\u001b[0m \u001b[31m24.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m295.8/295.8 kB\u001b[0m \u001b[31m15.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m85.4/85.4 kB\u001b[0m \u001b[31m5.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.6/1.6 MB\u001b[0m \u001b[31m48.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m134.8/134.8 kB\u001b[0m \u001b[31m10.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m194.1/194.1 kB\u001b[0m \u001b[31m12.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m60.8/60.8 kB\u001b[0m \u001b[31m4.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m12.6/12.6 MB\u001b[0m \u001b[31m73.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m141.9/141.9 kB\u001b[0m \u001b[31m11.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m54.5/54.5 kB\u001b[0m \u001b[31m4.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m82.6/82.6 kB\u001b[0m \u001b[31m6.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m388.4/388.4 kB\u001b[0m \u001b[31m26.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m147.8/147.8 kB\u001b[0m \u001b[31m10.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m82.1/82.1 kB\u001b[0m \u001b[31m6.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m58.3/58.3 kB\u001b[0m \u001b[31m4.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m26.7/26.7 MB\u001b[0m \u001b[31m56.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m5.6/5.6 MB\u001b[0m \u001b[31m89.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m56.8/56.8 kB\u001b[0m \u001b[31m4.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m233.2/233.2 kB\u001b[0m \u001b[31m15.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m128.2/128.2 kB\u001b[0m \u001b[31m9.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m85.0/85.0 kB\u001b[0m \u001b[31m6.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m52.8/52.8 kB\u001b[0m \u001b[31m3.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m565.6/565.6 kB\u001b[0m \u001b[31m30.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m207.3/207.3 kB\u001b[0m \u001b[31m15.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m203.2/203.2 kB\u001b[0m \u001b[31m14.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m78.6/78.6 kB\u001b[0m \u001b[31m5.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m62.7/62.7 kB\u001b[0m \u001b[31m4.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Building wheel for sentence-transformers (setup.py) ... \u001b[?25l\u001b[?25hdone\n"
          ]
        }
      ],
      "source": [
        "!pip install -qU \\\n",
        "  pinecone-client \\\n",
        "  sentence-transformers==2.2.2 \\\n",
        "  langchain \\\n",
        "  openai \\\n",
        "  cohere \\\n",
        "  datasets\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "tSHSd8y3__2o"
      },
      "outputs": [],
      "source": [
        "from pinecone import Pinecone, ServerlessSpec\n",
        "import cohere\n",
        "import os\n",
        "from typing import List, Dict\n",
        "import time\n",
        "\n",
        "from datasets import load_dataset"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "OSNT4HhRXD6v"
      },
      "outputs": [],
      "source": [
        "co = cohere.Client(COHERE_API_KEY)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "CX72QHnaZXj3"
      },
      "outputs": [],
      "source": [
        "papers1 = pd.read_csv(\"/content/alzheimer_biomarker_1000.csv\")\n",
        "papers2 = pd.read_csv(\"/content/alzheimer_biomarker_2000.csv\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "Ww5S91yo1wpO"
      },
      "outputs": [],
      "source": [
        "pc = Pinecone(PINECONE_API_KEY)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "6FQFZ--P1hyA"
      },
      "outputs": [],
      "source": [
        "index_name = \"scientific-papers\"\n",
        "if index_name not in pc.list_indexes().names():\n",
        "    pc.create_index(index_name, dimension=1024, metric=\"cosine\",\n",
        "                    spec=ServerlessSpec(\n",
        "                            cloud=\"aws\",\n",
        "                            region=\"us-east-1\")\n",
        "                    )\n",
        "\n",
        "index = pc.Index(index_name)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "0lcssB5P2-kP",
        "outputId": "9a3c208e-081c-4f33-fab7-5346e6fd6501"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "{'dimension': 1024,\n",
              " 'index_fullness': 0.0,\n",
              " 'namespaces': {},\n",
              " 'total_vector_count': 0}"
            ]
          },
          "execution_count": 26,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "index.describe_index_stats()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "_AIvfyae44YQ"
      },
      "outputs": [],
      "source": [
        "# embed = papers1[\"fulltext\"][:3].tolist()\n",
        "\n",
        "# meta = [{\n",
        "#     \"id\": i,\n",
        "#     \"paper_pmid\": paper['pmid'],\n",
        "#     \"paper_pmic\": paper['pmcid'],\n",
        "#     \"title\": paper['title'],\n",
        "#     \"abstract\": paper['abstract'],\n",
        "#     \"authors\": paper['authors'],\n",
        "#     \"methods_materials\": paper[\"methods_materials\"],\n",
        "#     \"results\": paper[\"results\"],\n",
        "#     \"conclusion\": paper[\"conclusion\"]\n",
        "#     } for i,paper in papers1.iloc[:3].iterrows()\n",
        "# ]\n",
        "\n",
        "# list(zip(meta, embed))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "VTfkhM8h4aE7"
      },
      "outputs": [],
      "source": [
        "def process_and_store_papers(papers_df, embedding=None):\n",
        "    ### START OF CODE FROM EXTERNAL SOURCE https://docs.pinecone.io/guides/get-started/quickstart\n",
        "    index_name = \"scientific-papers\"\n",
        "    if index_name not in pc.list_indexes().names():\n",
        "        pc.create_index(index_name, dimension=1024, metric=\"cosine\",\n",
        "                        spec=ServerlessSpec(\n",
        "                                cloud=\"aws\",\n",
        "                                region=\"us-east-1\"))\n",
        "\n",
        "    index = pc.Index(index_name)\n",
        "    print(f\"Chosen index: {index_name}\")\n",
        "\n",
        "    vc = index.describe_index_stats()[\"total_vector_count\"]\n",
        "\n",
        "    # ids = [\"0\", \"1\", ...]\n",
        "    ids = [str(i) for i in range(vc, len(papers_df)+vc)]\n",
        "\n",
        "    # metadata = [{meta: data}, {meta: data}, ...]\n",
        "    metadata = [{\"paper_pmid\": paper['pmid'],\n",
        "                \"paper_pmic\": paper['pmcid'],\n",
        "                \"title\": paper['title'],\n",
        "                \"conclusion\": paper[\"conclusion\"]\n",
        "                    } for i,paper in papers_df.iterrows()]\n",
        "\n",
        "    if embedding is None:\n",
        "        # pappers = [paperfulltext, paperfulltext, ...]\n",
        "        papers = papers_df[\"fulltext\"].tolist()\n",
        "\n",
        "        embedding = co.embed(\n",
        "                texts=papers,\n",
        "                model=\"embed-english-v3.0\",\n",
        "                input_type=\"search_document\"\n",
        "            ).embeddings\n",
        "\n",
        "    upsert_data = list(zip(ids,embedding,metadata))\n",
        "    print(\"Upsert data created\")\n",
        "\n",
        "    batch_size = 128\n",
        "    failed_idx = []\n",
        "    shape = np.array(embedding).shape\n",
        "    for i in tqdm(range(0, shape[0], batch_size)):\n",
        "        i_end = min(i+batch_size, shape[0])\n",
        "        try:\n",
        "            index.upsert(vectors=upsert_data[i:i_end])\n",
        "        except:\n",
        "            print(f\"index: [{i}: {i_end}] upserting unsuccessfull]\")\n",
        "            failed_idx.append((i,i_end))\n",
        "\n",
        "    print(index.describe_index_stats())\n",
        "\n",
        "    if failed_idx != []:\n",
        "        return upsert_data, failed_idx\n",
        "\n",
        "    return [], []\n",
        "\n",
        "def upsert_failed_indexes(upsert_data, failed_idx):\n",
        "    index_name = \"scientific-papers\"\n",
        "    if index_name not in pc.list_indexes().names():\n",
        "        pc.create_index(index_name, dimension=1024, metric=\"cosine\",\n",
        "                        spec=ServerlessSpec(\n",
        "                                cloud=\"aws\",\n",
        "                                region=\"us-east-1\"))\n",
        "\n",
        "    index = pc.Index(index_name)\n",
        "    print(f\"Chosen index: {index_name}\")\n",
        "\n",
        "    refailed_batch = []\n",
        "\n",
        "    for idxs in failed_idx:\n",
        "        start, end = idxs\n",
        "        for i in tqdm(range(start, end+1)):\n",
        "            try:\n",
        "                index.upsert(vectors=upsert_data[i])\n",
        "            except:\n",
        "                print(f\"index: {i} upserting unsuccessfull\")\n",
        "                refailed_batch.append(i)\n",
        "\n",
        "    print(index.describe_index_stats())\n",
        "    ### END OF CODE FROM EXTERNAL SOURCE https://docs.pinecone.io/guides/get-started/quickstart\n",
        "\n",
        "    return refailed_batch\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "def create_cohere_embeddings(papers_df):\n",
        "    ### START OF CODE FROM EXTERNAL SOURCE https://docs.cohere.com/v2/reference/embed\n",
        "    # pappers = [paperfulltext, paperfulltext, ...]\n",
        "    papers = papers_df[\"fulltext\"].tolist()\n",
        "\n",
        "    embedding = []\n",
        "    batch_size = 100\n",
        "\n",
        "    for i in tqdm(range(0, len(papers), batch_size)):\n",
        "\n",
        "        i_end = min(i+batch_size, len(papers))\n",
        "        # DOCUMENTATION CODE https://docs.cohere.com/v2/reference/embed\n",
        "        embedding = embedding + co.embed(\n",
        "                texts=papers[i:i_end],\n",
        "                model=\"embed-english-v3.0\",\n",
        "                input_type=\"search_document\"\n",
        "            ).embeddings\n",
        "        # END OF DOCUMENTATION CODE\n",
        "\n",
        "        time.sleep(60)\n",
        "    ### END OF CODE FROM EXTERNAL SOURCE https://docs.cohere.com/v2/reference/embed\n",
        "\n",
        "    return embedding"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "GGV4SxMzAZs3",
        "outputId": "c08ae8ea-30d7-4437-9d7b-d42d88468df5"
      },
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "100%|██████████| 10/10 [10:39<00:00, 63.94s/it]\n"
          ]
        }
      ],
      "source": [
        "embedding = create_cohere_embeddings(papers1.dropna(subset=[\"fulltext\"]))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "IqzoWW_fBJyg"
      },
      "outputs": [],
      "source": [
        "np.save(arr=np.array(embedding), file=\"dataset1_999.npy\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VCTpa6HdagbR",
        "outputId": "aaacd224-3e97-4bad-8e7c-68b4ea076ea1"
      },
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "100%|██████████| 10/10 [10:44<00:00, 64.46s/it]\n"
          ]
        }
      ],
      "source": [
        "embedding2 = create_cohere_embeddings(papers2.dropna(subset=[\"fulltext\"]))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "2NxD-W6WLkpP"
      },
      "outputs": [],
      "source": [
        "np.save(arr=np.array(embedding2), file=\"dataset2_998.npy\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "PmaXbAfPNHoJ",
        "outputId": "966d55a8-f4a9-4aee-f1bb-de045aa578ed"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Chosen index: scientific-papers\n",
            "Upsert data created\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "100%|██████████| 8/8 [00:09<00:00,  1.18s/it]"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{'dimension': 1024,\n",
            " 'index_fullness': 0.0,\n",
            " 'namespaces': {'': {'vector_count': 999}},\n",
            " 'total_vector_count': 999}\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\n"
          ]
        }
      ],
      "source": [
        "upd, failed_idx = process_and_store_papers(papers1.dropna(subset=[\"fulltext\"]).astype(str), embedding)"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "upd, failed_idx = process_and_store_papers(papers2.dropna(subset=[\"fulltext\"]).astype(str), embedding2)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Y2vSRQB8Sykn",
        "outputId": "051284c4-71a7-4553-8263-8d570bc7dea6"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Chosen index: scientific-papers\n",
            "Upsert data created\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "100%|██████████| 8/8 [00:07<00:00,  1.05it/s]"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{'dimension': 1024,\n",
            " 'index_fullness': 0.0,\n",
            " 'namespaces': {'': {'vector_count': 2023}},\n",
            " 'total_vector_count': 2023}\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# _ = upsert_failed_indexes(upd, failed_idx)"
      ],
      "metadata": {
        "id": "R47gHVi2VLB0"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "# start, end = failed_idx[0]\n",
        "# print(f\"{start} to {end} were not uploaded\")"
      ],
      "metadata": {
        "id": "5QZDzytjXVbS"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "example_kw = [\n",
        "    \"Alzheimer's disease biomarkers\",\n",
        "    \"Early detection of Alzheimer's\",\n",
        "    \"Blood-based biomarkers for Alzheimer's\",\n",
        "    \"Amyloid PET imaging in Alzheimer's\",\n",
        "    \"Tau protein as Alzheimer's biomarker\",\n",
        "    \"Cerebrospinal fluid biomarkers in Alzheimer's\",\n",
        "    \"Genetic risk factors for Alzheimer's\",\n",
        "    \"Neurofilament light chain in Alzheimer's\",\n",
        "    \"Lipid biomarkers for Alzheimer's\",\n",
        "    \"Alzheimer's disease progression markers\",\n",
        "    \"Non-invasive Alzheimer's diagnostic tools\",\n",
        "    \"Molecular imaging in Alzheimer's diagnosis\",\n",
        "    \"Cognitive decline markers in Alzheimer's\",\n",
        "    \"Predictive biomarkers for Alzheimer's\",\n",
        "    \"Alzheimer's disease neuroimaging initiative\",\n",
        "    \"Proteomics in Alzheimer's research\",\n",
        "    \"Fluid biomarkers for Alzheimer's\",\n",
        "    \"Eye and vision changes in Alzheimer's\",\n",
        "    \"Sleep disturbances as Alzheimer's biomarkers\",\n",
        "    \"Standardization of Alzheimer's biomarkers\",\n",
        "]"
      ],
      "metadata": {
        "id": "BGyyKIClY7il"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "KaNoiycBUSSj"
      },
      "outputs": [],
      "source": [
        "def retrieve_similar_articles(keyword: str, k: int = 5):\n",
        "    index_name = \"scientific-papers\"\n",
        "    index = pc.Index(index_name)\n",
        "    print(f\"Opened index {index_name}\")\n",
        "\n",
        "    # DOCUMENTATION CODE https://docs.cohere.com/v2/reference/embed\n",
        "    query_embedding = co.embed(\n",
        "        texts=[keyword],\n",
        "        model=\"embed-english-v3.0\",\n",
        "        input_type=\"search_query\"\n",
        "    ).embeddings\n",
        "    # END OF DOCUMENTATION CODE https://docs.cohere.com/v2/reference/embed\n",
        "\n",
        "    ### DOCUMENTATION CODE https://docs.pinecone.io/guides/get-started/quickstart\n",
        "    results = index.query(vector=query_embedding, top_k=k, include_metadata=True)\n",
        "    ### END OF DOCUMENTATION CODE\n",
        "\n",
        "    similar_articles = []\n",
        "    for match in results['matches']:\n",
        "        similar_articles.append({\n",
        "            \"paper_id\": match['id'],\n",
        "            \"pmcid\": match[\"metadata\"][\"paper_pmic\"],\n",
        "            \"pmid\": match[\"metadata\"][\"paper_pmid\"],\n",
        "            \"title\": match['metadata']['title'],\n",
        "            \"score\": match['score']\n",
        "        })\n",
        "\n",
        "    return similar_articles\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "j_gi-B4W9GkC",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6f5dc1b1-2db2-4c0c-d2ec-881a76e9d1c8"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Opened index scientific-papers\n",
            "Paper ID: 234\n",
            "Title: From Organotypic Mouse Brain Slices to Human Alzheimer Plasma Biomarkers: A Focus on Microglia\n",
            "Similarity Score: 0.738853335\n",
            "---\n",
            "Paper ID: 958\n",
            "Title: Peptide-Bound Glycative, AGE and Oxidative Modifications as Biomarkers for the Diagnosis of Alzheimer’s Disease—A Feasibility Study\n",
            "Similarity Score: 0.725872576\n",
            "---\n",
            "Paper ID: 391\n",
            "Title: Functional Nanomaterials for the Diagnosis of Alzheimer’s Disease: Recent Progress and Future Perspectives\n",
            "Similarity Score: 0.716538608\n",
            "---\n",
            "Paper ID: 977\n",
            "Title: Blood-based biomarkers in the oldest old: towards Alzheimer's disease detection in primary care\n",
            "Similarity Score: 0.713781536\n",
            "---\n",
            "Paper ID: 1756\n",
            "Title: Performance of plasma p-tau217 for the detection of amyloid-β positivity in a memory clinic cohort using an electrochemiluminescence immunoassay\n",
            "Similarity Score: 0.706662\n",
            "---\n"
          ]
        }
      ],
      "source": [
        "keyword = \"Alzheimer's disease biomarkers\"\n",
        "similar_articles = retrieve_similar_articles(keyword, k=5)\n",
        "for article in similar_articles:\n",
        "    print(f\"Paper ID: {article['paper_id']}\")\n",
        "    print(f\"Title: {article['title']}\")\n",
        "    print(f\"Similarity Score: {article['score']}\")\n",
        "    print(\"---\")"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "keyword = \"Genetic risk factors for Alzheimer's\"\n",
        "similar_articles = retrieve_similar_articles(keyword, k=5)\n",
        "for article in similar_articles:\n",
        "    print(f\"Paper ID: {article['paper_id']}\")\n",
        "    print(f\"Title: {article['title']}\")\n",
        "    print(f\"Similarity Score: {article['score']}\")\n",
        "    print(\"---\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "QROGUW2OcGIC",
        "outputId": "f8449cdb-85c2-4baf-d668-78c93d34dbdb"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Opened index scientific-papers\n",
            "Paper ID: 56\n",
            "Title: Exploring first-degree family history in a cohort of Portuguese Alzheimer’s disease patients: population evidence for X-chromosome linked and recessive inheritance of risk factors\n",
            "Similarity Score: 0.724336207\n",
            "---\n",
            "Paper ID: 556\n",
            "Title: Body Composition and Alzheimer’s Disease: A Holistic Review\n",
            "Similarity Score: 0.719978154\n",
            "---\n",
            "Paper ID: 40\n",
            "Title: Connecting dementia risk loci to the CSF proteome identifies pathophysiological leads for dementia\n",
            "Similarity Score: 0.708768308\n",
            "---\n",
            "Paper ID: 1562\n",
            "Title: Impact of Apolipoprotein E ε4 in Alzheimer’s Disease: Insights From a Meta-Analysis\n",
            "Similarity Score: 0.706729531\n",
            "---\n",
            "Paper ID: 127\n",
            "Title: Coupling of Alzheimer’s Disease Genetic Risk Factors with Viral Susceptibility and Inflammation\n",
            "Similarity Score: 0.698289931\n",
            "---\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "ew2Q5s11cjME"
      },
      "execution_count": null,
      "outputs": []
    }
  ],
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3 (ipykernel)",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.12.4"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}